{
  "cells": [
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "AyWK0F86W9hl",
        "outputId": "c7558e1f-a31f-4fff-c43c-569032fccc95"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Cloning into 'ExTraMapper'...\n",
            "remote: Enumerating objects: 813, done.\u001b[K\n",
            "remote: Counting objects: 100% (174/174), done.\u001b[K\n",
            "remote: Compressing objects: 100% (85/85), done.\u001b[K\n",
            "remote: Total 813 (delta 144), reused 106 (delta 89), pack-reused 639\u001b[K\n",
            "Receiving objects: 100% (813/813), 82.51 MiB | 11.38 MiB/s, done.\n",
            "Resolving deltas: 100% (444/444), done.\n"
          ]
        }
      ],
      "source": [
        "#clone forked git\n",
        "!git clone https://github.com/mattobu83/ExTraMapper.git"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "VuUgNCeRfGSL",
        "outputId": "9d57e6b1-1e6c-418b-8796-95567be688e0"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "--2023-07-18 19:58:03--  http://archive.ubuntu.com/ubuntu/pool/main/libp/libpng/libpng12-0_1.2.54-1ubuntu1_amd64.deb\n",
            "Resolving archive.ubuntu.com (archive.ubuntu.com)... 185.125.190.39, 185.125.190.36, 91.189.91.38, ...\n",
            "Connecting to archive.ubuntu.com (archive.ubuntu.com)|185.125.190.39|:80... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 116466 (114K) [application/x-debian-package]\n",
            "Saving to: ‘libpng12-0_1.2.54-1ubuntu1_amd64.deb’\n",
            "\n",
            "libpng12-0_1.2.54-1 100%[===================>] 113.74K   250KB/s    in 0.5s    \n",
            "\n",
            "2023-07-18 19:58:03 (250 KB/s) - ‘libpng12-0_1.2.54-1ubuntu1_amd64.deb’ saved [116466/116466]\n",
            "\n"
          ]
        }
      ],
      "source": [
        "#this package, libpng12, is obsolete (current version is libpng16), need to download from archive\n",
        "!wget http://archive.ubuntu.com/ubuntu/pool/main/libp/libpng/libpng12-0_1.2.54-1ubuntu1_amd64.deb"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yq9dP7d_ZIV7",
        "outputId": "00d4adae-e067-4601-93ab-a5a602d03569"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "deb http://archive.ubuntu.com/ubuntu trusty main universe multiverse\n",
            "deb http://archive.ubuntu.com/ubuntu trusty-security main universe multiverse\n"
          ]
        }
      ],
      "source": [
        "!echo \"deb http://archive.ubuntu.com/ubuntu trusty main universe multiverse\" | tee -a /etc/apt/sources.list.d/trusty.list\n",
        "!echo \"deb http://archive.ubuntu.com/ubuntu trusty-security main universe multiverse\" | tee -a /etc/apt/sources.list.d/trusty.list"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "b9-dU4i-ZJXD",
        "outputId": "dc799509-560d-4159-e74a-70a871db57cf"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Hit:1 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64  InRelease\n",
            "Get:2 http://security.ubuntu.com/ubuntu focal-security InRelease [114 kB]\n",
            "Hit:3 http://archive.ubuntu.com/ubuntu focal InRelease\n",
            "Get:4 https://cloud.r-project.org/bin/linux/ubuntu focal-cran40/ InRelease [3,622 B]\n",
            "Hit:5 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu focal InRelease\n",
            "Get:6 http://archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]\n",
            "Hit:7 http://ppa.launchpad.net/cran/libgit2/ubuntu focal InRelease\n",
            "Hit:8 http://ppa.launchpad.net/deadsnakes/ppa/ubuntu focal InRelease\n",
            "Get:9 http://archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]\n",
            "Get:10 http://security.ubuntu.com/ubuntu focal-security/universe amd64 Packages [1,073 kB]\n",
            "Hit:11 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu focal InRelease\n",
            "Ign:12 http://archive.ubuntu.com/ubuntu trusty InRelease\n",
            "Get:13 http://archive.ubuntu.com/ubuntu trusty-security InRelease [56.4 kB]\n",
            "Hit:14 http://ppa.launchpad.net/ubuntugis/ppa/ubuntu focal InRelease\n",
            "Get:15 http://archive.ubuntu.com/ubuntu trusty Release [58.5 kB]\n",
            "Get:16 http://security.ubuntu.com/ubuntu focal-security/main amd64 Packages [2,866 kB]\n",
            "Get:17 http://archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [3,347 kB]\n",
            "Get:18 http://archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1,372 kB]\n",
            "Get:19 http://archive.ubuntu.com/ubuntu trusty-security/main amd64 Packages [877 kB]\n",
            "Get:20 http://archive.ubuntu.com/ubuntu trusty-security/universe amd64 Packages [490 kB]\n",
            "Get:21 http://archive.ubuntu.com/ubuntu trusty-security/multiverse amd64 Packages [6,133 B]\n",
            "Get:22 http://archive.ubuntu.com/ubuntu trusty Release.gpg [933 B]\n",
            "Get:23 http://archive.ubuntu.com/ubuntu trusty/multiverse amd64 Packages [169 kB]\n",
            "Get:24 http://archive.ubuntu.com/ubuntu trusty/universe amd64 Packages [7,589 kB]\n",
            "Get:25 http://archive.ubuntu.com/ubuntu trusty/main amd64 Packages [1,743 kB]\n",
            "Fetched 20.0 MB in 5s (3,780 kB/s)\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "15 packages can be upgraded. Run 'apt list --upgradable' to see them.\n"
          ]
        }
      ],
      "source": [
        "!sudo apt update"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "i2avPmKOa2la"
      },
      "outputs": [],
      "source": [
        "!mkdir /lib/x86_64-linux-gnu/libpng12.so.0"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "tvibjUg9ajyg",
        "outputId": "61dd7b21-265d-4ce5-dc41-c7619bed253e"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  multiarch-support\n",
            "The following NEW packages will be installed:\n",
            "  libpng12-0 multiarch-support\n",
            "0 upgraded, 2 newly installed, 0 to remove and 15 not upgraded.\n",
            "Need to get 122 kB of archives.\n",
            "After this operation, 522 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu trusty-security/main amd64 multiarch-support amd64 2.19-0ubuntu6.15 [4,490 B]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu trusty-security/main amd64 libpng12-0 amd64 1.2.50-1ubuntu2.14.04.3 [118 kB]\n",
            "Fetched 122 kB in 1s (147 kB/s)\n",
            "Selecting previously unselected package multiarch-support.\n",
            "(Reading database ... 123105 files and directories currently installed.)\n",
            "Preparing to unpack .../multiarch-support_2.19-0ubuntu6.15_amd64.deb ...\n",
            "Unpacking multiarch-support (2.19-0ubuntu6.15) ...\n",
            "Setting up multiarch-support (2.19-0ubuntu6.15) ...\n",
            "Selecting previously unselected package libpng12-0:amd64.\n",
            "(Reading database ... 123108 files and directories currently installed.)\n",
            "Preparing to unpack .../libpng12-0_1.2.50-1ubuntu2.14.04.3_amd64.deb ...\n",
            "Unpacking libpng12-0:amd64 (1.2.50-1ubuntu2.14.04.3) ...\n",
            "Setting up libpng12-0:amd64 (1.2.50-1ubuntu2.14.04.3) ...\n",
            "Processing triggers for libc-bin (2.31-0ubuntu9.9) ...\n",
            "/sbin/ldconfig.real: /lib/x86_64-linux-gnu/libpng12.so.0 is not a symbolic link\n",
            "\n"
          ]
        }
      ],
      "source": [
        "!apt-get install libpng12-0"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "lCTSiKKTbRYn"
      },
      "outputs": [],
      "source": [
        "!rm -rf /lib/x86_64-linux-gnu/libpng12.so.0"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "iMhLnFUdl8q1",
        "outputId": "4c8a833f-2992-48a2-cd24-a1661cde18da"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "(Reading database ... \r(Reading database ... 5%\r(Reading database ... 10%\r(Reading database ... 15%\r(Reading database ... 20%\r(Reading database ... 25%\r(Reading database ... 30%\r(Reading database ... 35%\r(Reading database ... 40%\r(Reading database ... 45%\r(Reading database ... 50%\r(Reading database ... 55%\r(Reading database ... 60%\r(Reading database ... 65%\r(Reading database ... 70%\r(Reading database ... 75%\r(Reading database ... 80%\r(Reading database ... 85%\r(Reading database ... 90%\r(Reading database ... 95%\r(Reading database ... 100%\r(Reading database ... 123121 files and directories currently installed.)\n",
            "Preparing to unpack libpng12-0_1.2.54-1ubuntu1_amd64.deb ...\n",
            "Unpacking libpng12-0:amd64 (1.2.54-1ubuntu1) over (1.2.50-1ubuntu2.14.04.3) ...\n",
            "\u001b[1mdpkg:\u001b[0m error processing archive libpng12-0_1.2.54-1ubuntu1_amd64.deb (--install):\n",
            " unable to install new version of '/lib/x86_64-linux-gnu/libpng12.so.0': No such file or directory\n",
            "Processing triggers for libc-bin (2.31-0ubuntu9.9) ...\n",
            "Errors were encountered while processing:\n",
            " libpng12-0_1.2.54-1ubuntu1_amd64.deb\n"
          ]
        }
      ],
      "source": [
        "#libpng package install\n",
        "!dpkg -i libpng12-0_1.2.54-1ubuntu1_amd64.deb"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "0hFTsZHC-21H",
        "outputId": "18b68dbf-d4c5-4677-bc37-fdadccf427e3"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "-rw-r--r--  1 root root  151K Jul 10  2018 libpng12.so.0.50.0\n",
            "lrwxrwxrwx  1 root root    11 Feb  7  2020 libpng.so -> libpng16.so\n",
            "lrwxrwxrwx  1 root root    10 Feb  7  2020 libpng.a -> libpng16.a\n",
            "-rw-r--r--  1 root root  219K Feb  7  2020 libpng16.so.16.37.0\n",
            "lrwxrwxrwx  1 root root    19 Feb  7  2020 libpng16.so.16 -> libpng16.so.16.37.0\n",
            "lrwxrwxrwx  1 root root    19 Feb  7  2020 libpng16.so -> libpng16.so.16.37.0\n",
            "-rw-r--r--  1 root root  351K Feb  7  2020 libpng16.a\n",
            "lrwxrwxrwx  1 root root    18 Jul 18 19:58 libpng12.so.0 -> libpng12.so.0.50.0\n"
          ]
        }
      ],
      "source": [
        "#check there exists libpng12.so.0 -> libpng12.so.0.54.0\n",
        "!ls -hltr /usr/lib/x86_64-linux-gnu | grep libpng"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "Wf_rL62tYAAy"
      },
      "outputs": [],
      "source": [
        "#make extMpreprocess is executable\n",
        "!chmod +x /content/ExTraMapper/Human-Mouse-Preprocess-Data/extMpreprocess"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "gAIqV5vGYL3R",
        "outputId": "9b817f45-48f1-464c-8faf-9d11ac5499d1"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "/content/ExTraMapper/Human-Mouse-Preprocess-Data\n"
          ]
        }
      ],
      "source": [
        "%cd /content/ExTraMapper/Human-Mouse-Preprocess-Data"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "1oRNiB90YOQU",
        "outputId": "29252e9e-a896-4cf0-87d6-b6facbc50654"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "/content/ExTraMapper/Human-Mouse-Preprocess-Data\n"
          ]
        }
      ],
      "source": [
        "!pwd"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "lEsgTqF0ZdFV",
        "outputId": "15557729-af0e-43e5-bcc1-f4fd6ec127af"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Collecting genomedata\n",
            "  Downloading genomedata-1.6.0-cp37-abi3-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.2 MB)\n",
            "\u001b[?25l     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m0.0/1.2 MB\u001b[0m \u001b[31m?\u001b[0m eta \u001b[36m-:--:--\u001b[0m\r\u001b[2K     \u001b[91m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m\u001b[90m╺\u001b[0m\u001b[90m━━━━━━\u001b[0m \u001b[32m1.0/1.2 MB\u001b[0m \u001b[31m30.3 MB/s\u001b[0m eta \u001b[36m0:00:01\u001b[0m\r\u001b[2K     \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m1.2/1.2 MB\u001b[0m \u001b[31m24.2 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
            "\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.10/dist-packages (from genomedata) (1.22.4)\n",
            "Requirement already satisfied: tables!=3.4.1,>=3.0 in /usr/local/lib/python3.10/dist-packages (from genomedata) (3.8.0)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.10/dist-packages (from genomedata) (1.16.0)\n",
            "Collecting textinput>=0.2.0 (from genomedata)\n",
            "  Downloading textinput-0.2.0.tar.gz (6.4 kB)\n",
            "  Preparing metadata (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "Collecting path.py>=11 (from genomedata)\n",
            "  Downloading path.py-12.5.0-py3-none-any.whl (2.3 kB)\n",
            "Collecting path (from path.py>=11->genomedata)\n",
            "  Downloading path-16.7.1-py3-none-any.whl (25 kB)\n",
            "Requirement already satisfied: cython>=0.29.21 in /usr/local/lib/python3.10/dist-packages (from tables!=3.4.1,>=3.0->genomedata) (0.29.36)\n",
            "Requirement already satisfied: numexpr>=2.6.2 in /usr/local/lib/python3.10/dist-packages (from tables!=3.4.1,>=3.0->genomedata) (2.8.4)\n",
            "Requirement already satisfied: blosc2~=2.0.0 in /usr/local/lib/python3.10/dist-packages (from tables!=3.4.1,>=3.0->genomedata) (2.0.0)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.10/dist-packages (from tables!=3.4.1,>=3.0->genomedata) (23.1)\n",
            "Requirement already satisfied: py-cpuinfo in /usr/local/lib/python3.10/dist-packages (from tables!=3.4.1,>=3.0->genomedata) (9.0.0)\n",
            "Requirement already satisfied: msgpack in /usr/local/lib/python3.10/dist-packages (from blosc2~=2.0.0->tables!=3.4.1,>=3.0->genomedata) (1.0.5)\n",
            "Building wheels for collected packages: textinput\n",
            "  Building wheel for textinput (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
            "  Created wheel for textinput: filename=textinput-0.2.0-py3-none-any.whl size=10125 sha256=10407ff2863082545c6ce0781a47a5bc3e693ba4d9a0dc3e223cf2084ca618aa\n",
            "  Stored in directory: /root/.cache/pip/wheels/b8/49/d5/1822614972041244e2b1210728c4f9b5cb76bc7ed607ea102b\n",
            "Successfully built textinput\n",
            "Installing collected packages: textinput, path, path.py, genomedata\n",
            "Successfully installed genomedata-1.6.0 path-16.7.1 path.py-12.5.0 textinput-0.2.0\n"
          ]
        }
      ],
      "source": [
        "!pip install genomedata"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "fUaAS5otfEwB"
      },
      "outputs": [],
      "source": [
        "#make both liftover scripts executable, make sure they are when listing dir\n",
        "!chmod +x /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "_Cs2vZWPe-RM"
      },
      "outputs": [],
      "source": [
        "!chmod +x /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftover-withMultiples"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "dHxSEWxoe8aL",
        "outputId": "9ba8ef95-98f9-43b0-ed76-6eaf38a154ba"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "total 15M\n",
            "-rw-r--r-- 1 root root  49K Jul 18 19:58 ensemblUtils.py\n",
            "-rw-r--r-- 1 root root  507 Jul 18 19:58 splitExonsIntoIndividualFiles.py\n",
            "-rw-r--r-- 1 root root 4.9K Jul 18 19:58 parseAndPicklePerPair.py\n",
            "-rwxr-xr-x 1 root root  11K Jul 18 19:58 liftover-withMultiples\n",
            "-rwxr-xr-x 1 root root  15M Jul 18 19:58 liftOver\n"
          ]
        }
      ],
      "source": [
        "!ls -hltr /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "SZB24BOaIa45",
        "outputId": "989c3390-d37c-453a-aef3-c796a113ac75"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "The following NEW packages will be installed:\n",
            "  bedtools\n",
            "0 upgraded, 1 newly installed, 0 to remove and 15 not upgraded.\n",
            "Need to get 580 kB of archives.\n",
            "After this operation, 2,055 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu focal/universe amd64 bedtools amd64 2.27.1+dfsg-4ubuntu1 [580 kB]\n",
            "Fetched 580 kB in 1s (1,025 kB/s)\n",
            "Selecting previously unselected package bedtools.\n",
            "(Reading database ... 123121 files and directories currently installed.)\n",
            "Preparing to unpack .../bedtools_2.27.1+dfsg-4ubuntu1_amd64.deb ...\n",
            "Unpacking bedtools (2.27.1+dfsg-4ubuntu1) ...\n",
            "Setting up bedtools (2.27.1+dfsg-4ubuntu1) ...\n"
          ]
        }
      ],
      "source": [
        "#install necessary packages\n",
        "!apt-get install bedtools"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "background_save": true,
          "base_uri": "https://localhost:8080/"
        },
        "id": "OChIVd7eXqIT",
        "outputId": "3612947b-d318-4ac3-b0e9-2e6be121f45c"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "Running step 1:\n",
            "Downloading per organism specific files and keep the original organism names for future reuse\n",
            "Downloading the two reference genomes from UCSC and get rid of unknown, random and alt contigs\n",
            "Creating /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/reference_genomes/hg38 folder\n",
            "Running: wget --timestamping ftp://hgdownload.cse.ucsc.edu/goldenPath/hg38/chromosomes/* --directory-prefix=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/reference_genomes/hg38 2>&1 | grep \"Login incorrect\"\n",
            "Creating /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/reference_genomes/mm10 folder\n",
            "Running: wget --timestamping ftp://hgdownload.cse.ucsc.edu/goldenPath/mm10/chromosomes/* --directory-prefix=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/reference_genomes/mm10 2>&1 | grep \"Login incorrect\"\n",
            "Creating /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver folder\n",
            "Running: wget http://hgdownload.cse.ucsc.edu/goldenPath/hg38/liftOver/hg38ToMm10.over.chain.gz --directory-prefix=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver\n",
            "LiftOver chain saved to /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz\n",
            "Creating /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver folder\n",
            "Running: wget http://hgdownload.cse.ucsc.edu/goldenPath/mm10/liftOver/mm10ToHg38.over.chain.gz --directory-prefix=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver\n",
            "LiftOver chain saved to /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz\n",
            "Downloading GTF files\n",
            "wget ftp://ftp.ensembl.org/pub/release-109/gtf/homo_sapiens/*.109.gtf.gz -O /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1.gtf.gz\n",
            "GTF files downloaded in /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "wget ftp://ftp.ensembl.org/pub/release-109/gtf/mus_musculus/*.109.gtf.gz -O /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2.gtf.gz\n",
            "GTF files downloaded in /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "Downloading ENSEMBL homologs\n",
            "wget ftp://ftp.ensembl.org/pub/release-109/mysql/ensembl_mart_109/hsapiens_gene_ensembl__homolog_mmusculus__dm.txt.gz -O /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1_homolog_org2.txt.gz\n",
            "ENSEMBL homolog downloaded in /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "wget ftp://ftp.ensembl.org/pub/release-109/mysql/ensembl_mart_109/mmusculus_gene_ensembl__homolog_hsapiens__dm.txt.gz -O /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2_homolog_org1.txt.gz\n",
            "ENSEMBL homolog downloaded in /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "---------------------- Step 1 Finished ----------------------\n",
            "Running step 2:\n",
            "Initialize the genomedata archives with the genomes of org1 and org2\n",
            "Make sure genomedata is installed first\n",
            "Installation: pip install genomedata --user\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/reference_genomes/hg38/*.fa.gz > hg38.fa\n",
            "Started at Wed Jul 19 11:59:41 2023\n",
            "Ended at Wed Jul 19 12:00:16 2023\n",
            "Running : genomedata-load-seq -d org1 hg38.fa\n",
            "Started at Wed Jul 19 12:00:16 2023\n",
            "Ended at Wed Jul 19 12:59:45 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/reference_genomes/mm10/*.fa.gz > mm10.fa\n",
            "Started at Wed Jul 19 12:59:45 2023\n",
            "Ended at Wed Jul 19 13:00:20 2023\n",
            "Running : genomedata-load-seq -d org2 mm10.fa\n",
            "Started at Wed Jul 19 13:00:20 2023\n",
            "Ended at Wed Jul 19 15:06:21 2023\n",
            "---------------------- Step 2 Finished ----------------------\n",
            "Running step 3:\n",
            "Creating pickle files\n",
            "Running : gunzip -k /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1.gtf.gz\n",
            "Running : gunzip -k /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2.gtf.gz\n",
            "Running : gunzip -k /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1_homolog_org2.txt.gz\n",
            "Running : gunzip -k /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2_homolog_org1.txt.gz\n",
            "Running : python /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/parseAndPicklePerPair.py /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/GTFsummaries /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perGenePairPickledInfo\n",
            "Started at Wed Jul 19 15:06:45 2023\n",
            "Parsing organism GTF for org1 from file /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1.gtf\n",
            "100000\t200000\t300000\t400000\t500000\t600000\t700000\t800000\t900000\t1000000\t1100000\t1200000\t1300000\t1400000\t1500000\t1600000\t1700000\t1800000\t1900000\t2000000\t2100000\t2200000\t2300000\t2400000\t2500000\t2600000\t2700000\t2800000\t2900000\t3000000\t3100000\t3200000\t3300000\t3400000\t\n",
            "GTF parsing summary: {'CDS': 885469, 'exon': 1648040, 'gene': 62619, 'start_codon': 97942, 'stop_codon': 91888, 'transcript': 252702, 'UTR': 0, 'other': 380981}\n",
            "\n",
            "Parsing organism GTF for org2 from file /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2.gtf\n",
            "100000\t200000\t300000\t400000\t500000\t600000\t700000\t800000\t900000\t1000000\t1100000\t1200000\t1300000\t1400000\t1500000\t1600000\t1700000\t1800000\t1900000\t\n",
            "GTF parsing summary: {'CDS': 527422, 'exon': 868700, 'gene': 56916, 'start_codon': 59792, 'stop_codon': 55584, 'transcript': 149347, 'UTR': 0, 'other': 182788}\n",
            "\n",
            "Parsing gene and protein pairings from file /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1_homolog_org2.txt\n",
            "Gene pair mappings summary: {}\n",
            "\n",
            "['1', 20389, 0, 20389]\n",
            "Parsing gene and protein pairings from file /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2_homolog_org1.txt\n",
            "Gene pair mappings summary: {'ortholog_many2many': 635, 'ortholog_one2many': 1467, 'ortholog_one2one': 17114}\n",
            "\n",
            "['2', 39606, 19216, 39606]\n",
            "[62619, 56916]\n",
            "16275\n",
            "16275\n",
            "/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/GTFsummaries/onlyOrthologAndCodingGenes\n",
            "Ended at Wed Jul 19 15:10:50 2023\n",
            "---------------------- Step 3 Finished ----------------------\n",
            "Running step 4:\n",
            "liftOver the exon lists but this time allow multiple mappings and also compute intersections with the other set of exons\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/GTFsummaries/onlyOrthologAndCodingGenes/org1-allExons-GTFparsed.txt | awk -v OFS='\\t' 'NR>1{print $1,$2,$3,$4,$5}' | sort -k1,1 -k2,2n > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed\n",
            "Started at Wed Jul 19 15:10:50 2023\n",
            "Ended at Wed Jul 19 15:10:52 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/GTFsummaries/onlyOrthologAndCodingGenes/org2-allExons-GTFparsed.txt | awk -v OFS='\\t' 'NR>1{print $1,$2,$3,$4,$5}' | sort -k1,1 -k2,2n > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed\n",
            "Started at Wed Jul 19 15:10:52 2023\n",
            "Ended at Wed Jul 19 15:10:53 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/GTFsummaries/onlyOrthologAndCodingGenes/org1-allExons-GTFparsed.txt |awk -v OFS='\\t' '$6==\"partCoding\" {print $1,$7,$8,$4,$5}' | sort -k1,1 -k2,2n > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.bed\n",
            "Started at Wed Jul 19 15:10:53 2023\n",
            "Ended at Wed Jul 19 15:10:54 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/GTFsummaries/onlyOrthologAndCodingGenes/org2-allExons-GTFparsed.txt |awk -v OFS='\\t' '$6==\"partCoding\" {print $1,$7,$8,$4,$5}' | sort -k1,1 -k2,2n > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.bed\n",
            "Started at Wed Jul 19 15:10:54 2023\n",
            "Ended at Wed Jul 19 15:10:55 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/GTFsummaries/onlyOrthologAndCodingGenes/org1-allExons-GTFparsed.txt |awk -v OFS='\\t' '$6==\"fullCoding\" {print $1,$2,$3,$4,$5}' > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_f.temp\n",
            "Started at Wed Jul 19 15:10:55 2023\n",
            "Ended at Wed Jul 19 15:10:56 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/GTFsummaries/onlyOrthologAndCodingGenes/org2-allExons-GTFparsed.txt |awk -v OFS='\\t' '$6==\"fullCoding\" {print $1,$2,$3,$4,$5}' > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_f.temp\n",
            "Started at Wed Jul 19 15:10:56 2023\n",
            "Ended at Wed Jul 19 15:10:56 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_f.temp | sort -k1,1 -k2,2n > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allCodingExonsList.bed\n",
            "Started at Wed Jul 19 15:10:56 2023\n",
            "Ended at Wed Jul 19 15:10:57 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_f.temp | sort -k1,1 -k2,2n > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allCodingExonsList.bed\n",
            "Started at Wed Jul 19 15:10:57 2023\n",
            "Ended at Wed Jul 19 15:10:57 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allCodingExonsList.bed |awk '{print $5,$0}' | sort -k1,1 > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allCodingExonsList.sorted.temp\n",
            "Started at Wed Jul 19 15:10:57 2023\n",
            "Ended at Wed Jul 19 15:10:59 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allCodingExonsList.bed |awk '{print $5,$0}' | sort -k1,1 > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allCodingExonsList.sorted.temp\n",
            "Started at Wed Jul 19 15:10:59 2023\n",
            "Ended at Wed Jul 19 15:11:02 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed |awk '{print $5,$0}' | sort -k1,1 > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.sorted.temp\n",
            "Started at Wed Jul 19 15:11:02 2023\n",
            "Ended at Wed Jul 19 15:11:06 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed |awk '{print $5,$0}' | sort -k1,1 > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.sorted.temp\n",
            "Started at Wed Jul 19 15:11:06 2023\n",
            "Ended at Wed Jul 19 15:11:09 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.bed |awk '{print $5,$0}' | sort -k1,1 > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.sorted.temp\n",
            "Started at Wed Jul 19 15:11:09 2023\n",
            "Ended at Wed Jul 19 15:11:10 2023\n",
            "Running : cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.bed |awk '{print $5,$0}' | sort -k1,1 > /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.sorted.temp\n",
            "Started at Wed Jul 19 15:11:10 2023\n",
            "Ended at Wed Jul 19 15:11:11 2023\n",
            "Running : /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftover-withMultiples 0 1 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz\n",
            "Started at Wed Jul 19 15:11:11 2023\n",
            "+ set -o pipefail\n",
            "+ set -o errexit\n",
            "+ source config.human-mouse.conf\n",
            "++ ref1=hg38\n",
            "++ ref2=mm10\n",
            "++ org1=human\n",
            "++ org2=mouse\n",
            "++ releaseNo=109\n",
            "++ org1EnsemblName=homo_sapiens\n",
            "++ org2EnsemblName=mus_musculus\n",
            "++ org1EnsemblMartName=hsapiens\n",
            "++ org2EnsemblMartName=mmusculus\n",
            "++ org1EnsemblMartNameShort=hsap\n",
            "++ org2EnsemblMartNameShort=mmus\n",
            "++ liftOver=/content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver\n",
            "+ dataDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data\n",
            "+ dataDirPerPair=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse\n",
            "+ chainsDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains\n",
            "+ ensemblDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "+ liftOverFilesDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles\n",
            "+ perExonLiftoverDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perExonLiftoverCoords\n",
            "+ outdir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles\n",
            "+ flank=0\n",
            "+ minMatch=1\n",
            "+ chain1to2=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz\n",
            "+ chain2to1=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz\n",
            "+ mkdir -p /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "+ GTFfile1=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1.gtf.gz\n",
            "+ GTFfile2=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2.gtf.gz\n",
            "+ org1to2homologFile=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1_homolog_org2.txt.gz\n",
            "+ org2to1homologFile=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2_homolog_org1.txt.gz\n",
            "+ refGDdir1=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1\n",
            "+ refGDdir2=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2\n",
            "+ suffix=flank0-minMatch1-multiples-partCoding\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz org2_mapped-flank0-minMatch1-multiples-partCoding.bed org2_unmapped-flank0-minMatch1-multiples-partCoding.bed -minMatch=1 -multiple\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.bed\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz org1_mapped-flank0-minMatch1-multiples-partCoding.bed org1_unmapped-flank0-minMatch1-multiples-partCoding.bed -minMatch=1 -multiple\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.bed\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org1_mapped-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org2_mapped-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ cat org1_unmapped-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ cat org2_unmapped-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ rm -rf org1_mapped-flank0-minMatch1-multiples-partCoding.bed org2_mapped-flank0-minMatch1-multiples-partCoding.bed org1_unmapped-flank0-minMatch1-multiples-partCoding.bed org2_unmapped-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.sorted.temp mapped.temp\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allCodingExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allCodingExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.temp -sorted -v\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.sorted.temp mapped.temp\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allCodingExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allCodingExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.temp -sorted -v\n",
            "+ rm -rf /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.temp /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.temp mapped.temp\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch1-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch1-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_nonintersectingExonsList-flank0-minMatch1-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_nonintersectingExonsList-flank0-minMatch1-multiples-partCoding.bed\n",
            "+ suffix=flank0-minMatch1-multiples\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz org2_mapped-flank0-minMatch1-multiples.bed org2_unmapped-flank0-minMatch1-multiples.bed -minMatch=1 -multiple\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz org1_mapped-flank0-minMatch1-multiples.bed org1_unmapped-flank0-minMatch1-multiples.bed -minMatch=1 -multiple\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org1_mapped-flank0-minMatch1-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org2_mapped-flank0-minMatch1-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ cat org1_unmapped-flank0-minMatch1-multiples.bed\n",
            "+ cat org2_unmapped-flank0-minMatch1-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ rm -rf org1_mapped-flank0-minMatch1-multiples.bed org2_mapped-flank0-minMatch1-multiples.bed org1_unmapped-flank0-minMatch1-multiples.bed org2_unmapped-flank0-minMatch1-multiples.bed\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.sorted.temp mapped.temp\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples.temp -sorted -v\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.sorted.temp mapped.temp\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples.temp -sorted -v\n",
            "+ rm -rf /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples.temp /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples.temp mapped.temp\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch1-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch1-multiples.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch1-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch1-multiples.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch1-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch1-multiples.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_nonintersectingExonsList-flank0-minMatch1-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_nonintersectingExonsList-flank0-minMatch1-multiples.bed\n",
            "Ended at Wed Jul 19 15:49:08 2023\n",
            "Running : /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftover-withMultiples 0 0.95 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz\n",
            "Started at Wed Jul 19 15:49:08 2023\n",
            "+ set -o pipefail\n",
            "+ set -o errexit\n",
            "+ source config.human-mouse.conf\n",
            "++ ref1=hg38\n",
            "++ ref2=mm10\n",
            "++ org1=human\n",
            "++ org2=mouse\n",
            "++ releaseNo=109\n",
            "++ org1EnsemblName=homo_sapiens\n",
            "++ org2EnsemblName=mus_musculus\n",
            "++ org1EnsemblMartName=hsapiens\n",
            "++ org2EnsemblMartName=mmusculus\n",
            "++ org1EnsemblMartNameShort=hsap\n",
            "++ org2EnsemblMartNameShort=mmus\n",
            "++ liftOver=/content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver\n",
            "+ dataDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data\n",
            "+ dataDirPerPair=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse\n",
            "+ chainsDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains\n",
            "+ ensemblDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "+ liftOverFilesDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles\n",
            "+ perExonLiftoverDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perExonLiftoverCoords\n",
            "+ outdir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles\n",
            "+ flank=0\n",
            "+ minMatch=0.95\n",
            "+ chain1to2=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz\n",
            "+ chain2to1=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz\n",
            "+ mkdir -p /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "+ GTFfile1=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1.gtf.gz\n",
            "+ GTFfile2=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2.gtf.gz\n",
            "+ org1to2homologFile=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1_homolog_org2.txt.gz\n",
            "+ org2to1homologFile=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2_homolog_org1.txt.gz\n",
            "+ refGDdir1=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1\n",
            "+ refGDdir2=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2\n",
            "+ suffix=flank0-minMatch0.95-multiples-partCoding\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz org2_mapped-flank0-minMatch0.95-multiples-partCoding.bed org2_unmapped-flank0-minMatch0.95-multiples-partCoding.bed -minMatch=0.95 -multiple\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.bed\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz org1_mapped-flank0-minMatch0.95-multiples-partCoding.bed org1_unmapped-flank0-minMatch0.95-multiples-partCoding.bed -minMatch=0.95 -multiple\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.bed\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ cat org1_mapped-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org2_mapped-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org1_unmapped-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ cat org2_unmapped-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ rm -rf org1_mapped-flank0-minMatch0.95-multiples-partCoding.bed org2_mapped-flank0-minMatch0.95-multiples-partCoding.bed org1_unmapped-flank0-minMatch0.95-multiples-partCoding.bed org2_unmapped-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.sorted.temp mapped.temp\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allCodingExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allCodingExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.temp -sorted -v\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.sorted.temp mapped.temp\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allCodingExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allCodingExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.temp -sorted -v\n",
            "+ rm -rf /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.temp /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.temp mapped.temp\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch0.95-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch0.95-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_nonintersectingExonsList-flank0-minMatch0.95-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_nonintersectingExonsList-flank0-minMatch0.95-multiples-partCoding.bed\n",
            "+ suffix=flank0-minMatch0.95-multiples\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz org2_mapped-flank0-minMatch0.95-multiples.bed org2_unmapped-flank0-minMatch0.95-multiples.bed -minMatch=0.95 -multiple\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz org1_mapped-flank0-minMatch0.95-multiples.bed org1_unmapped-flank0-minMatch0.95-multiples.bed -minMatch=0.95 -multiple\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ cat org1_mapped-flank0-minMatch0.95-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org2_mapped-flank0-minMatch0.95-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org1_unmapped-flank0-minMatch0.95-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ cat org2_unmapped-flank0-minMatch0.95-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ rm -rf org1_mapped-flank0-minMatch0.95-multiples.bed org2_mapped-flank0-minMatch0.95-multiples.bed org1_unmapped-flank0-minMatch0.95-multiples.bed org2_unmapped-flank0-minMatch0.95-multiples.bed\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.sorted.temp mapped.temp\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.temp -sorted -v\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.sorted.temp mapped.temp\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.temp -sorted -v\n",
            "+ rm -rf /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.temp /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.temp mapped.temp\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.95-multiples.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch0.95-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch0.95-multiples.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch0.95-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch0.95-multiples.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_nonintersectingExonsList-flank0-minMatch0.95-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_nonintersectingExonsList-flank0-minMatch0.95-multiples.bed\n",
            "Ended at Wed Jul 19 16:30:32 2023\n",
            "Running : /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftover-withMultiples 0 0.9 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz\n",
            "Started at Wed Jul 19 16:30:32 2023\n",
            "+ set -o pipefail\n",
            "+ set -o errexit\n",
            "+ source config.human-mouse.conf\n",
            "++ ref1=hg38\n",
            "++ ref2=mm10\n",
            "++ org1=human\n",
            "++ org2=mouse\n",
            "++ releaseNo=109\n",
            "++ org1EnsemblName=homo_sapiens\n",
            "++ org2EnsemblName=mus_musculus\n",
            "++ org1EnsemblMartName=hsapiens\n",
            "++ org2EnsemblMartName=mmusculus\n",
            "++ org1EnsemblMartNameShort=hsap\n",
            "++ org2EnsemblMartNameShort=mmus\n",
            "++ liftOver=/content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver\n",
            "+ dataDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data\n",
            "+ dataDirPerPair=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse\n",
            "+ chainsDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains\n",
            "+ ensemblDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "+ liftOverFilesDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles\n",
            "+ perExonLiftoverDir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perExonLiftoverCoords\n",
            "+ outdir=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles\n",
            "+ flank=0\n",
            "+ minMatch=0.9\n",
            "+ chain1to2=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz\n",
            "+ chain2to1=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz\n",
            "+ mkdir -p /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads\n",
            "+ GTFfile1=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1.gtf.gz\n",
            "+ GTFfile2=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2.gtf.gz\n",
            "+ org1to2homologFile=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1_homolog_org2.txt.gz\n",
            "+ org2to1homologFile=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2_homolog_org1.txt.gz\n",
            "+ refGDdir1=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org1\n",
            "+ refGDdir2=/content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/ensemblDownloads/org2\n",
            "+ suffix=flank0-minMatch0.9-multiples-partCoding\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz org2_mapped-flank0-minMatch0.9-multiples-partCoding.bed org2_unmapped-flank0-minMatch0.9-multiples-partCoding.bed -minMatch=0.9 -multiple\n",
            "Reading liftover chains\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.bed\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "Mapping coordinates\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz org1_mapped-flank0-minMatch0.9-multiples-partCoding.bed org1_unmapped-flank0-minMatch0.9-multiples-partCoding.bed -minMatch=0.9 -multiple\n",
            "Reading liftover chains\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.bed\n",
            "Mapping coordinates\n",
            "+ cat org1_mapped-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org2_mapped-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org1_unmapped-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ cat org2_unmapped-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ rm -rf org1_mapped-flank0-minMatch0.9-multiples-partCoding.bed org2_mapped-flank0-minMatch0.9-multiples-partCoding.bed org1_unmapped-flank0-minMatch0.9-multiples-partCoding.bed org2_unmapped-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_partCodingExonsList.sorted.temp mapped.temp\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allCodingExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allCodingExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.temp -sorted -v\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_partCodingExonsList.sorted.temp mapped.temp\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allCodingExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allCodingExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.temp -sorted -v\n",
            "+ rm -rf /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.temp /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.temp mapped.temp\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch0.9-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch0.9-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_nonintersectingExonsList-flank0-minMatch0.9-multiples-partCoding.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_nonintersectingExonsList-flank0-minMatch0.9-multiples-partCoding.bed\n",
            "+ suffix=flank0-minMatch0.9-multiples\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/hg38/liftOver/hg38ToMm10.over.chain.gz org2_mapped-flank0-minMatch0.9-multiples.bed org2_unmapped-flank0-minMatch0.9-multiples.bed -minMatch=0.9 -multiple\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/liftOver /dev/fd/63 /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/liftover_chains/mm10/liftOver/mm10ToHg38.over.chain.gz org1_mapped-flank0-minMatch0.9-multiples.bed org1_unmapped-flank0-minMatch0.9-multiples.bed -minMatch=0.9 -multiple\n",
            "++ awk '{print $1,$2-s,$3+s,$5,$4,$4}' s=0\n",
            "++ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed\n",
            "Reading liftover chains\n",
            "Mapping coordinates\n",
            "+ cat org1_mapped-flank0-minMatch0.9-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ cat org2_mapped-flank0-minMatch0.9-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ cat org1_unmapped-flank0-minMatch0.9-multiples.bed\n",
            "+ cat org2_unmapped-flank0-minMatch0.9-multiples.bed\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{l1=$1; getline; printf(\"%s\\t%s\\t%s\\t%s\\t%s\\t%s\\n\",$1,$2,$3,$4,l1,$5)}'\n",
            "+ rm -rf org1_mapped-flank0-minMatch0.9-multiples.bed org2_mapped-flank0-minMatch0.9-multiples.bed org1_unmapped-flank0-minMatch0.9-multiples.bed org2_unmapped-flank0-minMatch0.9-multiples.bed\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.sorted.temp mapped.temp\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.temp -sorted -v\n",
            "+ cat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.bed\n",
            "+ sort -k1,1\n",
            "+ awk '{print $4,$0}'\n",
            "+ sort -k1,1 -k2,2n\n",
            "+ join /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_allExonsList.sorted.temp mapped.temp\n",
            "+ awk '{s=$8; e=$9; if ($5!=$12) {s=s+1; e=e+1;}; print $7\"\\t\"s\"\\t\"e\"\\t\"$10\"\\t\"$11\"\\t\"$12}'\n",
            "+ bedtools intersect -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.temp -sorted -wao\n",
            "+ bedtools intersect -b /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_allExonsList.bed -a /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.temp -sorted -v\n",
            "+ rm -rf /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.temp /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.temp mapped.temp\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_mappedExonsList-flank0-minMatch0.9-multiples.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch0.9-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch0.9-multiples.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch0.9-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch0.9-multiples.bed\n",
            "+ gzip /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_nonintersectingExonsList-flank0-minMatch0.9-multiples.bed /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_nonintersectingExonsList-flank0-minMatch0.9-multiples.bed\n",
            "Ended at Wed Jul 19 17:12:48 2023\n",
            "---------------------- Step 4 Finished ----------------------\n",
            "Running step 5:\n",
            "Putting together, sorting, making them uniq and then splitting into one file per exon for all the liftover files created so far\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch1-multiples-partCoding.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch1-multiples-partCoding >> oneHugeFile-2to1-partCoding.txt\n",
            "Started at Wed Jul 19 17:12:48 2023\n",
            "Ended at Wed Jul 19 17:12:49 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch1-multiples-partCoding.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch1-multiples-partCoding >> oneHugeFile-1to2-partCoding.txt\n",
            "Started at Wed Jul 19 17:12:49 2023\n",
            "Started at Wed Jul 19 17:12:49 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch0.95-multiples-partCoding.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch0.95-multiples-partCoding >> oneHugeFile-2to1-partCoding.txt\n",
            "Started at Wed Jul 19 17:12:49 2023\n",
            "Ended at Wed Jul 19 17:12:49 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch0.95-multiples-partCoding.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch0.95-multiples-partCoding >> oneHugeFile-1to2-partCoding.txt\n",
            "Started at Wed Jul 19 17:12:49 2023\n",
            "Started at Wed Jul 19 17:12:49 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch0.9-multiples-partCoding.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch0.9-multiples-partCoding >> oneHugeFile-2to1-partCoding.txt\n",
            "Started at Wed Jul 19 17:12:49 2023\n",
            "Ended at Wed Jul 19 17:12:50 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch0.9-multiples-partCoding.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch0.9-multiples-partCoding >> oneHugeFile-1to2-partCoding.txt\n",
            "Started at Wed Jul 19 17:12:50 2023\n",
            "Started at Wed Jul 19 17:12:50 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch1-multiples.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch1-multiples >> oneHugeFile-2to1-others.txt\n",
            "Started at Wed Jul 19 17:12:50 2023\n",
            "Ended at Wed Jul 19 17:12:50 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch1-multiples.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch1-multiples >> oneHugeFile-1to2-others.txt\n",
            "Started at Wed Jul 19 17:12:50 2023\n",
            "Ended at Wed Jul 19 17:12:51 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch0.95-multiples.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch0.95-multiples >> oneHugeFile-2to1-others.txt\n",
            "Started at Wed Jul 19 17:12:51 2023\n",
            "Ended at Wed Jul 19 17:12:51 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch0.95-multiples.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch0.95-multiples >> oneHugeFile-1to2-others.txt\n",
            "Started at Wed Jul 19 17:12:51 2023\n",
            "Ended at Wed Jul 19 17:12:52 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_intersectingExonsList-flank0-minMatch0.9-multiples.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch0.9-multiples >> oneHugeFile-2to1-others.txt\n",
            "Started at Wed Jul 19 17:12:52 2023\n",
            "Ended at Wed Jul 19 17:12:53 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_intersectingExonsList-flank0-minMatch0.9-multiples.bed.gz |awk -v OFS='\\t' '$6!=\".\"{print $1,$2,$3,$4,$5,$6,$7,$8,$11,$9,$12,s}' s=flank0-minMatch0.9-multiples >> oneHugeFile-1to2-others.txt\n",
            "Started at Wed Jul 19 17:12:53 2023\n",
            "Ended at Wed Jul 19 17:12:53 2023\n",
            "Running : cat oneHugeFile-1to2-partCoding.txt oneHugeFile-1to2-others.txt | sort -k10,10 >oneHugeFile-1to2.txt.sorted\n",
            "Started at Wed Jul 19 17:12:53 2023\n",
            "Ended at Wed Jul 19 17:12:57 2023\n",
            "Running : cat oneHugeFile-2to1-partCoding.txt oneHugeFile-2to1-others.txt | sort -k10,10 >oneHugeFile-2to1.txt.sorted\n",
            "Started at Wed Jul 19 17:12:57 2023\n",
            "Ended at Wed Jul 19 17:13:00 2023\n",
            "Running : python /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/splitExonsIntoIndividualFiles.py oneHugeFile-1to2.txt.sorted /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perExonLiftoverCoords/org1 10 _mapped.txt\n",
            "Started at Wed Jul 19 17:13:00 2023\n",
            "Ended at Wed Jul 19 17:13:05 2023\n",
            "Running : python /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/splitExonsIntoIndividualFiles.py oneHugeFile-2to1.txt.sorted /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perExonLiftoverCoords/org2 10 _mapped.txt\n",
            "Started at Wed Jul 19 17:13:05 2023\n",
            "Ended at Wed Jul 19 17:13:09 2023\n",
            "Removing temporary files\n",
            "---------------------- Step 5 Finished ----------------------\n",
            "Running step 6:\n",
            "Putting together, sorting, making them uniq and then splitting into one file per exon for all the liftover files created for UNMAPPED EXONS so far\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch1-multiples-partCoding.bed.gz |awk -v OFS='\\t' '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch1-multiples-partCoding >> oneHugeFile-1to2-partCoding.txt\n",
            "Started at Wed Jul 19 17:13:09 2023\n",
            "Ended at Wed Jul 19 17:13:09 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch1-multiples-partCoding.bed.gz |awk -v OFS='\\t' '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch1-multiples-partCoding >> oneHugeFile-2to1-partCoding.txt\n",
            "Started at Wed Jul 19 17:13:09 2023\n",
            "Ended at Wed Jul 19 17:13:09 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch0.95-multiples-partCoding.bed.gz |awk -v OFS='\\t' '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.95-multiples-partCoding >> oneHugeFile-1to2-partCoding.txt\n",
            "Started at Wed Jul 19 17:13:09 2023\n",
            "Ended at Wed Jul 19 17:13:09 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch0.95-multiples-partCoding.bed.gz |awk -v OFS='\\t' '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.95-multiples-partCoding >> oneHugeFile-2to1-partCoding.txt\n",
            "Started at Wed Jul 19 17:13:09 2023\n",
            "Ended at Wed Jul 19 17:13:10 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch0.9-multiples-partCoding.bed.gz |awk -v OFS='\\t' '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.9-multiples-partCoding >> oneHugeFile-1to2-partCoding.txt\n",
            "Started at Wed Jul 19 17:13:10 2023\n",
            "Ended at Wed Jul 19 17:13:10 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch0.9-multiples-partCoding.bed.gz |awk -v OFS='\\t' '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.9-multiples-partCoding >> oneHugeFile-2to1-partCoding.txt\n",
            "Started at Wed Jul 19 17:13:10 2023\n",
            "Ended at Wed Jul 19 17:13:10 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch1-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch1-multiples >> oneHugeFile-1to2-others.txt\n",
            "Started at Wed Jul 19 17:13:10 2023\n",
            "Ended at Wed Jul 19 17:13:10 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch1-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch1-multiples >> oneHugeFile-2to1-others.txt\n",
            "Started at Wed Jul 19 17:13:10 2023\n",
            "Ended at Wed Jul 19 17:13:11 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch0.95-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.95-multiples >> oneHugeFile-1to2-others.txt\n",
            "Started at Wed Jul 19 17:13:11 2023\n",
            "Ended at Wed Jul 19 17:13:11 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch0.95-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.95-multiples >> oneHugeFile-2to1-others.txt\n",
            "Started at Wed Jul 19 17:13:11 2023\n",
            "Ended at Wed Jul 19 17:13:12 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_to_org2_liftOver_unmappedExonsList-flank0-minMatch0.9-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.9-multiples >> oneHugeFile-1to2-others.txt\n",
            "Started at Wed Jul 19 17:13:12 2023\n",
            "Ended at Wed Jul 19 17:13:12 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_to_org1_liftOver_unmappedExonsList-flank0-minMatch0.9-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.9-multiples >> oneHugeFile-2to1-others.txt\n",
            "Started at Wed Jul 19 17:13:12 2023\n",
            "Ended at Wed Jul 19 17:13:13 2023\n",
            "Running : cat oneHugeFile-1to2-partCoding.txt oneHugeFile-1to2-others.txt | sort -k5,5 >oneHugeFile-1to2.txt.sorted\n",
            "Started at Wed Jul 19 17:13:13 2023\n",
            "Ended at Wed Jul 19 17:13:18 2023\n",
            "Running : cat oneHugeFile-2to1-partCoding.txt oneHugeFile-2to1-others.txt | sort -k5,5 >oneHugeFile-2to1.txt.sorted\n",
            "Started at Wed Jul 19 17:13:18 2023\n",
            "Ended at Wed Jul 19 17:13:32 2023\n",
            "Running : python /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/splitExonsIntoIndividualFiles.py oneHugeFile-1to2.txt.sorted /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perExonLiftoverCoords/org1 5 _unmapped.txt\n",
            "Started at Wed Jul 19 17:13:32 2023\n",
            "Ended at Wed Jul 19 17:14:03 2023\n",
            "Running : python /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/splitExonsIntoIndividualFiles.py oneHugeFile-2to1.txt.sorted /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perExonLiftoverCoords/org2 5 _unmapped.txt\n",
            "Started at Wed Jul 19 17:14:03 2023\n",
            "Ended at Wed Jul 19 17:14:34 2023\n",
            "---------------------- Step 6 Finished ----------------------\n",
            "Runing step 7:\n",
            "Putting together, sorting, making them uniq and then splitting into one file per exon for all the liftover files for MAPPED EXONS that DO NOT INTERSECT WITH AN EXON so far\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_nonintersectingExonsList-flank0-minMatch1-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch1-multiples >> oneHugeFile-2to1-others.txt\n",
            "Started at Wed Jul 19 17:14:34 2023\n",
            "Ended at Wed Jul 19 17:14:34 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_nonintersectingExonsList-flank0-minMatch1-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch1-multiples >> oneHugeFile-1to2-others.txt\n",
            "Started at Wed Jul 19 17:14:34 2023\n",
            "Ended at Wed Jul 19 17:14:34 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_nonintersectingExonsList-flank0-minMatch0.95-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.95-multiples >> oneHugeFile-2to1-others.txt\n",
            "Started at Wed Jul 19 17:14:34 2023\n",
            "Ended at Wed Jul 19 17:14:34 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_nonintersectingExonsList-flank0-minMatch0.95-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.95-multiples >> oneHugeFile-1to2-others.txt\n",
            "Started at Wed Jul 19 17:14:34 2023\n",
            "Ended at Wed Jul 19 17:14:35 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org1_VS_org2_to_org1_nonintersectingExonsList-flank0-minMatch0.9-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.9-multiples >> oneHugeFile-2to1-others.txt\n",
            "Started at Wed Jul 19 17:14:35 2023\n",
            "Ended at Wed Jul 19 17:14:35 2023\n",
            "Running : zcat /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/liftoverRelatedFiles/org2_VS_org1_to_org2_nonintersectingExonsList-flank0-minMatch0.9-multiples.bed.gz |awk '{print $1,$2,$3,$6,$4,$5,s}' s=flank0-minMatch0.9-multiples >> oneHugeFile-1to2-others.txt\n",
            "Started at Wed Jul 19 17:14:35 2023\n",
            "Ended at Wed Jul 19 17:14:35 2023\n",
            "Running : cat oneHugeFile-1to2-others.txt | sort -k5,5 >oneHugeFile-1to2.txt.sorted\n",
            "Started at Wed Jul 19 17:14:35 2023\n",
            "Ended at Wed Jul 19 17:14:47 2023\n",
            "Running : cat oneHugeFile-2to1-others.txt | sort -k5,5 >oneHugeFile-2to1.txt.sorted\n",
            "Started at Wed Jul 19 17:14:47 2023\n",
            "Ended at Wed Jul 19 17:14:49 2023\n",
            "Running : python /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/splitExonsIntoIndividualFiles.py oneHugeFile-1to2.txt.sorted /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perExonLiftoverCoords/org1 5 _nonintersecting.txt\n",
            "Started at Wed Jul 19 17:14:49 2023\n",
            "Ended at Wed Jul 19 17:15:23 2023\n",
            "Running : python /content/ExTraMapper/Human-Mouse-Preprocess-Data/scripts/splitExonsIntoIndividualFiles.py oneHugeFile-2to1.txt.sorted /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/perExonLiftoverCoords/org2 5 _nonintersecting.txt\n",
            "Started at Wed Jul 19 17:15:23 2023\n",
            "Ended at Wed Jul 19 17:15:29 2023\n",
            "Removing temporary files\n",
            "---------------------- Step 7 Finished ----------------------\n",
            "Preporcessing steps finished!\n"
          ]
        }
      ],
      "source": [
        "#run preprocessing\n",
        "!./extMpreprocess config.human-mouse.conf all"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "background_save": true
        },
        "id": "Qil6QHqiuakT",
        "outputId": "2eaf3c5c-9d5d-4669-be77-1fb982b257a9"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "/content/ExTraMapper\n"
          ]
        }
      ],
      "source": [
        "%cd /content/ExTraMapper/"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "background_save": true
        },
        "id": "4u1HXegAuffO"
      },
      "outputs": [],
      "source": [
        "import os\n",
        "os.environ['EXTRAMAPPER_DIR'] = '/content/ExTraMapper/Human-Mouse-Preprocess-Data'"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "background_save": true
        },
        "id": "273GKkC9uhB4",
        "outputId": "7745a5b1-c4e7-4f9d-a7e6-b0e7f6e2e3a0"
      },
      "outputs": [
        {
          "name": "stdout",
          "output_type": "stream",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "ACCTTGGGGATGGCCTCGCTCACAGCGTAGCTGGCCTTGTCACTGACCCACACTTTCTCCCTCGGGGACAGGTCAGCACACAGGGCCTTGAGCTCACTCAGGATGGACTTGTATGGCAGGACCTG\n",
            "ACCTTGGGGATGGCCTCGCTCACAGCGTAGCTGGCCTTGTCACTGACCCACACTTTCTCCCTCGGGGACAGGTCAGCACACAGGGCCTTGAGCTCACTCAGGATGGACTTGTATGGCAGGACCTG\n",
            "TGTCTCCCCTGCTCAGACTCCTGTGCAGAGGCAAATGCAAGGCCAGGCCTGTGCTGCTCCATCTGCATGCCTTACGCGTAGTGGATGATGGCGCCGTTGGGTCCCGTGCTGGAAATCGTTGGGAAGCTCAGATCCACAAAGTCAGCCTGCTGCCT\n",
            "TGTCTCCCCTGCTCAGACTCCTGTGCAGAGGCAAATGCAAGGCCAGGCCTGTGCTGCTCCATCTGCATGCCTTACGCGTAGTGGATGATGGCGCCGTTGGGTCCCGTGCTGGAAATCGTTGGGAAGCTCAGATCCACAAAGTCAGCCTGCTGCCT\n",
            "AATTCTGGCACCACCACCACCACCCCCCGCCCCAACAAATATAAGGAGGACACAGCATTCTTTCCATTC\n",
            "AATTCTGGCACCACCACCACCACCCCCCGCCCCAACAAATATAAGGAGGACACAGCATTCTTTCCATTC\n",
            "AATTCTGGCACCACCACCACCACCCCCCGCCCCAACAAATATAAGGAGGACACAGCATTCTTTCCATTC\n",
            "CGGAGTTCTTCACGGCTTTGGCAATGCAGATGGGTGTGTAAGGCATACAGCAGCGGTGATCCT\n",
            "CGGAGTTCTTCACGGCTTTGGCAATGCAGATGGGTGTGTAAGGCATACAGCAGCGGTGATCCT\n",
            "CGGAGTTCTTCACGGCTTTGGCAATGCAGATGGGTGTGTAAGGCATACAGCAGCGGTGATCCT\n",
            "ACATGATCGTCTCTAGGCCTACGATCGCATAGGAGAAAAACACAGGATTGTGTTCCACATCT\n",
            "ACATGATCGTCTCTAGGCCTACGATCGCATAGGAGAAAAACACAGGATTGTGTTCCACATCT\n",
            "ACATGATCGTCTCTAGGCCTACGATCGCATAGGAGAAAAACACAGGATTGTGTTCCACATCT\n",
            "TTCTTCACGGCTTTGGCAATGCAGATGGGTGTGTAAGGCATACAGCAGCGGTGATCCT\n",
            "TTCTTCACGGCTTTGGCAATGCAGATGGGTGTGTAAGGCATACAGCAGCGGTGATCCT\n",
            "TTCTTCACGGCTTTGGCAATGCAGATGGGTGTGTAAGGCATACAGCAGCGGTGATCCT\n",
            "ACCTGGTGGGCATCTCCTGACGGGATGATATAGGCCTGGATGGGTTCTGCCACGTATTCGGAGTTCCTCATAGCTTGTCGCAGCTGCCGAAGCAGTTCTGAAGTCACCTTCGGAGCCATCCTGTGGTCT\n",
            "ACCTGGTGGGCATCTCCTGACGGGATGATATAGGCCTGGATGGGTTCTGCCACGTATTCGGAGTTCCTCATAGCTTGTCGCAGCTGCCGAAGCAGTTCTGAAGTCACCTTCGGAGCCATCCTGTGGTCT\n",
            "ACCTGGTGGGCATCTCCTGACGGGATGATATAGGCCTGGATGGGTTCTGCCACGTATTCGGAGTTCCTCATAGCTTGTCGCAGCTGCCGAAGCAGTTCTGAAGTCACCTTCGGAGCCATCCTGTGGTCT\n",
            "TTGGGGATGGCCTCGCTCACAGCGTAGCTGGCCTTGTCACTGACCCACACTTTCTCCCTCGGGGACAGGTCAGCACACAGGGCCTTGAGCTCACTCAGGATGGACTTGTATGGCAGGACCTGGATTCTGTATTCCGCCTCCAGGCCCAAGTCAAGCAGCAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "TTGGGGATGGCCTCGCTCACAGCGTAGCTGGCCTTGTCACTGACCCACACTTTCTCCCTCGGGGACAGGTCAGCACACAGGGCCTTGAGCTCACTCAGGATGGACTTGTATGGCAGGACCTGGATTCTGTATTCCGCCTCCAGGCCCAAGTCAAGCAGCAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "TTGGGGATGGCCTCGCTCACAGCGTAGCTGGCCTTGTCACTGACCCACACTTTCTCCCTCGGGGACAGGTCAGCACACAGGGCCTTGAGCTCACTCAGGATGGACTTGTATGGCAGGACCTGGATTCTGTATTCCGCCTCCAGGCCCAAGTCAAGCAGCAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "ACCCATCTTCATGAGAGTCCAGTTGTTGTCCATCTGTTTGGCAGCCTGGAGGAAGTAGCGCCCGTCAGTCCACATGGCCGCATGCTCCTCCGTGATGATGGCTGTGCCT\n",
            "ACCCATCTTCATGAGAGTCCAGTTGTTGTCCATCTGTTTGGCAGCCTGGAGGAAGTAGCGCCCGTCAGTCCACATGGCCGCATGCTCCTCCGTGATGATGGCTGTGCCT\n",
            "ACCCATCTTCATGAGAGTCCAGTTGTTGTCCATCTGTTTGGCAGCCTGGAGGAAGTAGCGCCCGTCAGTCCACATGGCCGCATGCTCCTCCGTGATGATGGCTGTGCCT\n",
            "CTCACTCAGGATGGACTTGTATGGCAGGACCTGGATTCTGTATTCCGCCTCCAGGCCCAAGTCAAGCAGCAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "CTCACTCAGGATGGACTTGTATGGCAGGACCTGGATTCTGTATTCCGCCTCCAGGCCCAAGTCAAGCAGCAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "CTCACTCAGGATGGACTTGTATGGCAGGACCTGGATTCTGTATTCCGCCTCCAGGCCCAAGTCAAGCAGCAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "ACCTGGTGGGCATCTCCTGACGGGATGATATAGGCCTGGATGGGTTCTGCCACGTATTCGGAGTTCCTCATAGCTTGTCGCAGCTGCCGAAGCAGTTCTGAAGTCACCTTCGGAGCCATCCTGTGGTCTGCAA\n",
            "ACCTGGTGGGCATCTCCTGACGGGATGATATAGGCCTGGATGGGTTCTGCCACGTATTCGGAGTTCCTCATAGCTTGTCGCAGCTGCCGAAGCAGTTCTGAAGTCACCTTCGGAGCCATCCTGTGGTCTGCAA\n",
            "ACCTGGTGGGCATCTCCTGACGGGATGATATAGGCCTGGATGGGTTCTGCCACGTATTCGGAGTTCCTCATAGCTTGTCGCAGCTGCCGAAGCAGTTCTGAAGTCACCTTCGGAGCCATCCTGTGGTCTGCAA\n",
            "CAGGACCTGGATTCTGTATTCCGCCTCCAGGCCCAAGTCAAGCAGCAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "CAGGACCTGGATTCTGTATTCCGCCTCCAGGCCCAAGTCAAGCAGCAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "CAGGACCTGGATTCTGTATTCCGCCTCCAGGCCCAAGTCAAGCAGCAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "ACCTGCAGAGCCATCGAATCCAGAGACAAACGCCCTCCGACAGTCACAGGGTGCAATGTACTCGCTCT\n",
            "ACCTGCAGAGCCATCGAATCCAGAGACAAACGCCCTCCGACAGTCACAGGGTGCAATGTACTCGCTCT\n",
            "ACCTGCAGAGCCATCGAATCCAGAGACAAACGCCCTCCGACAGTCACAGGGTGCAATGTACTCGCTCT\n",
            "CAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "CAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "CAGGTGCTGCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "GCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "GCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "GCTTCACGCCCGGCGCGTCGACGCGGTCCCCGTCAATGAAGAGCCT\n",
            "TACATGATCGTCTCTAGGCCTACGATCGCATAGGAGAAAAACACAGGATTGTGTTCCACATCTGACCCGCGGAGATTGAACAGCCCT\n",
            "TACATGATCGTCTCTAGGCCTACGATCGCATAGGAGAAAAACACAGGATTGTGTTCCACATCTGACCCGCGGAGATTGAACAGCCCT\n",
            "TACATGATCGTCTCTAGGCCTACGATCGCATAGGAGAAAAACACAGGATTGTGTTCCACATCTGACCCGCGGAGATTGAACAGCCCT\n",
            "AGCCCAAAGGGGAAACCGCCCAAAAAGTCAAGGGTGTAACAGTTTTGGAAAAGTTAGGGCATAACACCAAAGGACCTCAACATCAGCCACTGTTATCTAACGCTGTTAA\n",
            "AGCCCAAAGGGGAAACCGCCCAAAAAGTCAAGGGTGTAACAGTTTTGGAAAAGTTAGGGCATAACACCAAAGGACCTCAACATCAGCCACTGTTATCTAACGCTGTTAA\n",
            "ACACAGGATTGTGTTCCACATCTGACCCGCGGAGATTGAACAGCCCT\n",
            "ACACAGGATTGTGTTCCACATCTGACCCGCGGAGATTGAACAGCCCT\n",
            "ACACAGGATTGTGTTCCACATCTGACCCGCGGAGATTGAACAGCCCT\n",
            "CTGTGACTACAAACCACGCGATGCTCCTCTCTGCCATTTTCAACCGGAGGTCTGCAACCTTCTCCTTCCACGAGATGCCT\n",
            "CTGTGACTACAAACCACGCGATGCTCCTCTCTGCCATTTTCAACCGGAGGTCTGCAACCTTCTCCTTCCACGAGATGCCT\n",
            "CTGTGACTACAAACCACGCGATGCTCCTCTCTGCCATTTTCAACCGGAGGTCTGCAACCTTCTCCTTCCACGAGATGCCT\n",
            "GACTACAAACCACGCGATGCTCCTCTCTGCCATTTTCAACCGGAGGTCTGCAACCTTCTCCTTCCACGAGATGCCT\n",
            "GACTACAAACCACGCGATGCTCCTCTCTGCCATTTTCAACCGGAGGTCTGCAACCTTCTCCTTCCACGAGATGCCT\n",
            "GACTACAAACCACGCGATGCTCCTCTCTGCCATTTTCAACCGGAGGTCTGCAACCTTCTCCTTCCACGAGATGCCT\n",
            "GGCCGGTCTGTCCAGATCTTGTCCACAAGGTTCTCCTTCACAGGCACGAGGTGGTGACCAGCACTTCGCAGTACCTTGGCCATCTTCTTCCAGTAATCT\n",
            "GGCCGGTCTGTCCAGATCTTGTCCACAAGGTTCTCCTTCACAGGCACGAGGTGGTGACCAGCACTTCGCAGTACCTTGGCCATCTTCTTCCAGTAATCT\n",
            "GGCCGGTCTGTCCAGATCTTGTCCACAAGGTTCTCCTTCACAGGCACGAGGTGGTGACCAGCACTTCGCAGTACCTTGGCCATCTTCTTCCAGTAATCT\n",
            "GTCACTCACCTGTGGGAATGATGAGGGGGTCCACACCAACCCTGGATCCTTCTGGAAGCACACTCACCAGCCAGTCCTCCTGAGTTGGTGTGTCTTTCAGACCT\n",
            "GTCACTCACCTGTGGGAATGATGAGGGGGTCCACACCAACCCTGGATCCTTCTGGAAGCACACTCACCAGCCAGTCCTCCTGAGTTGGTGTGTCTTTCAGACCT\n",
            "GTCACTCACCTGTGGGAATGATGAGGGGGTCCACACCAACCCTGGATCCTTCTGGAAGCACACTCACCAGCCAGTCCTCCTGAGTTGGTGTGTCTTTCAGACCT\n",
            "ACCTGGTGGGCATCTCCTGACGGGATGATATAGGCCTGGATGGGTTCTGCCACGTATTCGGAGTTCCTCATAGCTTGTCGCAGCTGCCGAAGCAGTTCTGAAGTCACCTTCGGAGCCATCCTGTGGTCTGCA\n",
            "ACCTGGTGGGCATCTCCTGACGGGATGATATAGGCCTGGATGGGTTCTGCCACGTATTCGGAGTTCCTCATAGCTTGTCGCAGCTGCCGAAGCAGTTCTGAAGTCACCTTCGGAGCCATCCTGTGGTCTGCA\n",
            "ACCTGGTGGGCATCTCCTGACGGGATGATATAGGCCTGGATGGGTTCTGCCACGTATTCGGAGTTCCTCATAGCTTGTCGCAGCTGCCGAAGCAGTTCTGAAGTCACCTTCGGAGCCATCCTGTGGTCTGCA\n",
            "TGGTTTTTTGTTTTTCCTTGGCTTTTCAAAGGATCCAAGATGTCAGGAGAAGCACTAAGATCCTGGGTCCCCAAAGCATTCTACCAGCCCTGCCCAGACCATGTAAGTTCTGCCAGGGGGGCTGGCCCCTGCTTCTACCCACCCATCTTCTACCCACCCATCTTCATGAGAGTCCAGTTGTTGTCCATCTGTTTGGCAGCCTGGAGGAAGTAGCGCCCGTCAGTCCACATGGCCGCATGCTCCTCCGTGATGATGGCTGTGCCT\n",
            "CCACCCATCTTCATGAGAGTCCAGTTGTTGTCCATCTGTTTGGCAGCCTGGAGGAAGTAGCGCCCGTCAGTCCACATGGCCGCATGCTCCTCCGTGATGATGGCTGTGCCT\n",
            "CCACCCATCTTCATGAGAGTCCAGTTGTTGTCCATCTGTTTGGCAGCCTGGAGGAAGTAGCGCCCGTCAGTCCACATGGCCGCATGCTCCTCCGTGATGATGGCTGTGCCT\n",
            "CCACCCATCTTCATGAGAGTCCAGTTGTTGTCCATCTGTTTGGCAGCCTGGAGGAAGTAGCGCCCGTCAGTCCACATGGCCGCATGCTCCTCCGTGATGATGGCTGTGCCT\n",
            "Exon file type summaries for the first gene from: ENSG00000108039-ENSMUSG00000025027\n",
            "\t0 exons with: No file exists\n",
            "\t4 exons with: Only Mapped\n",
            "\t70 exons with: Only nonintersecting\n",
            "\t8 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t16 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "CTGGCACTGGTCCCTGCACAAAAAGGACTCCACTTGCACAGTATGAGTCAAGAACAGAGAATGGAGAACTGTAGACCTATGGCCAAAGTGGTCTCGTATAATCAACAAGACACAGGGAGGGATGGTTCCTGCTCAGCAATTACAGCCACTCCTCTGCTCCACCAGACACCCCAGGGG\n",
            "CTGGCACTGGTCCCTGCACAAAAAGGACTCCACTTGCACAGTATGAGTCAAGAACAGAGAATGGAGAACTGTAGACCTATGGCCAAAGTGGTCTCGTATAATCAACAAGACACAGGGAGGGATGGTTCCTGCTCAGCAATTACAGCCACTCCTCTGCTCCACCAGACACCCCAGGGG\n",
            "AGCCAAGGACTGTAACATTTCTATCAAACAGTCTTTTGTGAGAACAGAGTCACAACTGGGCTATTTCAAATAGGGGTGAAAAAAATGACAAGACACAGGAACGATTCCATTAGTGGCCAGCACAGAGGGCGATGACCAGCTGAGTGCTCTCAGCTGGAGCTCAGC\n",
            "AGCCAAGGACTGTAACATTTCTATCAAACAGTCTTTTGTGAGAACAGAGTCACAACTGGGCTATTTCAAATAGGGGTGAAAAAAATGACAAGACACAGGAACGATTCCATTAGTGGCCAGCACAGAGGGCGATGACCAGCTGAGTGCTCTCAGCTGGAGCTCAGC\n",
            "AGCCAAGGACTGTAACATTTCTATCAAACAGTCTTTTGTGAGAACAGAGTCACAACTGGGCTATTTCAAATAGGGGTGAAAAAAATGACAAGACACAGGAACGATTCCATTAGTGGCCAGCACAGAGGGCGATGACCAGCTGAGTGCTCTCAGCTGGAGCTCAGC\n",
            "CCTGCCCCACTACATTGCATAAAACCCTTTTCCTTGGATAATCTTCCTGAAGGAGCTCTCTTGGACAAACAGATTCACTTAGGTGCTCAATCAGCATTTGCTGAGCAGTGACTATGATCCAGGAACTT\n",
            "CCTGCCCCACTACATTGCATAAAACCCTTTTCCTTGGATAATCTTCCTGAAGGAGCTCTCTTGGACAAACAGATTCACTTAGGTGCTCAATCAGCATTTGCTGAGCAGTGACTATGATCCAGGAACTT\n",
            "CTGGCCATATTCATATCTGACCAAGGTCCTTTGCTGCCATCTGGTGATAAGTGATGACGATGCAGACCCAAACCAAAAACTCCCTAAAGTGACTCCAAAATAA\n",
            "CTGGCCATATTCATATCTGACCAAGGTCCTTTGCTGCCATCTGGTGATAAGTGATGACGATGCAGACCCAAACCAAAAACTCCCTAAAGTGACTCCAAAATAA\n",
            "CTGGCCATATTCATATCTGACCAAGGTCCTTTGCTGCCATCTGGTGATAAGTGATGACGATGCAGACCCAAACCAAAAACTCCCTAAAGTGACTCCAAAATAA\n",
            "CTGGCACTGGTCCCTGCACAAAAAGGACTCCACTTGCACAGTATGAGTCAAGAACAGAGAATGGAGAACTGTAGACCTATGGCCAAAGTGGTCTCGTATAATCAACAAGACACAGGGAGGGATGGTTCCTGCTCAGCAATTACAGCCACTCCTCTGCTCCACCAGACACCCCAGGGG\n",
            "CTGGCACTGGTCCCTGCACAAAAAGGACTCCACTTGCACAGTATGAGTCAAGAACAGAGAATGGAGAACTGTAGACCTATGGCCAAAGTGGTCTCGTATAATCAACAAGACACAGGGAGGGATGGTTCCTGCTCAGCAATTACAGCCACTCCTCTGCTCCACCAGACACCCCAGGGG\n",
            "AGCCAAGGACTGTAACATTTCTATCAAACAGTCTTTTGTGAGAACAGAGTCACAACTGGGCTATTTCAAATAGGGGTGAAAAAAATGACAAGACACAGGAACGATTCCATTAGTGGCCAGCACAGAGGGCGATGACCAGCTGAGTGCTCTCAGCTGGAGCTCAGC\n",
            "AGCCAAGGACTGTAACATTTCTATCAAACAGTCTTTTGTGAGAACAGAGTCACAACTGGGCTATTTCAAATAGGGGTGAAAAAAATGACAAGACACAGGAACGATTCCATTAGTGGCCAGCACAGAGGGCGATGACCAGCTGAGTGCTCTCAGCTGGAGCTCAGC\n",
            "AGCCAAGGACTGTAACATTTCTATCAAACAGTCTTTTGTGAGAACAGAGTCACAACTGGGCTATTTCAAATAGGGGTGAAAAAAATGACAAGACACAGGAACGATTCCATTAGTGGCCAGCACAGAGGGCGATGACCAGCTGAGTGCTCTCAGCTGGAGCTCAGC\n",
            "CCTGCCCCACTACATTGCATAAAACCCTTTTCCTTGGATAATCTTCCTGAAGGAGCTCTCTTGGACAAACAGATTCACTTAGGTGCTCAATCAGCATTTGCTGAGCAGTGACTATGATCCAGGAACTT\n",
            "CCTGCCCCACTACATTGCATAAAACCCTTTTCCTTGGATAATCTTCCTGAAGGAGCTCTCTTGGACAAACAGATTCACTTAGGTGCTCAATCAGCATTTGCTGAGCAGTGACTATGATCCAGGAACTT\n",
            "CTGGCCATATTCATATCTGACCAAGGTCCTTTGCTGCCATCTGGTGATAAGTGATGACGATGCAGACCCAAACCAAAAACTCCCTAAAGTGACTCCAAAATAA\n",
            "CTGGCCATATTCATATCTGACCAAGGTCCTTTGCTGCCATCTGGTGATAAGTGATGACGATGCAGACCCAAACCAAAAACTCCCTAAAGTGACTCCAAAATAA\n",
            "CTGGCCATATTCATATCTGACCAAGGTCCTTTGCTGCCATCTGGTGATAAGTGATGACGATGCAGACCCAAACCAAAAACTCCCTAAAGTGACTCCAAAATAA\n",
            "ATCATAGCTGTAGTCCTATGTCTACATAACAAAACAACTTCAGAA\n",
            "ATCATAGCTGTAGTCCTATGTCTACATAACAAAACAACTTCAGAA\n",
            "ATCATAGCTGTAGTCCTATGTCTACATAACAAAACAACTTCAGAA\n",
            "CTGGCACTGGTCCCTGCACAAAAAGGACTCCACTTGCACAGTATGAGTCAAGAACAGAGAATGGAGAACTGTAGACCTATGGCCAAAGTGGTCTCGTATAATCAACAAGACACAGGGAGGGATGGTTCCTGCTCAGCAATTACAGCCACTCCTCTGCTCCACCAGACACCCCAGGGG\n",
            "CTGGCACTGGTCCCTGCACAAAAAGGACTCCACTTGCACAGTATGAGTCAAGAACAGAGAATGGAGAACTGTAGACCTATGGCCAAAGTGGTCTCGTATAATCAACAAGACACAGGGAGGGATGGTTCCTGCTCAGCAATTACAGCCACTCCTCTGCTCCACCAGACACCCCAGGGG\n",
            "CTGGCCATATTCATATCTGACCAAGGTCCTTTGCTGCCATCTGGTGATAAGTGATGACGATGCAGACCCAAACCAAAAACTCCCTAAAGTGACTCCAAAATAAAGGCTAAATAGTAAAGGGCAAATGACCAGATATTTTA\n",
            "AGAAAAACTGAGACCCTCCCACAAGGGAAGCCAGGCAAGCCTCCTCTGTAAGGGTCAGTGAGCCCCATGGAGACGGAGCTTCCTGTCAGAGGTGGAAAGAAAAGCAGTGAGTCAGCAAGGCTGCAGCTCACCAATGAAATCAGAGCCTGCAGCCCACCCATGGGTCTGCTCTGGCACCACGACTGGGTGGGGGACCTGCGGACAGCCAGTAACAGTGATGTTGCTGAACGTCTGAGAACTAGACAGACCAGGAGAGAGCACAGTTCTGGTCCCAGGACTTCCAGCTTCCGTCACTTGTTCTCATGAGGATCAGAGCAAAAAAACGCCTGGGCATTAAAAGTATAACCTCTATTTTAAAATAATGGCTTCTTAAAGTGATGAGGCACCATT\n",
            "AGAAAAACTGAGACCCTCCCACAAGGGAAGCCAGGCAAGCCTCCTCTGTAAGGGTCAGTGAGCCCCATGGAGACGGAGCTTCCTGTCAGAGGTGGAAAGAAAAGCAGTGAGTCAGCAAGGCTGCAGCTCACCAATGAAATCAGAGCCTGCAGCCCACCCATGGGTCTGCTCTGGCACCACGACTGGGTGGGGGACCTGCGGACAGCCAGTAACAGTGATGTTGCTGAACGTCTGAGAACTAGACAGACCAGGAGAGAGCACAGTTCTGGTCCCAGGACTTCCAGCTTCCGTCACTTGTTCTCATGAGGATCAGAGCAAAAAAACGCCTGGGCATTAAAAGTATAACCTCTATTTTAAAATAATGGCTTCTTAAAGTGATGAGGCACCATT\n",
            "AGAAAAACTGAGACCCTCCCACAAGGGAAGCCAGGCAAGCCTCCTCTGTAAGGGTCAGTGAGCCCCATGGAGACGGAGCTTCCTGTCAGAGGTGGAAAGAAAAGCAGTGAGTCAGCAAGGCTGCAGCTCACCAATGAAATCAGAGCCTGCAGCCCACCCATGGGTCTGCTCTGGCACCACGACTGGGTGGGGGACCTGCGGACAGCCAGTAACAGTGAT\n",
            "AGAAAAACTGAGACCCTCCCACAAGGGAAGCCAGGCAAGCCTCCTCTGTAAGGGTCAGTGAGCCCCATGGAGACGGAGCTTCCTGTCAGAGGTGGAAAGAAAAGCAGTGAGTCAGCAAGGCTGCAGCTCACCAATGAAATCAGAGCCTGCAGCCCACCCATGGGTCTGCTCTGGCACCACGACTGGGTGGGGGACCTGCGGACAGCCAGTAACAGTGAT\n",
            "ACCTGCCCCACTACATTGCATAAAACCCTTTTCCTTGGATAATCTTCCTGAAGGAGCTCTCTTGGACAAACAGATTCACTTAGGTGCTCAATCAGCATTTGCTGAGCAGTGACTATGATCCAGGAACTT\n",
            "ACCTGCCCCACTACATTGCATAAAACCCTTTTCCTTGGATAATCTTCCTGAAGGAGCTCTCTTGGACAAACAGATTCACTTAGGTGCTCAATCAGCATTTGCTGAGCAGTGACTATGATCCAGGAACTT\n",
            "ATCATAGCTGTAGTCCTATGTCTACATAACAAAACAACTTCAGAAGGAGCCA\n",
            "ATCATAGCTGTAGTCCTATGTCTACATAACAAAACAACTTCAGAAGGAGCCA\n",
            "ATCATAGCTGTAGTCCTATGTCTACATAACAAAACAACTTCAGAAGGAGCCA\n",
            "CTGGCACTGGTCCCTGCACAAAAAGGACTCCACTTGCACAGTATGAGTCAAGAACAGAGAATGGAGAACTGTAGACCTATGGCCAAAGTGGTCTCGTATAATCAACAAGACACAGGGAGGGATGGTTCCTGCTCAGCAATTACAGCCACTCCTCTGCTCCACCAGACACCCCAGGGG\n",
            "CTGGCACTGGTCCCTGCACAAAAAGGACTCCACTTGCACAGTATGAGTCAAGAACAGAGAATGGAGAACTGTAGACCTATGGCCAAAGTGGTCTCGTATAATCAACAAGACACAGGGAGGGATGGTTCCTGCTCAGCAATTACAGCCACTCCTCTGCTCCACCAGACACCCCAGGGG\n",
            "CCATCTTTAAATCAGCTTTATTAAATTCAACTTTATTTTAAATAATTATATCATGAACTCATGCTAGTTATGTTTTTTCTATTATATGCTAAATGCTGTTAAAACAGATATATTTAAGCAAACCTAAATAATAATGT\n",
            "CCATCTTTAAATCAGCTTTATTAAATTCAACTTTATTTTAAATAATTATATCATGAACTCATGCTAGTTATGTTTTTTCTATTATATGCTAAATGCTGTTAAAACAGATATATTTAAGCAAACCTAAATAATAATGT\n",
            "ATCATAGCTGTAGTCCTATGTCTACATAACAAAACAACTTCAGAAGGAGCCAATTA\n",
            "ATCATAGCTGTAGTCCTATGTCTACATAACAAAACAACTTCAGAAGGAGCCAATTA\n",
            "ATCATAGCTGTAGTCCTATGTCTACATAACAAAACAACTTCAGAAGGAGCCAATTA\n",
            "Exon file type summaries for the second gene from: ENSG00000108039-ENSMUSG00000025027\n",
            "\t0 exons with: No file exists\n",
            "\t3 exons with: Only Mapped\n",
            "\t7 exons with: Only nonintersecting\n",
            "\t53 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t11 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108039-ENSMUSG00000025027/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108039-ENSMUSG00000025027/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 13, Nonintersecting exons= 71, OTHER= 15\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 54, Nonintersecting exons= 18, OTHER= 3\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108039-ENSMUSG00000025027/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108039-ENSMUSG00000025027/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108039-ENSMUSG00000025027/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108039-ENSMUSG00000025027/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108039-ENSMUSG00000025027/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3212\tENSG00000108055-ENSMUSG00000024974\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108055-ENSMUSG00000024974\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108055\tchr10\t110567684\t110606048\t+\tENSG00000108055\tSMC3\tprotein_coding\t13\t79\tgene\n",
            "\n",
            "ENSMUSG00000024974\tchr19\t53588829\t53634264\t+\tENSMUSG00000024974\tSmc3\tprotein_coding\t3\t33\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t79\t54\n",
            "\n",
            "Org2\t33\t33\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "CCGCCGCCATTTTGTTTGGCCACAGGGGGAGCGAGCGGGAGCGAGCGGAGCGCTGTGAGGAGGGGGCTGTCTTGCTCGTCTTCCGAGCCTGCGGCCGCGCTGCTGCGGTGCTGAGGCGACGCCTTCCCCCTGCCGAGGGGCGCGGAGCCATGTACATCAAGCAGGT\n",
            "CCGCCGCCATTTTGTTTGGCCACAGGGGGAGCGAGCGGGAGCGAGCGGAGCGCTGTGAGGAGGGGGCTGTCTTGCTCGTCTTCCGAGCCTGCGGCCGCGCTGCTGCGGTGCTGAGGCGACGCCTTCCCCCTGCCGAGGGGCGCGGAGCCATGTACATCAAGCAGGT\n",
            "AGGTGATCATCCAGGGCTTCCGAAGTTACCGAGACCAAACAATTGTAGATCCCTTCAGCTCCAAACATAATGTCATCGGT\n",
            "AGGTGATCATCCAGGGCTTCCGAAGTTACCGAGACCAAACAATTGTAGATCCCTTCAGCTCCAAACATAATGTCATCGGT\n",
            "AGGTGATCATCCAGGGCTTCCGAAGTTACCGAGACCAAACAATTGTAGATCCCTTCAGCTCCAAACATAATGTCATCGGT\n",
            "AGTGGGCAGAAATGGCTCTGGAAAAAGCAACTTCTTTTATGGT\n",
            "AGTGGGCAGAAATGGCTCTGGAAAAAGCAACTTCTTTTATGGT\n",
            "AGTGGGCAGAAATGGCTCTGGAAAAAGCAACTTCTTTTATGGT\n",
            "AGCAATCCAGTTTGTTCTCAGTGATGAATTCAGCCATCTTCGTCCAGAGCAGCGATTGGCTTTGTTGCATGT\n",
            "AGCAATCCAGTTTGTTCTCAGTGATGAATTCAGCCATCTTCGTCCAGAGCAGCGATTGGCTTTGTTGCATGT\n",
            "AGCAATCCAGTTTGTTCTCAGTGATGAATTCAGCCATCTTCGTCCAGAGCAGCGATTGGCTTTGTTGCATGT\n",
            "AGGAGGGTACAGGTCCTCGTGTTATTTCTGCTTTTGTGGAAATCATTTTTGACAATTCGGACAACCGGTTACCAGT\n",
            "AGGAGGGTACAGGTCCTCGTGTTATTTCTGCTTTTGTGGAAATCATTTTTGACAATTCGGACAACCGGTTACCAGT\n",
            "AGGAGGGTACAGGTCCTCGTGTTATTTCTGCTTTTGTGGAAATCATTTTTGACAATTCGGACAACCGGTTACCAGT\n",
            "AGATTGATAAAGAGGAAGTTTCACTTCGAAGAGTTATTGGTGCCAAAAAGGATCAGTATTTCCTAGATAAGAAGATGGTCACGT\n",
            "AGATTGATAAAGAGGAAGTTTCACTTCGAAGAGTTATTGGTGCCAAAAAGGATCAGTATTTCCTAGATAAGAAGATGGTCACGT\n",
            "AGATTGATAAAGAGGAAGTTTCACTTCGAAGAGTTATTGGTGCCAAAAAGGATCAGTATTTCCTAGATAAGAAGATGGTCACGT\n",
            "AGGAAAAATGATGTGATGAATCTCCTTGAAAGTGCTGGGTTTTCCAGAAGTAATCCTTATTACATTGTTAAACAAGGAAAGGT\n",
            "AGGAAAAATGATGTGATGAATCTCCTTGAAAGTGCTGGGTTTTCCAGAAGTAATCCTTATTACATTGTTAAACAAGGAAAGGT\n",
            "AGGAAAAATGATGTGATGAATCTCCTTGAAAGTGCTGGGTTTTCCAGAAGTAATCCTTATTACATTGTTAAACAAGGAAAGGT\n",
            "AGATCAACCAAATGGCAACGGCACCAGATTCTCAGAGATTAAAGCTGTTGAGAGAAGTAGCTGGTACTAGAGTGTATGATGAACGGAAAGAAGAAAGCATCTCCCTGATGAAAGAAACAGGT\n",
            "AGATCAACCAAATGGCAACGGCACCAGATTCTCAGAGATTAAAGCTGTTGAGAGAAGTAGCTGGTACTAGAGTGTATGATGAACGGAAAGAAGAAAGCATCTCCCTGATGAAAGAAACAGGT\n",
            "AGATCAACCAAATGGCAACGGCACCAGATTCTCAGAGATTAAAGCTGTTGAGAGAAGTAGCTGGTACTAGAGTGTATGATGAACGGAAAGAAGAAAGCATCTCCCTGATGAAAGAAACAGGT\n",
            "AGAGGGCAAACGGGAAAAGATCAATGAGTTGTTGAAGTACATTGAAGAGCGCCTGCACACTCTAGAGGAGGAGAAGGAGGAGCTGGCCCAGTATCAGAAGTGGGATAAGATGAGGCGCGCCCTGGAGTACACCATCTACAACCAGGAGCTCAACGAGACGCGCGCTAAGCTCGACGAGGT\n",
            "AGAGGGCAAACGGGAAAAGATCAATGAGTTGTTGAAGTACATTGAAGAGCGCCTGCACACTCTAGAGGAGGAGAAGGAGGAGCTGGCCCAGTATCAGAAGTGGGATAAGATGAGGCGCGCCCTGGAGTACACCATCTACAACCAGGAGCTCAACGAGACGCGCGCTAAGCTCGACGAGGT\n",
            "AGAGGGCAAACGGGAAAAGATCAATGAGTTGTTGAAGTACATTGAAGAGCGCCTGCACACTCTAGAGGAGGAGAAGGAGGAGCTGGCCCAGTATCAGAAGTGGGATAAGATGAGGCGCGCCCTGGAGTACACCATCTACAACCAGGAGCTCAACGAGACGCGCGCTAAGCTCGACGAGGT\n",
            "AGCTTTCTGCTAAGCGAGAAACGAGTGGAGAGAAATCCAGACAACTAAGGGATGCCCAGCAGGATGCAAGAGACAAAATGGAGGT\n",
            "AGCTTTCTGCTAAGCGAGAAACGAGTGGAGAGAAATCCAGACAACTAAGGGATGCCCAGCAGGATGCAAGAGACAAAATGGAGGT\n",
            "AGCTTTCTGCTAAGCGAGAAACGAGTGGAGAGAAATCCAGACAACTAAGGGATGCCCAGCAGGATGCAAGAGACAAAATGGAGGT\n",
            "AGGATATTGAGCGCCAGGTTAGAGAACTGAAAACAAAAATTTCAGCCATGAAAGAAGAAAAAGAACAGCTTAGTGCTGAAAGACAAGAGCAGATTAAGCAAAGGACCAAGTTGGAGCTTAAAGCCAAGGATTTACAAGATGAGCTGGCGGGCAACAGTGAACAGCGGGT\n",
            "AGGATATTGAGCGCCAGGTTAGAGAACTGAAAACAAAAATTTCAGCCATGAAAGAAGAAAAAGAACAGCTTAGTGCTGAAAGACAAGAGCAGATTAAGCAAAGGACCAAGTTGGAGCTTAAAGCCAAGGATTTACAAGATGAGCTGGCGGGCAACAGTGAACAGCGGGT\n",
            "AGGATATTGAGCGCCAGGTTAGAGAACTGAAAACAAAAATTTCAGCCATGAAAGAAGAAAAAGAACAGCTTAGTGCTGAAAGACAAGAGCAGATTAAGCAAAGGACCAAGTTGGAGCTTAAAGCCAAGGATTTACAAGATGAGCTGGCGGGCAACAGTGAACAGCGGGT\n",
            "AGAAACGTTTATTAAAAGAGAGGCAGAAGCTGCTTGAAAAAATAGAAGAAAAGCAGAAAGAACTGGCAGAAACAGAACCTAAATTCAACAGCGTAAAAGAAAAAGAAGAGCGAGGAATTGCGAGGT\n",
            "AGAAACGTTTATTAAAAGAGAGGCAGAAGCTGCTTGAAAAAATAGAAGAAAAGCAGAAAGAACTGGCAGAAACAGAACCTAAATTCAACAGCGTAAAAGAAAAAGAAGAGCGAGGAATTGCGAGGT\n",
            "AGAAACGTTTATTAAAAGAGAGGCAGAAGCTGCTTGAAAAAATAGAAGAAAAGCAGAAAGAACTGGCAGAAACAGAACCTAAATTCAACAGCGTAAAAGAAAAAGAAGAGCGAGGAATTGCGAGGT\n",
            "AGTTACTTGTGGAGAGAGGAGAATGCAGAACAACAAGCACTTGCTGCTAAAAGAGAAGACCTTGAGAAGAAGCAGCAACTTCTTAGAGCGGCAACAGGAAAGGT\n",
            "AGTTACTTGTGGAGAGAGGAGAATGCAGAACAACAAGCACTTGCTGCTAAAAGAGAAGACCTTGAGAAGAAGCAGCAACTTCTTAGAGCGGCAACAGGAAAGGT\n",
            "AGTTACTTGTGGAGAGAGGAGAATGCAGAACAACAAGCACTTGCTGCTAAAAGAGAAGACCTTGAGAAGAAGCAGCAACTTCTTAGAGCGGCAACAGGAAAGGT\n",
            "AGGCCATTTTAAATGGAATAGATAGCATTAACAAAGTGCTAGAACATTTTCGGCGAAAAGGTATAAACCAGCATGTTCAAAATGGCTACCATGGCATCGTAATGAATAACTTTGAGTGTGAACCAGCTTTCTATACTTGTGTGGAAGTCACTGCTGGTAACAGGT\n",
            "AGGCCATTTTAAATGGAATAGATAGCATTAACAAAGTGCTAGAACATTTTCGGCGAAAAGGTATAAACCAGCATGTTCAAAATGGCTACCATGGCATCGTAATGAATAACTTTGAGTGTGAACCAGCTTTCTATACTTGTGTGGAAGTCACTGCTGGTAACAGGT\n",
            "AGGCCATTTTAAATGGAATAGATAGCATTAACAAAGTGCTAGAACATTTTCGGCGAAAAGGTATAAACCAGCATGTTCAAAATGGCTACCATGGCATCGTAATGAATAACTTTGAGTGTGAACCAGCTTTCTATACTTGTGTGGAAGTCACTGCTGGTAACAGGT\n",
            "AGGATGCTATTCCTATGATCAGTAAGCTGAGGTACAATCCCAGATTTGACAAAGCTTTCAAACATGTATTTGGAAAGACACTCATCTGTCGGAGCATGGAAGTTTCAACTCAGCTGGCCCGGGCCTTCACTATGGACTGTATTACTTTGGAAGGT\n",
            "AGGATGCTATTCCTATGATCAGTAAGCTGAGGTACAATCCCAGATTTGACAAAGCTTTCAAACATGTATTTGGAAAGACACTCATCTGTCGGAGCATGGAAGTTTCAACTCAGCTGGCCCGGGCCTTCACTATGGACTGTATTACTTTGGAAGGT\n",
            "AGGATGCTATTCCTATGATCAGTAAGCTGAGGTACAATCCCAGATTTGACAAAGCTTTCAAACATGTATTTGGAAAGACACTCATCTGTCGGAGCATGGAAGTTTCAACTCAGCTGGCCCGGGCCTTCACTATGGACTGTATTACTTTGGAAGGT\n",
            "TTGTTTGGCCACAGGGGGAGCGAGCGGGAGCGAGCGGAGCGCTGTGAGGAGGGGGCTGTCTTGCTCGTCTTCCGAGCCTGCGGCCGCGCTGCTGCGGTGCTGAGGCGACGCCTTCCCCCTGCCGAGGGGCGCGGAGCCATGTACATCAAGCAGGT\n",
            "TTGTTTGGCCACAGGGGGAGCGAGCGGGAGCGAGCGGAGCGCTGTGAGGAGGGGGCTGTCTTGCTCGTCTTCCGAGCCTGCGGCCGCGCTGCTGCGGTGCTGAGGCGACGCCTTCCCCCTGCCGAGGGGCGCGGAGCCATGTACATCAAGCAGGT\n",
            "AGGTGATCATCCAGGGCTTCCGAAGTTACCGAGACCAAACAATTGTAGATCCCTTCAGCTCCAAACATAATGTCATCGGT\n",
            "AGGTGATCATCCAGGGCTTCCGAAGTTACCGAGACCAAACAATTGTAGATCCCTTCAGCTCCAAACATAATGTCATCGGT\n",
            "AGGTGATCATCCAGGGCTTCCGAAGTTACCGAGACCAAACAATTGTAGATCCCTTCAGCTCCAAACATAATGTCATCGGT\n",
            "AGTGGGCAGAAATGGCTCTGGAAAAAGCAACTTCTTTTATGGT\n",
            "AGTGGGCAGAAATGGCTCTGGAAAAAGCAACTTCTTTTATGGT\n",
            "AGTGGGCAGAAATGGCTCTGGAAAAAGCAACTTCTTTTATGGT\n",
            "AGCAATCCAGTTTGTTCTCAGTGATGAATTCAGCCATCTTCGTCCAGAGCAGCGATTGGCTTTGTTGCATGT\n",
            "AGCAATCCAGTTTGTTCTCAGTGATGAATTCAGCCATCTTCGTCCAGAGCAGCGATTGGCTTTGTTGCATGT\n",
            "AGCAATCCAGTTTGTTCTCAGTGATGAATTCAGCCATCTTCGTCCAGAGCAGCGATTGGCTTTGTTGCATGT\n",
            "AGGAGGGTACAGGTCCTCGTGTTATTTCTGCTTTTGTGGAAATCATTTTTGACAATTCGGACAACCGGTTACCAGT\n",
            "AGGAGGGTACAGGTCCTCGTGTTATTTCTGCTTTTGTGGAAATCATTTTTGACAATTCGGACAACCGGTTACCAGT\n",
            "AGGAGGGTACAGGTCCTCGTGTTATTTCTGCTTTTGTGGAAATCATTTTTGACAATTCGGACAACCGGTTACCAGT\n",
            "AGATTGATAAAGAGGAAGTTTCACTTCGAAGAGTTATTGGTGCCAAAAAGGATCAGTATTTCCTAGATAAGAAGATGGTCACGT\n",
            "AGATTGATAAAGAGGAAGTTTCACTTCGAAGAGTTATTGGTGCCAAAAAGGATCAGTATTTCCTAGATAAGAAGATGGTCACGT\n",
            "AGATTGATAAAGAGGAAGTTTCACTTCGAAGAGTTATTGGTGCCAAAAAGGATCAGTATTTCCTAGATAAGAAGATGGTCACGT\n",
            "AGGAAAAATGATGTGATGAATCTCCTTGAAAGTGCTGGGTTTTCCAGAAGTAATCCTTATTACATTGTTAAACAAGGAAAGGT\n",
            "AGGAAAAATGATGTGATGAATCTCCTTGAAAGTGCTGGGTTTTCCAGAAGTAATCCTTATTACATTGTTAAACAAGGAAAGGT\n",
            "AGGAAAAATGATGTGATGAATCTCCTTGAAAGTGCTGGGTTTTCCAGAAGTAATCCTTATTACATTGTTAAACAAGGAAAGGT\n",
            "AGATCAACCAAATGGCAACGGCACCAGATTCTCAGAGATTAAAGCTGTTGAGAGAAGTAGCTGGTACTAGAGTGTATGATGAACGGAAAGAAGAAAGCATCTCCCTGATGAAAGAAACAGGT\n",
            "AGATCAACCAAATGGCAACGGCACCAGATTCTCAGAGATTAAAGCTGTTGAGAGAAGTAGCTGGTACTAGAGTGTATGATGAACGGAAAGAAGAAAGCATCTCCCTGATGAAAGAAACAGGT\n",
            "AGATCAACCAAATGGCAACGGCACCAGATTCTCAGAGATTAAAGCTGTTGAGAGAAGTAGCTGGTACTAGAGTGTATGATGAACGGAAAGAAGAAAGCATCTCCCTGATGAAAGAAACAGGT\n",
            "AGAGGGCAAACGGGAAAAGATCAATGAGTTGTTGAAGTACATTGAAGAGCGCCTGCACACTCTAGAGGAGGAGAAGGAGGAGCTGGCCCAGTATCAGAAGTGGGATAAGATGAGGCGCGCCCTGGAGTACACCATCTACAACCAGGAGCTCAACGAGACGCGCGCTAAGCTCGACGAGGT\n",
            "AGAGGGCAAACGGGAAAAGATCAATGAGTTGTTGAAGTACATTGAAGAGCGCCTGCACACTCTAGAGGAGGAGAAGGAGGAGCTGGCCCAGTATCAGAAGTGGGATAAGATGAGGCGCGCCCTGGAGTACACCATCTACAACCAGGAGCTCAACGAGACGCGCGCTAAGCTCGACGAGGT\n",
            "AGAGGGCAAACGGGAAAAGATCAATGAGTTGTTGAAGTACATTGAAGAGCGCCTGCACACTCTAGAGGAGGAGAAGGAGGAGCTGGCCCAGTATCAGAAGTGGGATAAGATGAGGCGCGCCCTGGAGTACACCATCTACAACCAGGAGCTCAACGAGACGCGCGCTAAGCTCGACGAGGT\n",
            "AGCTTTCTGCTAAGCGAGAAACGAGTGGAGAGAAATCCAGACAACTAAGGGATGCCCAGCAGGATGCAAGAGACAAAATGGAGGT\n",
            "AGCTTTCTGCTAAGCGAGAAACGAGTGGAGAGAAATCCAGACAACTAAGGGATGCCCAGCAGGATGCAAGAGACAAAATGGAGGT\n",
            "AGCTTTCTGCTAAGCGAGAAACGAGTGGAGAGAAATCCAGACAACTAAGGGATGCCCAGCAGGATGCAAGAGACAAAATGGAGGT\n",
            "AGGATATTGAGCGCCAGGTTAGAGAACTGAAAACAAAAATTTCAGCCATGAAAGAAGAAAAAGAACAGCTTAGTGCTGAAAGACAAGAGCAGATTAAGCAAAGGACCAAGTTGGAGCTTAAAGCCAAGGATTTACAAGATGAGCTGGCGGGCAACAGTGAACAGCGGGT\n",
            "AGGATATTGAGCGCCAGGTTAGAGAACTGAAAACAAAAATTTCAGCCATGAAAGAAGAAAAAGAACAGCTTAGTGCTGAAAGACAAGAGCAGATTAAGCAAAGGACCAAGTTGGAGCTTAAAGCCAAGGATTTACAAGATGAGCTGGCGGGCAACAGTGAACAGCGGGT\n",
            "AGGATATTGAGCGCCAGGTTAGAGAACTGAAAACAAAAATTTCAGCCATGAAAGAAGAAAAAGAACAGCTTAGTGCTGAAAGACAAGAGCAGATTAAGCAAAGGACCAAGTTGGAGCTTAAAGCCAAGGATTTACAAGATGAGCTGGCGGGCAACAGTGAACAGCGGGT\n",
            "AGAAACGTTTATTAAAAGAGAGGCAGAAGCTGCTTGAAAAAATAGAAGAAAAGCAGAAAGAACTGGCAGAAACAGAACCTAAATTCAACAGCGTAAAAGAAAAAGAAGAGCGAGGAATTGCGAGGT\n",
            "AGAAACGTTTATTAAAAGAGAGGCAGAAGCTGCTTGAAAAAATAGAAGAAAAGCAGAAAGAACTGGCAGAAACAGAACCTAAATTCAACAGCGTAAAAGAAAAAGAAGAGCGAGGAATTGCGAGGT\n",
            "AGAAACGTTTATTAAAAGAGAGGCAGAAGCTGCTTGAAAAAATAGAAGAAAAGCAGAAAGAACTGGCAGAAACAGAACCTAAATTCAACAGCGTAAAAGAAAAAGAAGAGCGAGGAATTGCGAGGT\n",
            "AGTTACTTGTGGAGAGAGGAGAATGCAGAACAACAAGCACTTGCTGCTAAAAGAGAAGACCTTGAGAAGAAGCAGCAACTTCTTAGAGCGGCAACAGGAAAGGT\n",
            "AGTTACTTGTGGAGAGAGGAGAATGCAGAACAACAAGCACTTGCTGCTAAAAGAGAAGACCTTGAGAAGAAGCAGCAACTTCTTAGAGCGGCAACAGGAAAGGT\n",
            "AGTTACTTGTGGAGAGAGGAGAATGCAGAACAACAAGCACTTGCTGCTAAAAGAGAAGACCTTGAGAAGAAGCAGCAACTTCTTAGAGCGGCAACAGGAAAGGT\n",
            "AGGCCATTTTAAATGGAATAGATAGCATTAACAAAGTGCTAGAACATTTTCGGCGAAAAGGTATAAACCAGCATGTTCAAAATGGCTACCATGGCATCGTAATGAATAACTTTGAGTGTGAACCAGCTTTCTATACTTGTGTGGAAGTCACTGCTGGTAACAGGT\n",
            "AGGCCATTTTAAATGGAATAGATAGCATTAACAAAGTGCTAGAACATTTTCGGCGAAAAGGTATAAACCAGCATGTTCAAAATGGCTACCATGGCATCGTAATGAATAACTTTGAGTGTGAACCAGCTTTCTATACTTGTGTGGAAGTCACTGCTGGTAACAGGT\n",
            "AGGCCATTTTAAATGGAATAGATAGCATTAACAAAGTGCTAGAACATTTTCGGCGAAAAGGTATAAACCAGCATGTTCAAAATGGCTACCATGGCATCGTAATGAATAACTTTGAGTGTGAACCAGCTTTCTATACTTGTGTGGAAGTCACTGCTGGTAACAGGT\n",
            "AGGATGCTATTCCTATGATCAGTAAGCTGAGGTACAATCCCAGATTTGACAAAGCTTTCAAACATGTATTTGGAAAGACACTCATCTGTCGGAGCATGGAAGTTTCAACTCAGCTGGCCCGGGCCTTCACTATGGACTGTATTACTTTGGAAGGT\n",
            "AGGATGCTATTCCTATGATCAGTAAGCTGAGGTACAATCCCAGATTTGACAAAGCTTTCAAACATGTATTTGGAAAGACACTCATCTGTCGGAGCATGGAAGTTTCAACTCAGCTGGCCCGGGCCTTCACTATGGACTGTATTACTTTGGAAGGT\n",
            "AGGATGCTATTCCTATGATCAGTAAGCTGAGGTACAATCCCAGATTTGACAAAGCTTTCAAACATGTATTTGGAAAGACACTCATCTGTCGGAGCATGGAAGTTTCAACTCAGCTGGCCCGGGCCTTCACTATGGACTGTATTACTTTGGAAGGT\n",
            "AGGAGGGGGCTGTCTTGCTCGTCTTCCGAGCCTGCGGCCGCGCTGCTGCGGTGCTGAGGCGACGCCTTCCCCCTGCCGAGGGGCGCGGAGCCATGTACATCAAGCAGGT\n",
            "AGGAGGGGGCTGTCTTGCTCGTCTTCCGAGCCTGCGGCCGCGCTGCTGCGGTGCTGAGGCGACGCCTTCCCCCTGCCGAGGGGCGCGGAGCCATGTACATCAAGCAGGT\n",
            "AGAGGGCAAACGGGAAAAGATCAATGAGTTGTTGAAGTACATTGAAGAGCGCCTGCACACTCTAGAGGAGGAGAAGGAGGAGCTGGCC\n",
            "AGAGGGCAAACGGGAAAAGATCAATGAGTTGTTGAAGTACATTGAAGAGCGCCTGCACACTCTAGAGGAGGAGAAGGAGGAGCTGGCC\n",
            "AGAGGGCAAACGGGAAAAGATCAATGAGTTGTTGAAGTACATTGAAGAGCGCCTGCACACTCTAGAGGAGGAGAAGGAGGAGCTGGCC\n",
            "GACCAAACAATTGTAGATCCCTTCAGCTCCAAACATAATGTCATCGGT\n",
            "GACCAAACAATTGTAGATCCCTTCAGCTCCAAACATAATGTCATCGGT\n",
            "GACCAAACAATTGTAGATCCCTTCAGCTCCAAACATAATGTCATCGGT\n",
            "Exon file type summaries for the first gene from: ENSG00000108055-ENSMUSG00000024974\n",
            "\t0 exons with: No file exists\n",
            "\t8 exons with: Only Mapped\n",
            "\t48 exons with: Only nonintersecting\n",
            "\t18 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "ACTAAGTGGTGTATTAGTTTGCTAAGGCAGCCATAACAAAATACCATAGACTGG\n",
            "ACTAAGTGGTGTATTAGTTTGCTAAGGCAGCCATAACAAAATACCATAGACTGG\n",
            "ACTAAGTGGTGTATTAGTTTGCTAAGGCAGCCATAACAAAATACCATAGACTGG\n",
            "AAAGTTTTCTACATCTGGCATTGATGCCTCCAAGCTTTTGGATTGTGGTGCTCTGGGAACTAACACCCGGAAGTAGTCAGGGA\n",
            "AAAGTTTTCTACATCTGGCATTGATGCCTCCAAGCTTTTGGATTGTGGTGCTCTGGGAACTAACACCCGGAAGTAGTCAGGGA\n",
            "CTTATTGCTTCCTCATCATTGTTTAAATATTGTTGGCATGAATGCTACATGAATGATATGTACTTATGGTGTCATATAATGTGTCCCATTGTTAGTGATACTAGGTTTGATCACCTGCAGATCTCTCC\n",
            "CTTATTGCTTCCTCATCATTGTTTAAATATTGTTGGCATGAATGCTACATGAATGATATGTACTTATGGTGTCATATAATGTGTCCCATTGTTAGTGATACTAGGTTTGATCACCTGCAGATCTCTCC\n",
            "CTTATTGCTTCCTCATCATTGTTTAAATATTGTTGGCATGAATGCTACATGAATGATATGTACTTATGGTGTCATATAATGTGTCCCATTGTTAGTGATACTAGGTTTGATCACCTGCAGATCTCTCC\n",
            "Exon file type summaries for the second gene from: ENSG00000108055-ENSMUSG00000024974\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t3 exons with: Only nonintersecting\n",
            "\t26 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108055-ENSMUSG00000024974/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108055-ENSMUSG00000024974/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 20, Nonintersecting exons= 33, OTHER= 26\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 29, Nonintersecting exons= 3, OTHER= 1\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108055-ENSMUSG00000024974/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108055-ENSMUSG00000024974/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108055-ENSMUSG00000024974/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108055-ENSMUSG00000024974/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108055-ENSMUSG00000024974/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3213\tENSG00000108061-ENSMUSG00000024976\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108061-ENSMUSG00000024976\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108061\tchr10\t110919367\t111017307\t+\tENSG00000108061\tSHOC2\tprotein_coding\t20\t52\tgene\n",
            "\n",
            "ENSMUSG00000024976\tchr19\t53932737\t54021564\t+\tENSMUSG00000024976\tShoc2\tprotein_coding\t7\t26\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t52\t42\n",
            "\n",
            "Org2\t26\t22\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "CGGAGGAAAGGAAAAGGGCGGGGACAGGGAAGCTGGCGCCAAGGTTTCCCTCTGATTGGGTGACTGCTTTTGCTCTGGC\n",
            "CGGAGGAAAGGAAAAGGGCGGGGACAGGGAAGCTGGCGCCAAGGTTTCCCTCTGATTGGGTGACTGCTTTTGCTCTGGC\n",
            "CGGAGGAAAGGAAAAGGGCGGGGACAGGGAAGCTGGCGCCAAGGTTTCCCTCTGATTGGGTGACTGCTTTTGCTCTGGC\n",
            "AGGGACTCTAATGAGCTGACCAGGACCATTTTACCAGTTGAAATGAGTTCTCAGAAATGAGCATAGTGCTGTTAGAGTCACCATGTAACGGACGGATGTGACTCCAAGCGCACCACTTCTTCATGCCAACCCTCTTTGACTGGGAAGGATCTTTGTCTTTTCATCGTTGATCTGGATTGCTGGCGAGGAGGAGGAGTTCATGTAGTGTTTGAGTGAAGCTTGAGTCACCATGAGTAGTAGTTTAGGAAAAGAAAAAGACTCTAAAGAGAAAGACCCCAAAGTACCGTCTGCCAAGGAAAGAGAAAAGGAGTCAAAAGCCTCAGGAGGCTTTGGGAAAGAGAGCAAAGAAAAGGAACCTAAAGCCAAAGGGAAAGACGCCAAAGATGGAAAGAAGGAGTCCAGCGCTGCCCAGCCAGGTGTGGCTTTTTCAGTCGACAATACCATCAAACGGCCAAATCCAGCACCCGGCACTAGAAAAAAGTCCAGCAATGCTGAGGTCATTAAGGAGCTTAACAAATGCCGGGAGGAGAACTCAATGCGGTTGGACTTGTCCAAGAGGTCTATACATATACTGCCACCATCAGTCAAAGAGTTGACTCAACTCACAGAACTTTATTTATACAGTAACAAATTGCAGTCCCTCCCAGCAGAGGTGGGCTGTCTAGTCAATCTCATGACGCTGGCTCTCAGTGAGAATTCACTCACCAGTTTGCCTGACTCTCTTGATAACTTGAAGAAGCTGCGGATGCTTGATTTACGGCATAATAAACTGAGAGAAATTCCTTCAGTGGTGTATAGGCTAGACTCTCTCACCACTCTCTATCTTCGCTTTAATCGCATAACTACTGTGGAAAAGGACATCAAAAACCTGCCGAAGCTCAGCATGCTCAGCATCCGAGAGAACAAAATCAAGCAGCTGCCTGCTGAAATTGGT\n",
            "AGGGACTCTAATGAGCTGACCAGGACCATTTTACCAGTTGAAATGAGTTCTCAGAAATGAGCATAGTGCTGTTAGAGTCACCATGTAACGGACGGATGTGACTCCAAGCGCACCACTTCTTCATGCCAACCCTCTTTGACTGGGAAGGATCTTTGTCTTTTCATCGTTGATCTGGATTGCTGGCGAGGAGGAGGAGTTCATGTAGTGTTTGAGTGAAGCTTGAGTCACCATGAGTAGTAGTTTAGGAAAAGAAAAAGACTCTAAAGAGAAAGACCCCAAAGTACCGTCTGCCAAGGAAAGAGAAAAGGAGTCAAAAGCCTCAGGAGGCTTTGGGAAAGAGAGCAAAGAAAAGGAACCTAAAGCCAAAGGGAAAGACGCCAAAGATGGAAAGAAGGAGTCCAGCGCTGCCCAGCCAGGTGTGGCTTTTTCAGTCGACAATACCATCAAACGGCCAAATCCAGCACCCGGCACTAGAAAAAAGTCCAGCAATGCTGAGGTCATTAAGGAGCTTAACAAATGCCGGGAGGAGAACTCAATGCGGTTGGACTTGTCCAAGAGGTCTATACATATACTGCCACCATCAGTCAAAGAGTTGACTCAACTCACAGAACTTTATTTATACAGTAACAAATTGCAGTCCCTCCCAGCAGAGGTGGGCTGTCTAGTCAATCTCATGACGCTGGCTCTCAGTGAGAATTCACTCACCAGTTTGCCTGACTCTCTTGATAACTTGAAGAAGCTGCGGATGCTTGATTTACGGCATAATAAACTGAGAGAAATTCCTTCAGTGGTGTATAGGCTAGACTCTCTCACCACTCTCTATCTTCGCTTTAATCGCATAACTACTGTGGAAAAGGACATCAAAAACCTGCCGAAGCTCAGCATGCTCAGCATCCGAGAGAACAAAATCAAGCAGCTGCCTGCTGAAATTGGT\n",
            "AGGTGAATTATGTAACCTCATTACCCTGGATGTAGCTCACAATCAACTTGAACACCTTCCAAAGGAGATTGGAAACTGCACACAGATAACCAACCTTGACTTGCAGCACAATGACCTACTGGACCTCCCAGATACAATAGGT\n",
            "AGGTGAATTATGTAACCTCATTACCCTGGATGTAGCTCACAATCAACTTGAACACCTTCCAAAGGAGATTGGAAACTGCACACAGATAACCAACCTTGACTTGCAGCACAATGACCTACTGGACCTCCCAGATACAATAGGT\n",
            "AGGTGAATTATGTAACCTCATTACCCTGGATGTAGCTCACAATCAACTTGAACACCTTCCAAAGGAGATTGGAAACTGCACACAGATAACCAACCTTGACTTGCAGCACAATGACCTACTGGACCTCCCAGATACAATAGGT\n",
            "AGGAAACCTGTCCAGTTTAAATCGCCTTGGCCTGAGATACAATAGATTGTCAGCAATACCCAGATCATTAGCAAAATGCAGTGCACTTGAGGAGTTAAATTTAGAGAACAATAACATTTCTACTCTACCAGAGGT\n",
            "AGGAAACCTGTCCAGTTTAAATCGCCTTGGCCTGAGATACAATAGATTGTCAGCAATACCCAGATCATTAGCAAAATGCAGTGCACTTGAGGAGTTAAATTTAGAGAACAATAACATTTCTACTCTACCAGAGGT\n",
            "AGGAAACCTGTCCAGTTTAAATCGCCTTGGCCTGAGATACAATAGATTGTCAGCAATACCCAGATCATTAGCAAAATGCAGTGCACTTGAGGAGTTAAATTTAGAGAACAATAACATTTCTACTCTACCAGAGGT\n",
            "GCAGGGGAGCGTCGCTCGGGGAGCGGAGGACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "GCAGGGGAGCGTCGCTCGGGGAGCGGAGGACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "AGGTGAATTATGTAACCTCATTACCCTGGATGTAGCTCACAATCAACTTGAACACCTTCCAAAGGAGATTGGAAACTGCACACAGATAACCAACCTTGACTTGCAGCACAATGACCTACTGGACCTCCCAGATACAATAGGT\n",
            "AGGTGAATTATGTAACCTCATTACCCTGGATGTAGCTCACAATCAACTTGAACACCTTCCAAAGGAGATTGGAAACTGCACACAGATAACCAACCTTGACTTGCAGCACAATGACCTACTGGACCTCCCAGATACAATAGGT\n",
            "AGGTGAATTATGTAACCTCATTACCCTGGATGTAGCTCACAATCAACTTGAACACCTTCCAAAGGAGATTGGAAACTGCACACAGATAACCAACCTTGACTTGCAGCACAATGACCTACTGGACCTCCCAGATACAATAGGT\n",
            "AGGAAACCTGTCCAGTTTAAATCGCCTTGGCCTGAGATACAATAGATTGTCAGCAATACCCAGATCATTAGCAAAATGCAGTGCACTTGAGGAGTTAAATTTAGAGAACAATAACATTTCTACTCTACCAGAGGT\n",
            "AGGAAACCTGTCCAGTTTAAATCGCCTTGGCCTGAGATACAATAGATTGTCAGCAATACCCAGATCATTAGCAAAATGCAGTGCACTTGAGGAGTTAAATTTAGAGAACAATAACATTTCTACTCTACCAGAGGT\n",
            "AGGAAACCTGTCCAGTTTAAATCGCCTTGGCCTGAGATACAATAGATTGTCAGCAATACCCAGATCATTAGCAAAATGCAGTGCACTTGAGGAGTTAAATTTAGAGAACAATAACATTTCTACTCTACCAGAGGT\n",
            "CAGGGGAGCGTCGCTCGGGGAGCGGAGGACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "CAGGGGAGCGTCGCTCGGGGAGCGGAGGACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "GCTCGGGGAGCGGAGGACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "CTCGGGGAGCGGAGGACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "CGGGGAGCGGAGGACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "GGGAGCGGAGGACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "AGGGACTCTAATGAGCTGACCAGGACCATTTTACCAGTTGAAATGAGTTCTCAGAAATGAGCATAGTGCTGTTAGAGTCACCATGTAACGGACGGATGTGACTCCAAGCGCACCACTTCTTCATGCCAACCCTCTTTGACTGGGAAGGATCTTTGTCTTTTCATCGTTGATCTGGATTGCTGGCGAGGAGGAGGAGTTCATGTAGTGTTTGAGTGAAGCTTGAGTCACCATGAGTAGTAGTTTAGGAAAAGAAAAAGACTCTAAAGAGAAAGACCCCAAAGTACCGTCTGCCAAGGAAAGAGAAAAGGAGTCAAAAGCCTCAGGAGGCTTTGGGAAAGAGAGCAAAGAAAAGGAACCTAAAGCCAAAGGGAAAGACGCCAAAGATGGAAAGAAGGAGTCCAGCGCTGCCCAGCCAGGTGTGGCTTTTTCAGTCGACAATACCATCAAACGGCCAAATCCAGCACCCGGCACTAGAAAAAAGTCCAGCAATGCTGAGGTCATTAAGGAGCTTAACAAATGCCGGGAGGAGAACTCAATGCGGTTGGACTTGTCCAAGAGGTCTATACATATACTGCCACCATCAGTCAAAGAGTTGACTCAACTCACAGAACTTTATTTATACAGTAACAAATTGCAGTCCCTCCCAGCAGAGGTGGGCTGTCTAGTCAATCTCATGACGCTGGCTCTCAGTGAGAATTCACTCACCAGTTTGCCTGACTCTCTTGATAACTTGAAGAAGCTGCGGATGCTTGATTTACGGCATAATAAACTGAGAGAAATTCCTTCAGTGGTGTATAGGCTAGACTCTCTCACCACTCTCTATCTTCGCTTTAATCGCATAACTACTGTGGAAAAGGACATCAAAAACCTGCCGAAGCTCAGCATGCTCAGCATCCGAGAGAACAAAATCAAGCAGCTGCCTGCTGAAATTGGT\n",
            "AGGGACTCTAATGAGCTGACCAGGACCATTTTACCAGTTGAAATGAGTTCTCAGAAATGAGCATAGTGCTGTTAGAGTCACCATGTAACGGACGGATGTGACTCCAAGCGCACCACTTCTTCATGCCAACCCTCTTTGACTGGGAAGGATCTTTGTCTTTTCATCGTTGATCTGGATTGCTGGCGAGGAGGAGGAGTTCATGTAGTGTTTGAGTGAAGCTTGAGTCACCATGAGTAGTAGTTTAGGAAAAGAAAAAGACTCTAAAGAGAAAGACCCCAAAGTACCGTCTGCCAAGGAAAGAGAAAAGGAGTCAAAAGCCTCAGGAGGCTTTGGGAAAGAGAGCAAAGAAAAGGAACCTAAAGCCAAAGGGAAAGACGCCAAAGATGGAAAGAAGGAGTCCAGCGCTGCCCAGCCAGGTGTGGCTTTTTCAGTCGACAATACCATCAAACGGCCAAATCCAGCACCCGGCACTAGAAAAAAGTCCAGCAATGCTGAGGTCATTAAGGAGCTTAACAAATGCCGGGAGGAGAACTCAATGCGGTTGGACTTGTCCAAGAGGTCTATACATATACTGCCACCATCAGTCAAAGAGTTGACTCAACTCACAGAACTTTATTTATACAGTAACAAATTGCAGTCCCTCCCAGCAGAGGTGGGCTGTCTAGTCAATCTCATGACGCTGGCTCTCAGTGAGAATTCACTCACCAGTTTGCCTGACTCTCTTGATAACTTGAAGAAGCTGCGGATGCTTGATTTACGGCATAATAAACTGAGAGAAATTCCTTCAGTGGTGTATAGGCTAGACTCTCTCACCACTCTCTATCTTCGCTTTAATCGCATAACTACTGTGGAAAAGGACATCAAAAACCTGCCGAAGCTCAGCATGCTCAGCATCCGAGAGAACAAAATCAAGCAGCTGCCTGCTGAAATTGGT\n",
            "GAGGACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "GACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "ACGAGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "AGGAGGAGGAAGGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "GGAGGTGAGCAGGGATGGTGGAGTCCGGGCTCCGGACGGC\n",
            "GGATGTCAGAAGTTCTAGCATCTACCCGGAAGGATTGCCGAATCAAAAACTTAGCTGTTTTTCAGAATCGAATGGT\n",
            "GGGGCGCGGGCGCGGGCCCGGGGCGGCGAGTCAGTCGCGCTCCGGCTGCTCCGGTCTCTCGGCGCCTCCCGGCCGGGACTGCGGGGCTGCCCGGGTCGCGAGGACGCTTGGCGCCGGGCGGGCGGGCGGGCGGGAGGGCGGGCGCGGCCGTAGGGCTGTGGGGT\n",
            "AGGGACTCTAATGAGCTGACCAGGACCATTTTACCAGTTGAAATGAGTTCTCAGAAATGAGCATAGTGCTGTTAGAGTCACCATGTAACGGACGGATGTGACTCCAAGCGCACCACTTCTTCATGCCAACCCTCTTTGACTGGGAAGGATCTTTGTCTTTTCATCGTTGATCTGGATTGCTGGCGAGGAGGAGGAGTTCATGTAGTGTTTGAGTGAAGCTTGAGTCACCATGAGTAGTAGTTTAGGAAAAGAAAAAGACTCTAAAGAGAAAGACCCCAAAGTACCGTCTGCCAAGGAAAGAGAAAAGGAGTCAAAAGCCTCAGGAGGCTTTGGGAAAGAGAGCAAAGAAAAGGAACCTAAAGCCAAAGGGAAAGACGCCAAAGATGGAAAGAAGGAGTCCAGCGCTGCCCAGCCAGGTGTGGCTTTTTCAGTCGACAATACCATCAAACGGCCAAATCCAGCACCCGGCACTAGAAAAAAGTCCAGCAATGCTGAGGTCATTAAGGAGCTTAACAAATGCCGGGAGGAGAACTCAATGCGGTTGGACTTGTCCAAGAGGTCTATACATATACTGCCACCATCAGTCAAAGAGTTGACTCAAC\n",
            "AGGGACTCTAATGAGCTGACCAGGACCATTTTACCAGTTGAAATGAGTTCTCAGAAATGAGCATAGTGCTGTTAGAGTCACCATGTAACGGACGGATGTGACTCCAAGCGCACCACTTCTTCATGCCAACCCTCTTTGACTGGGAAGGATCTTTGTCTTTTCATCGTTGATCTGGATTGCTGGCGAGGAGGAGGAGTTCATGTAGTGTTTGAGTGAAGCTTGAGTCACCATGAGTAGTAGTTTAGGAAAAGAAAAAGACTCTAAAGAGAAAGACCCCAAAGTACCGTCTGCCAAGGAAAGAGAAAAGGAGTCAAAAGCCTCAGGAGGCTTTGGGAAAGAGAGCAAAGAAAAGGAACCTAAAGCCAAAGGGAAAGACGCCAAAGATGGAAAGAAGGAGTCCAGCGCTGCCCAGCCAGGTGTGGCTTTTTCAGTCGACAATACCATCAAACGGCCAAATCCAGCACCCGGCACTAGAAAAAAGTCCAGCAATGCTGAGGTCATTAAGGAGCTTAACAAATGCCGGGAGGAGAACTCAATGCGGTTGGACTTGTCCAAGAGGTCTATACATATACTGCCACCATCAGTCAAAGAGTTGACTCAAC\n",
            "AGGAAAC\n",
            "AGGAAAC\n",
            "AGGAAAC\n",
            "GAACAAAATCAAGCAGCTGCCTGCTGAAATTGGT\n",
            "GAACAAAATCAAGCAGCTGCCTGCTGAAATTGGT\n",
            "GAACAAAATCAAGCAGCTGCCTGCTGAAATTGGT\n",
            "AGGTGAATTATGTAACCTCATTACCCTGGATGTAGCTCACAATCAACTTGAACACCTTCCAAAGGAGATTGGAAACTGCACACAGATAACCAACCTTGACTTGCAGCACAATGACCTACTGGACCTCCCAGATACAATAGGTATGTGGGGAGAAGGGAGGTACTGAGATCTGTGACTCATGGTTATGAAAAGGACGATTTTTGGTCTATTTTGGTAATACAAAATCAGAATTATCAAAGTAGTGATAAAACTTTTAAAGTAGTCTTTTAAATCACTGTT\n",
            "Exon file type summaries for the first gene from: ENSG00000108061-ENSMUSG00000024976\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t18 exons with: Only nonintersecting\n",
            "\t12 exons with: Only unmapped\n",
            "\t5 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t17 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "ACTAGGAAGAAAGTAAGTGTGCTGTATTCTACTTTAGCACCATGTAAATCTTTCTCATTTCATTTTGGGAACTGACAGTAAAGGTATTAAGGAAAAACTATATGGCTGGTTTTAAAATTCAGCTACCCCACCCAGTTCAGTTATAAAAAGAAAGATGATAAGTGATACAGAATG\n",
            "ACTAGGAAGAAAGTAAGTGTGCTGTATTCTACTTTAGCACCATGTAAATCTTTCTCATTTCATTTTGGGAACTGACAGTAAAGGTATTAAGGAAAAACT\n",
            "AAGCTCAGTACTATTAACCTATTTTTATTTTATATTTTAGTATTATAGTATTATGGTATTA\n",
            "ACTAGGAAGAAAGTAAGTGTGCTGTATTCTACTTTAGCACCATGTAAATCTTTCTCATTTCATTTTGGGAACTGACAGTAAAGGTATTAAGGAAAAACTATATGGCTGGTTTTAAAATTCAGCTACCCCACCCAGTTCAGTTATAAAAAGAAAGATGATAAGTGATACAGAATG\n",
            "Exon file type summaries for the second gene from: ENSG00000108061-ENSMUSG00000024976\n",
            "\t0 exons with: No file exists\n",
            "\t3 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t18 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108061-ENSMUSG00000024976/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108061-ENSMUSG00000024976/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 17, Nonintersecting exons= 24, OTHER= 11\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 19, Nonintersecting exons= 4, OTHER= 3\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108061-ENSMUSG00000024976/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108061-ENSMUSG00000024976/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108061-ENSMUSG00000024976/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108061-ENSMUSG00000024976/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108061-ENSMUSG00000024976/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3214\tENSG00000108064-ENSMUSG00000003923\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108064-ENSMUSG00000003923\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108064\tchr10\t58385345\t58399220\t+\tENSG00000108064\tTFAM\tprotein_coding\t4\t19\tgene\n",
            "\n",
            "ENSMUSG00000003923\tchr10\t71061294\t71074110\t-\tENSMUSG00000003923\tTfam\tprotein_coding\t5\t19\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t19\t14\n",
            "\n",
            "Org2\t19\t15\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108064-ENSMUSG00000003923\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t9 exons with: Only nonintersecting\n",
            "\t2 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108064-ENSMUSG00000003923\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t10 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t8 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108064-ENSMUSG00000003923/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108064-ENSMUSG00000003923/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 10, Nonintersecting exons= 0, OTHER= 9\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 18, Nonintersecting exons= 0, OTHER= 1\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108064-ENSMUSG00000003923/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108064-ENSMUSG00000003923/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108064-ENSMUSG00000003923/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108064-ENSMUSG00000003923/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108064-ENSMUSG00000003923/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3215\tENSG00000108091-ENSMUSG00000048701\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108091-ENSMUSG00000048701\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108091\tchr10\t59788747\t59906556\t-\tENSG00000108091\tCCDC6\tprotein_coding\t3\t14\tgene\n",
            "\n",
            "ENSMUSG00000048701\tchr10\t69932951\t70029030\t+\tENSMUSG00000048701\tCcdc6\tprotein_coding\t4\t14\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t14\t13\n",
            "\n",
            "Org2\t14\t14\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108091-ENSMUSG00000048701\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t9 exons with: Only nonintersecting\n",
            "\t2 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108091-ENSMUSG00000048701\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t7 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108091-ENSMUSG00000048701/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108091-ENSMUSG00000048701/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 5, Nonintersecting exons= 0, OTHER= 9\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 12, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108091-ENSMUSG00000048701/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108091-ENSMUSG00000048701/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108091-ENSMUSG00000048701/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108091-ENSMUSG00000048701/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108091-ENSMUSG00000048701/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3216\tENSG00000108094-ENSMUSG00000024231\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108094-ENSMUSG00000024231\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108094\tchr10\t35008504\t35127006\t-\tENSG00000108094\tCUL2\tprotein_coding\t49\t134\tgene\n",
            "\n",
            "ENSMUSG00000024231\tchr18\t3382988\t3436377\t+\tENSMUSG00000024231\tCul2\tprotein_coding\t7\t51\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t134\t106\n",
            "\n",
            "Org2\t51\t35\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "AGACTGCTGACCCTTAAGTCAGGCAACCCCTGGACCAGATCCTGGGCCTCCCGGATCCACTGTCGTCTGCCCCTCCCTCGGCCTCATTTTCCTCCTTAGGAGGGAAGGT\n",
            "AGACTGCTGACCCTTAAGTCAGGCAACCCCTGGACCAGATCCTGGGCCTCCCGGATCCACTGTCGTCTGCCCCTCCCTCGGCCTCATTTTCCTCCTTAGGAGGGAAGGT\n",
            "AGACAGTGAAGATTGATAGCTATGGGGAGGAGTATTCTACGTGCATGCTACATACACCTCTTGCCTCGGCACTTATGGGCTGGTGAAATAAGAGCCTCAGATTCTCTCCAGCACTTGTTTACATACTAGAGTCTTAAACCCTTCCCCACCATTACGTGGTGCGTGCAAGTAGAACATCTTT\n",
            "AGACAGTGAAGATTGATAGCTATGGGGAGGAGTATTCTACGTGCATGCTACATACACCTCTTGCCTCGGCACTTATGGGCTGGTGAAATAAGAGCCTCAGATTCTCTCCAGCACTTGTTTACATACTAGAGTCTTAAACCCTTCCCCACCATTACGTGGTGCGTGCAAGTAGAACATCTTT\n",
            "AGCGGGCTGAGAGCTGGAGGCCTGGCAAGATGGTGTCCTGTGAGTGACCTCGGCTCCCCAGACTGCTGACCCTTAAGTCAGGCAACCCCTGGACCAGATCCTGGGCCTCCCGGATCCACTGTCGTCTGCCCCTCCCTCGGCCTCATTTTCCTCCTTAGGAGGGAAGGT\n",
            "AGCGGGCTGAGAGCTGGAGGCCTGGCAAGATGGTGTCCTGTGAGTGACCTCGGCTCCCCAGACTGCTGACCCTTAAGTCAGGCAACCCCTGGACCAGATCCTGGGCCTCCCGGATCCACTGTCGTCTGCCCCTCCCTCGGCCTCATTTTCCTCCTTAGGAGGGAAGGT\n",
            "AGACTGCTGACCCTTAAGTCAGGCAACCCCTGGACCAGATCCTGGGCCTCCCGGATCCACTGTCGTCTGCCCCTCCCTCGGCCTCATTTTCCTCCTTAGGAGGGAAGGTGAGAGT\n",
            "AGACTGCTGACCCTTAAGTCAGGCAACCCCTGGACCAGATCCTGGGCCTCCCGGATCCACTGTCGTCTGCCCCTCCCTCGGCCTCATTTTCCTCCTTAGGAGGGAAGGTGAGAGT\n",
            "Exon file type summaries for the first gene from: ENSG00000108094-ENSMUSG00000024231\n",
            "\t0 exons with: No file exists\n",
            "\t56 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t35 exons with: Only unmapped\n",
            "\t39 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108094-ENSMUSG00000024231\n",
            "\t0 exons with: No file exists\n",
            "\t41 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t2 exons with: Only unmapped\n",
            "\t8 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108094-ENSMUSG00000024231/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108094-ENSMUSG00000024231/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 57, Unmapped exons= 63, Nonintersecting exons= 4, OTHER= 10\n",
            "\tFor org2: Mapped exons= 40, Unmapped exons= 8, Nonintersecting exons= 0, OTHER= 3\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108094-ENSMUSG00000024231/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108094-ENSMUSG00000024231/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108094-ENSMUSG00000024231/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t4 pairs with Condition1: Unique winner pair\n",
            "\t3 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108094-ENSMUSG00000024231/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108094-ENSMUSG00000024231/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3217\tENSG00000108100-ENSMUSG00000024286\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108100-ENSMUSG00000024286\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108100\tchr10\t35247025\t35572669\t+\tENSG00000108100\tCCNY\tprotein_coding\t10\t42\tgene\n",
            "\n",
            "ENSMUSG00000024286\tchr18\t9312304\t9450154\t-\tENSMUSG00000024286\tCcny\tprotein_coding\t5\t23\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t42\t33\n",
            "\n",
            "Org2\t23\t17\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108100-ENSMUSG00000024286\n",
            "\t0 exons with: No file exists\n",
            "\t22 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t12 exons with: Only unmapped\n",
            "\t7 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108100-ENSMUSG00000024286\n",
            "\t0 exons with: No file exists\n",
            "\t15 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t1 exons with: Only unmapped\n",
            "\t7 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108100-ENSMUSG00000024286/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108100-ENSMUSG00000024286/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 21, Unmapped exons= 15, Nonintersecting exons= 0, OTHER= 6\n",
            "\tFor org2: Mapped exons= 19, Unmapped exons= 4, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108100-ENSMUSG00000024286/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108100-ENSMUSG00000024286/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108100-ENSMUSG00000024286/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t3 pairs with Condition1: Unique winner pair\n",
            "\t1 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108100-ENSMUSG00000024286/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108100-ENSMUSG00000024286/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3218\tENSG00000108107-ENSMUSG00000030432\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108107-ENSMUSG00000030432\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108107\tchr19\t55385932\t55403250\t+\tENSG00000108107\tRPL28\tprotein_coding\t11\t24\tgene\n",
            "\n",
            "ENSMUSG00000030432\tchr7\t4795873\t4798065\t+\tENSMUSG00000030432\tRpl28\tprotein_coding\t5\t14\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t24\t21\n",
            "\n",
            "Org2\t14\t13\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108107-ENSMUSG00000030432\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t12 exons with: Only nonintersecting\n",
            "\t10 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108107-ENSMUSG00000030432\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t14 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108107-ENSMUSG00000030432/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108107-ENSMUSG00000030432/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 12, Nonintersecting exons= 0, OTHER= 12\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 14, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108107-ENSMUSG00000030432/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108107-ENSMUSG00000030432/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108107-ENSMUSG00000030432/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108107-ENSMUSG00000030432/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108107-ENSMUSG00000030432/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3219\tENSG00000108175-ENSMUSG00000007817\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108175-ENSMUSG00000007817\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108175\tchr10\t79068966\t79316519\t+\tENSG00000108175\tZMIZ1\tprotein_coding\t3\t33\tgene\n",
            "\n",
            "ENSMUSG00000007817\tchr14\t25459609\t25667167\t+\tENSMUSG00000007817\tZmiz1\tprotein_coding\t8\t46\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t33\t29\n",
            "\n",
            "Org2\t46\t39\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "AGGACTCAGACAGCATAGAATCTACTCAGCATCTCCTTCGCCACAGCCTCCCTTCCCCAACATCACATTAGCCTTGAGTTGTGATCGACTGGCAGGAGCCCTAGCAGGT\n",
            "AGGAGGATGGACCTTGAGTGAAGTGGAAGAGCTGTGAGTGGCTGGCCAGCGCAGGAACACTAGTGTCTCTCGGCTCACTGAGGT\n",
            "AGGAGGATGGACCTTGAGTGAAGTGGAAGAGCTGTGAGTGGCTGGCCAGCGCAGGAACACTAGTGTCTCTCGGCTCACTGAGGT\n",
            "AGGCTGGACCACGTTCATGGCTCTCGGGTAGGACCCAGCGAAACGGCCAGAATGAATTCTATGGACAGGCACATCCAGCAGACCAATGACCGTCTGCAGTGCATCAAGCAGGT\n",
            "AGGCTGGACCACGTTCATGGCTCTCGGGTAGGACCCAGCGAAACGGCCAGAATGAATTCTATGGACAGGCACATCCAGCAGACCAATGACCGTCTGCAGTGCATCAAGCAGGT\n",
            "AGGCTGGACCACGTTCATGGCTCTCGGGTAGGACCCAGCGAAACGGCCAGAATGAATTCTATGGACAGGCACATCCAGCAGACCAATGACCGTCTGCAGTGCATCAAGCAGGT\n",
            "AGCACTTACAGAACCCCGCCAACTTCCACAATGCTGCTACAGAGCTGCTGGATTGGTGTGGAGACCCTCGAGCCTTCCAGAGGCCCTTTGAGCAGAGCCTCATGGGCTGCCTGACGGT\n",
            "AGCACTTACAGAACCCCGCCAACTTCCACAATGCTGCTACAGAGCTGCTGGATTGGTGTGGAGACCCTCGAGCCTTCCAGAGGCCCTTTGAGCAGAGCCTCATGGGCTGCCTGACGGT\n",
            "AGCACTTACAGAACCCCGCCAACTTCCACAATGCTGCTACAGAGCTGCTGGATTGGTGTGGAGACCCTCGAGCCTTCCAGAGGCCCTTTGAGCAGAGCCTCATGGGCTGCCTGACGGT\n",
            "AGGTTGTCAGCCGTGTGGCTGCCCAACAAGGGTTTGACCTGGACCTGGGCTACAGACTCTTGGCTGTGTGCGCCGCAAATCGAGACAAGTTCACCCCCAAGTCTGCTGGT\n",
            "AGGTTGTCAGCCGTGTGGCTGCCCAACAAGGGTTTGACCTGGACCTGGGCTACAGACTCTTGGCTGTGTGCGCCGCAAATCGAGACAAGTTCACCCCCAAGTCTGCTGGT\n",
            "AGGTTGTCAGCCGTGTGGCTGCCCAACAAGGGTTTGACCTGGACCTGGGCTACAGACTCTTGGCTGTGTGCGCCGCAAATCGAGACAAGTTCACCCCCAAGTCTGCTGGT\n",
            "AGCCCTGCTGTCCTCCTGGTGTGAAGAACTTGGCCGCCTGCTGCTGCTGCGACATCAGAAGAGCCGCCAGAACGACCCCCCTGGGAAACTGCCCATGCAGCCCCCCCTCAGCTCCATGAGCTCCATGAAACCCACTCTGTCGCACAGGT\n",
            "AGCCCTGCTGTCCTCCTGGTGTGAAGAACTTGGCCGCCTGCTGCTGCTGCGACATCAGAAGAGCCGCCAGAACGACCCCCCTGGGAAACTGCCCATGCAGCCCCCCCTCAGCTCCATGAGCTCCATGAAACCCACTCTGTCGCACAGGT\n",
            "AGCCCTGCTGTCCTCCTGGTGTGAAGAACTTGGCCGCCTGCTGCTGCTGCGACATCAGAAGAGCCGCCAGAACGACCCCCCTGGGAAACTGCCCATGCAGCCCCCCCTCAGCTCCATGAGCTCCATGAAACCCACTCTGTCGCACAGGT\n",
            "AGTGATGGGTCATTTCCTTATGACTCTGTCCCTTGGCAGCAGAACACCAATCAGCCTCCCGGCTCCCTCTCCGTGGTCACCACGGTGTGGGGAGTGACCAACACATCCCAGAGTCAGGT\n",
            "AGTGATGGGTCATTTCCTTATGACTCTGTCCCTTGGCAGCAGAACACCAATCAGCCTCCCGGCTCCCTCTCCGTGGTCACCACGGTGTGGGGAGTGACCAACACATCCCAGAGTCAGGT\n",
            "AGTGATGGGTCATTTCCTTATGACTCTGTCCCTTGGCAGCAGAACACCAATCAGCCTCCCGGCTCCCTCTCCGTGGTCACCACGGTGTGGGGAGTGACCAACACATCCCAGAGTCAGGT\n",
            "AGGTCCTCGGGAACCCTATGGCCAATGCCAACAACCCTATGAATCCAGGTGGCAACCCCATGGCATCAGGCATGAGCACCAGCAACCCCGGCATCAACTCCCCACAGTTCGCAGGGCAACAGCAACAGTTCTCCACCAAAGCTGGCCCTGCACAGCCCTATATCCAGCCCAACATGTACGGCCGACCTGGCTACCCTGGTAGCGGGGGCTTTGGGGCCAGGT\n",
            "AGGTCCTCGGGAACCCTATGGCCAATGCCAACAACCCTATGAATCCAGGTGGCAACCCCATGGCATCAGGCATGAGCACCAGCAACCCCGGCATCAACTCCCCACAGTTCGCAGGGCAACAGCAACAGTTCTCCACCAAAGCTGGCCCTGCACAGCCCTATATCCAGCCCAACATGTACGGCCGACCTGGCTACCCTGGTAGCGGGGGCTTTGGGGCCAGGT\n",
            "AGGTCCTCGGGAACCCTATGGCCAATGCCAACAACCCTATGAATCCAGGTGGCAACCCCATGGCATCAGGCATGAGCACCAGCAACCCCGGCATCAACTCCCCACAGTTCGCAGGGCAACAGCAACAGTTCTCCACCAAAGCTGGCCCTGCACAGCCCTATATCCAGCCCAACATGTACGGCCGACCTGGCTACCCTGGTAGCGGGGGCTTTGGGGCCAGGT\n",
            "AGTTACCCTGGGGGTCCTAGTGCCCCAGCAGGCATGGGCATCCCTCCACATACTCGGCCTCCTGCTGACTTCACCCAGCCAGCGGCCGCTGCTGCAGCAGCTGCAGTAGCAGCCGCCGCTGCCACAGCCACAGCCACAGCCACAGCTACAGTGGCAGCCTTGCAAGAGACACAGAACAAGGATATAAACCAGTATGGACCGGT\n",
            "AGTTACCCTGGGGGTCCTAGTGCCCCAGCAGGCATGGGCATCCCTCCACATACTCGGCCTCCTGCTGACTTCACCCAGCCAGCGGCCGCTGCTGCAGCAGCTGCAGTAGCAGCCGCCGCTGCCACAGCCACAGCCACAGCCACAGCTACAGTGGCAGCCTTGCAAGAGACACAGAACAAGGATATAAACCAGTATGGACCGGT\n",
            "AGTTACCCTGGGGGTCCTAGTGCCCCAGCAGGCATGGGCATCCCTCCACATACTCGGCCTCCTGCTGACTTCACCCAGCCAGCGGCCGCTGCTGCAGCAGCTGCAGTAGCAGCCGCCGCTGCCACAGCCACAGCCACAGCCACAGCTACAGTGGCAGCCTTGCAAGAGACACAGAACAAGGATATAAACCAGTATGGACCGGT\n",
            "AGATGGGTCCCACCCAGGCGTATAACAGCCAATTCATGAACCAACCCGGGCCTCGGGGGCCTGCCTCCATGGGGGGCAGCCTGAACCCTGCCGGCATGGCAGCTGGCATGACACCCTCGGGCATGAGCGGCCCTCCCATGGGCATGAACCAGCCCCGACCACCCGGCATCAGCCCCTTTGGCACACACGGGCAAAGGATGCCCCAGCAGACCTACCCAGGCCCCCGGCCCCAGTCCCTTCCTATTCAGAGCATAAAGAGGCCATACCCGGGAGAGGT\n",
            "AGATGGGTCCCACCCAGGCGTATAACAGCCAATTCATGAACCAACCCGGGCCTCGGGGGCCTGCCTCCATGGGGGGCAGCCTGAACCCTGCCGGCATGGCAGCTGGCATGACACCCTCGGGCATGAGCGGCCCTCCCATGGGCATGAACCAGCCCCGACCACCCGGCATCAGCCCCTTTGGCACACACGGGCAAAGGATGCCCCAGCAGACCTACCCAGGCCCCCGGCCCCAGTCCCTTCCTATTCAGAGCATAAAGAGGCCATACCCGGGAGAGGT\n",
            "AGATGGGTCCCACCCAGGCGTATAACAGCCAATTCATGAACCAACCCGGGCCTCGGGGGCCTGCCTCCATGGGGGGCAGCCTGAACCCTGCCGGCATGGCAGCTGGCATGACACCCTCGGGCATGAGCGGCCCTCCCATGGGCATGAACCAGCCCCGACCACCCGGCATCAGCCCCTTTGGCACACACGGGCAAAGGATGCCCCAGCAGACCTACCCAGGCCCCCGGCCCCAGTCCCTTCCTATTCAGAGCATAAAGAGGCCATACCCGGGAGAGGT\n",
            "AGCCAGAACAGTTTAATGGACAGAACAACACCTTCTCCTCCGGAAGCAGCTACAGCAGCTACAGCCAAGGGAGCGTCAACAGGGT\n",
            "AGCCAGAACAGTTTAATGGACAGAACAACACCTTCTCCTCCGGAAGCAGCTACAGCAGCTACAGCCAAGGGAGCGTCAACAGGGT\n",
            "AGCCAGAACAGTTTAATGGACAGAACAACACCTTCTCCTCCGGAAGCAGCTACAGCAGCTACAGCCAAGGGAGCGTCAACAGGGT\n",
            "AGCCTCCCAGGCCGGTTCCTGTGGCGAATTACCCCCACTCACCTGTTCCAGGGAACCCCACTCCCCCCATGACGCCCGGGAGCAGCATACCTCCCTATCTGTCCCCCAGCCAAGATGTCAAACCACCTTTCCCACCTGATATCAAGCCAAATATGAGTGCTCTGCCACCACCCCCAGGT\n",
            "AGCCTCCCAGGCCGGTTCCTGTGGCGAATTACCCCCACTCACCTGTTCCAGGGAACCCCACTCCCCCCATGACGCCCGGGAGCAGCATACCTCCCTATCTGTCCCCCAGCCAAGATGTCAAACCACCTTTCCCACCTGATATCAAGCCAAATATGAGTGCTCTGCCACCACCCCCAGGT\n",
            "AGCCTCCCAGGCCGGTTCCTGTGGCGAATTACCCCCACTCACCTGTTCCAGGGAACCCCACTCCCCCCATGACGCCCGGGAGCAGCATACCTCCCTATCTGTCCCCCAGCCAAGATGTCAAACCACCTTTCCCACCTGATATCAAGCCAAATATGAGTGCTCTGCCACCACCCCCAGGT\n",
            "AGCCAACCACAATGACGAGCTACGGCTCACATTCCCCGTGCGGGATGGCGTGGTTCTGGAGCCCTTCCGCCTGGAGCACAACCTTGCCGTCAGCAACCACGTGTTCCACCTGCGGCCCACGGTTCATCAGACGCTGATGTGGAGGT\n",
            "AGCCAACCACAATGACGAGCTACGGCTCACATTCCCCGTGCGGGATGGCGTGGTTCTGGAGCCCTTCCGCCTGGAGCACAACCTTGCCGTCAGCAACCACGTGTTCCACCTGCGGCCCACGGTTCATCAGACGCTGATGTGGAGGT\n",
            "AGCCAACCACAATGACGAGCTACGGCTCACATTCCCCGTGCGGGATGGCGTGGTTCTGGAGCCCTTCCGCCTGGAGCACAACCTTGCCGTCAGCAACCACGTGTTCCACCTGCGGCCCACGGTTCATCAGACGCTGATGTGGAGGT\n",
            "AGGTCAGACCTGGAGCTGCAGTTCAAGTGCTACCACCATGAAGATCGACAGATGAACACCAACTGGCCAGCCTCGGTTCAGGTCAGCGTCAACGCCACACCCCTCACGATCGAGCGGGGAGACAACAAGACCTCCCACAAGCCCCTGCACCTCAAGCATGTATGCCAGCCTGGCCGGAACACCATCCAGATCACTGTCACCGCCTGCTGCTGTGT\n",
            "AGGTCAGACCTGGAGCTGCAGTTCAAGTGCTACCACCATGAAGATCGACAGATGAACACCAACTGGCCAGCCTCGGTTCAGGTCAGCGTCAACGCCACACCCCTCACGATCGAGCGGGGAGACAACAAGACCTCCCACAAGCCCCTGCACCTCAAGCATGTATGCCAGCCTGGCCGGAACACCATCCAGATCACTGTCACCGCCTGCTGCTGTGT\n",
            "AGGTCAGACCTGGAGCTGCAGTTCAAGTGCTACCACCATGAAGATCGACAGATGAACACCAACTGGCCAGCCTCGGTTCAGGTCAGCGTCAACGCCACACCCCTCACGATCGAGCGGGGAGACAACAAGACCTCCCACAAGCCCCTGCACCTCAAGCATGTATGCCAGCCTGGCCGGAACACCATCCAGATCACTGTCACCGCCTGCTGCTGTGT\n",
            "AGTCCCACCTCTTTGTGCTGCAGCTGGTACACCGGCCCTCTGTGCGCTCCGTGCTGCAGGGCCTCCTCAAGAAGCGCCTACTGCCCGCTGAGCACTGCATCACGAAAAGT\n",
            "AGTCCCACCTCTTTGTGCTGCAGCTGGTACACCGGCCCTCTGTGCGCTCCGTGCTGCAGGGCCTCCTCAAGAAGCGCCTACTGCCCGCTGAGCACTGCATCACGAAAAGT\n",
            "AGTCCCACCTCTTTGTGCTGCAGCTGGTACACCGGCCCTCTGTGCGCTCCGTGCTGCAGGGCCTCCTCAAGAAGCGCCTACTGCCCGCTGAGCACTGCATCACGAAAAGT\n",
            "AGTCAAGCGGAATTTCAGCAGTGTGGCTGCCTCCTCGGGCAACACAACTCTCAATGGGGAGGATGGCGTGGAGCAGACCGCCATCAAGGTGTCTCTGAAGTGCCCCATCACATTCCGGCGCATACAGCTGCCTGCTCGAGGCCACGATTGCAAGCATGTGCAGGT\n",
            "AGTCAAGCGGAATTTCAGCAGTGTGGCTGCCTCCTCGGGCAACACAACTCTCAATGGGGAGGATGGCGTGGAGCAGACCGCCATCAAGGTGTCTCTGAAGTGCCCCATCACATTCCGGCGCATACAGCTGCCTGCTCGAGGCCACGATTGCAAGCATGTGCAGGT\n",
            "AGTCAAGCGGAATTTCAGCAGTGTGGCTGCCTCCTCGGGCAACACAACTCTCAATGGGGAGGATGGCGTGGAGCAGACCGCCATCAAGGTGTCTCTGAAGTGCCCCATCACATTCCGGCGCATACAGCTGCCTGCTCGAGGCCACGATTGCAAGCATGTGCAGGT\n",
            "AGTGCTTTGACCTGGAGTCATACCTCCAACTGAATTGTGAGCGAGGGACCTGGAGGTGTCCCGTGTGCAAGT\n",
            "AGTGCTTTGACCTGGAGTCATACCTCCAACTGAATTGTGAGCGAGGGACCTGGAGGTGTCCCGTGTGCAAGT\n",
            "AGTGCTTTGACCTGGAGTCATACCTCCAACTGAATTGTGAGCGAGGGACCTGGAGGTGTCCCGTGTGCAAGT\n",
            "AGTAAAACTGCTCTGCTCGAGGGTCTGGAAGTGGATCAGTACATGTGGGGGATCCTGAACGCCATCCAACAGT\n",
            "AGTAAAACTGCTCTGCTCGAGGGTCTGGAAGTGGATCAGTACATGTGGGGGATCCTGAACGCCATCCAACAGT\n",
            "AGTAAAACTGCTCTGCTCGAGGGTCTGGAAGTGGATCAGTACATGTGGGGGATCCTGAACGCCATCCAACAGT\n",
            "AGGAAACAACTACCAGGGTCATGGCAACTTTGACTTCCCCCATGGGAATCCCGGAGGGACATCCATGAACGACTTCATGCACGGTCCCCCCCAGCTCTCGCACCCACCGGACATGCCCAACAACATGGCCGCCCTCGAGAAACCCCTCAGTCACCCCATGCAGGAAACTGT\n",
            "AGGAAACAACTACCAGGGTCATGGCAACTTTGACTTCCCCCATGGGAATCCCGGAGGGACATCCATGAACGACTTCATGCACGGTCCCCCCCAGCTCTCGCACCCACCGGACATGCCCAACAACATGGCCGCCCTCGAGAAACCCCTCAGTCACCCCATGCAGGAAACTGT\n",
            "AGGAAACAACTACCAGGGTCATGGCAACTTTGACTTCCCCCATGGGAATCCCGGAGGGACATCCATGAACGACTTCATGCACGGTCCCCCCCAGCTCTCGCACCCACCGGACATGCCCAACAACATGGCCGCCCTCGAGAAACCCCTCAGTCACCCCATGCAGGAAACTGT\n",
            "AGATGCCCCACGCTGGCAGTTCTGACCAGCCCCATCCCTCCATACAACAAGGTTTGCACGTACCACACCCCAGCAGCCAGGCAGGGCCTCCATTACATCACAGTGGGGCTCCTCCTCCTTCCCAGCCTCCCCGGCAGCCACCACAGGCCGCTCCCGGCAACCATCCACACAGCGACCTGACCTTTAACCCCTCCTCAGCCTTAGAGGGTCAGGCCGGAGCACAGGGAGCATCCGACATGCCGGAACCTTCACTGGATGT\n",
            "AGATGCCCCACGCTGGCAGTTCTGACCAGCCCCATCCCTCCATACAACAAGGTTTGCACGTACCACACCCCAGCAGCCAGGCAGGGCCTCCATTACATCACAGTGGGGCTCCTCCTCCTTCCCAGCCTCCCCGGCAGCCACCACAGGCCGCTCCCGGCAACCATCCACACAGCGACCTGACCTTTAACCCCTCCTCAGCCTTAGAGGGTCAGGCCGGAGCACAGGGAGCATCCGACATGCCGGAACCTTCACTGGATGT\n",
            "TGCGGTCGCCGCGGACTGGGCTTTAGCCTCCGGGGGCGCCAGGGCGGGACCGCGGACTGTGCAACTTCTAGGT\n",
            "TGCGGTCGCCGCGGACTGGGCTTTAGCCTCCGGGGGCGCCAGGGCGGGACCGCGGACTGTGCAACTTCTAGGT\n",
            "AGCCCTGCTGTCCTCCTGGTGTGAAGAACTTGGCCGCCTGCTGCTGCTGCGACATCAGAAGAGCCGCCAGAACGACCCCCCTGGGAAACTGCCCATGCAGCCCCCCCTCAGCTCCATGAGCTCCATGAAACCCACTCTGTCGCACAGGT\n",
            "AGCCCTGCTGTCCTCCTGGTGTGAAGAACTTGGCCGCCTGCTGCTGCTGCGACATCAGAAGAGCCGCCAGAACGACCCCCCTGGGAAACTGCCCATGCAGCCCCCCCTCAGCTCCATGAGCTCCATGAAACCCACTCTGTCGCACAGGT\n",
            "AGCCCTGCTGTCCTCCTGGTGTGAAGAACTTGGCCGCCTGCTGCTGCTGCGACATCAGAAGAGCCGCCAGAACGACCCCCCTGGGAAACTGCCCATGCAGCCCCCCCTCAGCTCCATGAGCTCCATGAAACCCACTCTGTCGCACAGGT\n",
            "AGTGATGGGTCATTTCCTTATGACTCTGTCCCTTGGCAGCAGAACACCAATCAGCCTCCCGGCTCCCTCTCCGTGGTCACCACGGTGTGGGGAGTGACCAACACATCCCAGAGTCAGGT\n",
            "AGTGATGGGTCATTTCCTTATGACTCTGTCCCTTGGCAGCAGAACACCAATCAGCCTCCCGGCTCCCTCTCCGTGGTCACCACGGTGTGGGGAGTGACCAACACATCCCAGAGTCAGGT\n",
            "AGTGATGGGTCATTTCCTTATGACTCTGTCCCTTGGCAGCAGAACACCAATCAGCCTCCCGGCTCCCTCTCCGTGGTCACCACGGTGTGGGGAGTGACCAACACATCCCAGAGTCAGGT\n",
            "AGGTCCTCGGGAACCCTATGGCCAATGCCAACAACCCTATGAATCCAGGTGGCAACCCCATGGCATCAGGCATGAGCACCAGCAACCCCGGCATCAACTCCCCACAGTTCGCAGGGCAACAGCAACAGTTC\n",
            "AGGTCCTCGGGAACCCTATGGCCAATGCCAACAACCCTATGAATCCAGGTGGCAACCCCATGGCATCAGGCATGAGCACCAGCAACCCCGGCATCAACTCCCCACAGTTCGCAGGGCAACAGCAACAGTTC\n",
            "AGGTCCTCGGGAACCCTATGGCCAATGCCAACAACCCTATGAATCCAGGTGGCAACCCCATGGCATCAGGCATGAGCACCAGCAACCCCGGCATCAACTCCCCACAGTTCGCAGGGCAACAGCAACAGTTC\n",
            "GCAGCCCTGCTGTCCTCCTGGTGTGAAGAACTTGGCCGCCTGCTGCTGCTGCGACATCAGAAGAGCCGCCAGAACGACCCCCCTGGGAAACTGCCCATGCAGCCCCCCCTCAGCTCCATGAGCTCCATGAAACCCACTCTGTCGCACAGGT\n",
            "GCAGCCCTGCTGTCCTCCTGGTGTGAAGAACTTGGCCGCCTGCTGCTGCTGCGACATCAGAAGAGCCGCCAGAACGACCCCCCTGGGAAACTGCCCATGCAGCCCCCCCTCAGCTCCATGAGCTCCATGAAACCCACTCTGTCGCACAGGT\n",
            "GCAGCCCTGCTGTCCTCCTGGTGTGAAGAACTTGGCCGCCTGCTGCTGCTGCGACATCAGAAGAGCCGCCAGAACGACCCCCCTGGGAAACTGCCCATGCAGCCCCCCCTCAGCTCCATGAGCTCCATGAAACCCACTCTGTCGCACAGGT\n",
            "AGGTCCTCGGGAACCCTATGGCCAATGCCAACAACCCTATGAATCCAGGTGGCAACCCCATGGCATCAGGCATGAGCACCAGCAACCCCGGCATCAACTCCCCACAGTTCGCAGGGCAACAGCAACAGTTCTCCACCAAAGCTGGCCCTGCACAGCCCTATATCCAGCCCAACATGTACGGCCGACCTGGCTACCCTGGTAGCGGGGGCTTTGGGGCCAGGT\n",
            "AGGTCCTCGGGAACCCTATGGCCAATGCCAACAACCCTATGAATCCAGGTGGCAACCCCATGGCATCAGGCATGAGCACCAGCAACCCCGGCATCAACTCCCCACAGTTCGCAGGGCAACAGCAACAGTTCTCCACCAAAGCTGGCCCTGCACAGCCCTATATCCAGCCCAACATGTACGGCCGACCTGGCTACCCTGGTAGCGGGGGCTTTGGGGCCAGGT\n",
            "AGGTCCTCGGGAACCCTATGGCCAATGCCAACAACCCTATGAATCCAGGTGGCAACCCCATGGCATCAGGCATGAGCACCAGCAACCCCGGCATCAACTCCCCACAGTTCGCAGGGCAACAGCAACAGTTCTCCACCAAAGCTGGCCCTGCACAGCCCTATATCCAGCCCAACATGTACGGCCGACCTGGCTACCCTGGTAGCGGGGGCTTTGGGGCCAGGT\n",
            "AGTTACCCTGGGGGTCCTAGTGCCCCAGCAGGCATGGGCATCCCTCCACATACTCGGCCTCCTGCTGACTTCACCCAGCCAGCGGCCGCTGCTGCAGCAGCTGCAGTAGCAGCCGCCGCTGCCACAGCCACAGCCACAGCCACAGCTACAGTGGCAGCCTTGCAAGAGACACAGAACAAGGATATAAACCAGTATGGACCGGT\n",
            "AGTTACCCTGGGGGTCCTAGTGCCCCAGCAGGCATGGGCATCCCTCCACATACTCGGCCTCCTGCTGACTTCACCCAGCCAGCGGCCGCTGCTGCAGCAGCTGCAGTAGCAGCCGCCGCTGCCACAGCCACAGCCACAGCCACAGCTACAGTGGCAGCCTTGCAAGAGACACAGAACAAGGATATAAACCAGTATGGACCGGT\n",
            "AGTTACCCTGGGGGTCCTAGTGCCCCAGCAGGCATGGGCATCCCTCCACATACTCGGCCTCCTGCTGACTTCACCCAGCCAGCGGCCGCTGCTGCAGCAGCTGCAGTAGCAGCCGCCGCTGCCACAGCCACAGCCACAGCCACAGCTACAGTGGCAGCCTTGCAAGAGACACAGAACAAGGATATAAACCAGTATGGACCGGT\n",
            "Exon file type summaries for the first gene from: ENSG00000108175-ENSMUSG00000007817\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t23 exons with: Only nonintersecting\n",
            "\t2 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "GCTCCCCGGGGACTCTCGGGGTGCAAGCGTGGGGATTGGGTGCTGGGGTTTTTCCCTTAAAGTGTCCCCCGGAGCGGGGCGACCGGCCGGCGGCGCGGGCTCCGCTCCGCTCTCCTTGGCGGCCGGGGGCCGAGGCCCGGCGGCCGGCGAGCTCCCGGCTGCGCGGAGCCGGCTTGGGCTGCGCGTGGGGGCCGGGTTTGTCAGGGTGTGCGGGGCGTAAGTGTGGTCGTGCGCGCGCGGACCCGGTGCCCGCCTCCTGCCGGCGCGCCTCCAGCCCTCGCTCCCTACACCCGGGGGCCGGGCAGGACCGGGAATCGGAGGAGGGAGGGAAGGACCGCCTCGGTCTCCTTCGCCTCCTCTGGGGA\n",
            "GCTCCCCGGGGACTCTCGGGGTGCAAGCGTGGGGATTGGGTGCTGGGGTTTTTCCCTTAAAGTGTCCCCCGGAGCGGGGCGACCGGCCGGCGGCGCGGGCTCCGCTCCGCTCTCCTTGGCGGCCGGGGGCCGAGGCCCGGCGGCCGGCGAGCTCCCGGCTGCGCGGAGCCGGCTTGGGCTGCGCGTGGGGGCCGGGTTTGTCAGGGTGTGCGGGGCGTAAGTGTGGTCGTGCGCGCGCGGACCCGGTGCCCGCCTCCTGCCGGCGCGCCTCCAGCCCTCGCTCCCTACACCCGGGGGCCGGGCAGGACCGGGAATCGGAGGAGGGAGGGAAGGACCGCCTCGGTCTCCTTCGCCTCCTCTGGGGA\n",
            "AACTGTATTCTGGGTACTGTGCAAAGTACTATAGAGTTATTACTTCTTTCAGCATCACAGCAGCCTTATGAGAGCATTGCTGTAATCACCAGCATTGTACATGTGAGAAAACTGAACGTCACTCACTCAGGAAGTAACTTGCCCCAGGCGG\n",
            "AACTGTATTCTGGGTACTGTGCAAAGTACTATAGAGTTATTACTTCTTTCAGCATCACAGCAGCCTTATGAGAGCATTGCTGTAATCACCAGCATTGTACATGTGAGAAAACTGAACGTCACTCACTCAGGAAGTAACTTGCCCCAGGCGG\n",
            "AACTGTATTCTGGGTACTGTGCAAAGTACTATAGAGTTATTACTTCTTTCAGCATCACAGCAGCCTTATGAGAGCATTGCTGTAATCACCAGCATTGTACATGTGAGAAAACTGAACGTCACTCACTCAGGAAGTAACTTGCCCCAGGCGG\n",
            "GCCCACTCTCACAAACATGCCGAGATGGATACACTGGGCTCTGGGGTCAGACCACCTGGATTCAAATCCTGACTGGCATGCATGGTTACTCTGTGCCTCTGTTTACCCATCTATCAAATGGGAATGAGATGGCGGTGCCTGCTCCACTCTCGCAGTGAGGAGACTACACACCTGGCACACAGCAAGGACTAGGGACATGTCAGTCCTTTTCCTCAAGGGTGCTAGGACCCACCTTTCTCCCCACACACCTCAGTTTCCACCACCCTGCTCCCCCTGAGGACCCACCAGTGCCAGGATGGACACGAACTGGACCCCCCGCCTGATAAGGGGGGCAGGGCACCTTGAGGAGCAACTGAGAGCAGCTGGGGCCAGCAGGTCCTAAGGTCGGGGCCAGCAGGTCCTGAGGTCAAGCCCAGCTCTGTCCCGTGCCCCAGCATTTCTCATCATGGGGTGGCATTTCAGGGTTGATCTCACTTGGCCCTGTGCCCGAGGGTGACCTGTGTAATCCCTGTTTACTTGGCAACTCTCAACCTAGAGATGAGAGACTTCACAAAAATCCTTACTTGGCCCAGAGCCCTTTGTCCCAAGGCGTCGTGGTGCTGGGGCAGCCCGGGAGGGGGGAGTCTTCGCCCTGGTGTGGAGCTGTCTCCTAACATCCCCTGCTGCCCTCAGCGTCCCTCCTGGAGCTTAGCGCTTCCAGGGGCCTAGGGGCAGGGCCAGGCGAGCCTCAGTCAAGCTCCGACATTGGAGAGCCAGGCTCCATCTCCGTTCCTGCTGCAGGAAATCCCAGCTGTCTGCTGATTGGCTGCATGACCTTGGGCCAGTCCCTTCCCTTGTGTGAGAGCCAGCTTTCTCATCTGAAAATGAGGCTGATTATACTGACATTGCTGACAGAGCTGCTGTGAGGATGCTTAGAAGCCATGAATATTTTTAAATGTCACAGTCTCGTGAGCATCTGCTGTGTGTGTGTCAGGCACTATGCTCCTTTAATACCTTTTTACCCCCACAAGAATCCCACGGGGAGGTTTGTCTAATTCATTCTCCAGAGGCTCGGCGTGGTTAGGAAACCCACAGAGCCATCAGTGCTGGAATGGGGATTTGAACCGAGGTCCATCAGGATTAACCAGTGCATAAGGCCTTAGAGGCCAGGGGCGGGGCCCCAGGACAAGCATGGGAGCTGCTTGGGGAAAGCCACTCCATTTCTGCCCTGCAGCCTTTGCCCAGCCCGCCAAGTCCGCTGCTCTCTGGAGCCAGAGGGTGGCCCTCAGGCTGGTGGCCTGCCTCTCCCGATGGGGCCTGCCTGCAACTCGATAAGGCTAGGGGAACCCCCTGCTAATTGTCTGGTAACCTTTGAAATTCTCCTTCACGGTGGGTTCTCTTTTCGGATGTTTTTCAAGGTAATTCATTCCTTTTCCATAAATTGCCAGTGGCGGAGCATTAGGCAGCTGGTAATTAATCATAGGAGCAGCTTCGAGTTGATTCTTTGAAAAAGAAAAAAAAAATGAGGAAGAGAAAGCCTTGGTGGAAAAAGAAAAGAAAACCACTCTAAGGCGAGGGTGGTAATCAATACCTTGGTGTCCTAGACACGGCTGGGGGTGGGGACGGTGAACATGCGGCCTTCTCTCCTTCTTCCCAGGGCTGGCCCACTGTCAGCACCAACAGCGGCAGCGTGGGGCTCCGCTCTTGACCATTGGCTCCCTGTATGGCTTTGAGAAGTTGCTGCCTTTCCCTGGACTCAGCTTTGCCACCTGTAAAATGGGTGAAGGGCAGCTGTTAAACTGAAAAGAACATTGAATGAACCATTGTTCACTACCACGTCTTAGTCCCTGGCACAGAGCAGTGGCCCCATTGATATTCTGTTAATTGAGTGAGTGAGTGAGTGAGTGAGTGAGTGAGTGATTCTCATGCGTCTCTAGGTAGCTGTCTTGAGTGAAGGAGGTGGTACTGGCTATAGGATATTGTGACTGTAGCTCTGTTATAATTTGGAGTGACAGTCAGGGTGATGGGCCA\n",
            "GCCCACTCTCACAAACATGCCGAGATGGATACACTGGGCTCTGGGGTCAGACCACCTGGATTCAAATCCTGACTGGCATGCATGGTTACTCTGTGCCTCTGTTTACCCATCTATCAAATGGGAATGAGATGGCGGTGCCTGCTCCACTCTCGCAGTGAGGAGACTACACACCTGGCACACAGCAAGGACTAGGGACATGTCAGTCCTTTTCCTCAAGGGTGCTAGGACCCACCTTTCTCCCCACACACCTCAGTTTCCACCACCCTGCTCCCCCTGAGGACCCACCAGTGCCAGGATGGACACGAACTGGACCCCCCGCCTGATAAGGGGGGCAGGGCACCTTGAGGAGCAACTGAGAGCAGCTGGGGCCAGCAGGTCCTAAGGTCGGGGCCAGCAGGTCCTGAGGTCAAGCCCAGCTCTGTCCCGTGCCCCAGCATTTCTCATCATGGGGTGGCATTTCAGGGTTGATCTCACTTGGCCCTGTGCCCGAGGGTGACCTGTGTAATCCCTGTTTACTTGGCAACTCTCAACCTAGAGATGAGAGACTTCACAAAAATCCTTACTTGGCCCAGAGCCCTTTGTCCCAAGGCGTCGTGGTGCTGGGGCAGCCCGGGAGGGGGGAGTCTTCGCCCTGGTGTGGAGCTGTCTCCTAACATCCCCTGCTGCCCTCAGCGTCCCTCCTGGAGCTTAGCGCTTCCAGGGGCCTAGGGGCAGGGCCAGGCGAGCCTCAGTCAAGCTCCGACATTGGAGAGCCAGGCTCCATCTCCGTTCCTGCTGCAGGAAATCCCAGCTGTCTGCTGATTGGCTGCATGACCTTGGGCCAGTCCCTTCCCTTGTGTGAGAGCCAGCTTTCTCATCTGAAAATGAGGCTGATTATACTGACATTGCTGACAGAGCTGCTGTGAGGATGCTTAGAAGCCATGAATATTTTTAAATGTCACAGTCTCGTGAGCATCTGCTGTGTGTGTGTCAGGCACTATGCTCCTTTAATACCTTTTTACCCCCACAAGAATCCCACGGGGAGGTTTGTCTAATTCATTCTCCAGAGGCTCGGCGTGGTTAGGAAACCCACAGAGCCATCAGTGCTGGAATGGGGATTTGAACCGAGGTCCATCAGGATTAACCAGTGCATAAGGCCTTAGAGGCCAGGGGCGGGGCCCCAGGACAAGCATGGGAGCTGCTTGGGGAAAGCCACTCCATTTCTGCCCTGCAGCCTTTGCCCAGCCCGCCAAGTCCGCTGCTCTCTGGAGCCAGAGGGTGGCCCTCAGGCTGGTGGCCTGCCTCTCCCGATGGGGCCTGCCTGCAACTCGATAAGGCTAGGGGAACCCCCTGCTAATTGTCTGGTAACCTTTGAAATTCTCCTTCACGGTGGGTTCTCTTTTCGGATGTTTTTCAAGGTAATTCATTCCTTTTCCATAAATTGCCAGTGGCGGAGCATTAGGCAGCTGGTAATTAATCATAGGAGCAGCTTCGAGTTGATTCTTTGAAAAAGAAAAAAAAAATGAGGAAGAGAAAGCCTTGGTGGAAAAAGAAAAGAAAACCACTCTAAGGCGAGGGTGGTAATCAATACCTTGGTGTCCTAGACACGGCTGGGGGTGGGGACGGTGAACATGCGGCCTTCTCTCCTTCTTCCCAGGGCTGGCCCACTGTCAGCACCAACAGCGGCAGCGTGGGGCTCCGCTCTTGACCATTGGCTCCCTGTATGGCTTTGAGAAGTTGCTGCCTTTCCCTGGACTCAGCTTTGCCACCTGTAAAATGGGTGAAGGGCAGCTGTTAAACTGAAAAGAACATTGAATGAACCATTGTTCACTACCACGTCTTAGTCCCTGGCACAGAGCAGTGGCCCCATTGATATTCTGTTAATTGAGTGAGTGAGTGAGTGAGTGAGTGAGTGAGTGATTCTCATGCGTCTCTAGGTAGCTGTCTTGAGTGAAGGAGGTGGTACTGGCTATAGGATATTGTGACTGTAGCTCTGTTATAATTTGGAGTGACAGTCAGGGTGATGGGCCA\n",
            "TGCTGGGGTTTTTCCCTTAAAGTGTCCCCCGGAGCGGGGCGACCGGCCGGCGGCGCGGGCTCCGCTCCGCTCTCCTTGGCGGCCGGGGGCCGAGGCCCGGCGGCCGGCGAGCTCCCGGCTGCGCGGAGCCGGCTTGGGCTGCGCGTGGGGGCCGGGTTTGTCAGGGTGTGCGGGGCGTAAGTGTGGTCGTGCGCGCGCGGACCCGGTGCCCGCCTCCTGCCGGCGCGCCTCCAGCCCTCGCTCCCTACACCCGGGGGCCGGGCAGGACCGGGAATCGGAGGAGGGAGGGAAGGACCGCCTCGGTCTCCTTCGCCTCCTCTGGGGA\n",
            "GCCCACTCTCACAAACATGCCGAGATGGATACACTGGGCTCTGGGGTCAGACCACCTGGATTCAAATCCTGACTGGCATGCATGGTTACTCTGTGCCTCTGTTTACCCATCTATCAAA\n",
            "GCCCACTCTCACAAACATGCCGAGATGGATACACTGGGCTCTGGGGTCAGACCACCTGGATTCAAATCCTGACTGGCATGCATGGTTACTCTGTGCCTCTGTTTACCCATCTATCAAA\n",
            "TATGGAGCTTCTTTTGTCATCATCCATCCATTCAGGTAAGCATTGATTGAGCCCCTCTTACATCCATCCGTCTAACCAAATGAGCATTGATCTCTCATTGATCATGTATGATC\n",
            "CCACGGGGTTGGTGGTTCCAGCCAAGATGGGGACAGAGCCAGCCCAGGCCAGCCACCCTGGAGCCAAGTAAGACTGGGCTGGGTCCACCAGAAGCCCAGGGTCACAGGCAGGCATGGGCTTGCTGGAGGCCGACAGCACCGAGGGATTGTCCA\n",
            "CCACGGGGTTGGTGGTTCCAGCCAAGATGGGGACAGAGCCAGCCCAGGCCAGCCACCCTGGAGCCAAGTAAGACTGGGCTGGGTCCACCAGAAGCCCAGGGTCACAGGCAGGCATGGGCTTGCTGGAGGCCGACAGCACCGAGGGATTGTCCA\n",
            "TTGTCTTTAATGAATAGGAAAATATTGAGAAAATCTGGATTGGGAAAGTCGAAGAGCAGGTATAGATGGGGTCCAGTTGCCCAAAGACAGTGAAGCAGTCTGGGAATATCTGCTCTGCCTTGCTGATTGGTGAAAGCAGAACTATGCAGAGACTGCGAGTCCCTTGGCTGGGAAGAAAAGGGCAGGGACTCCCAGGATATGGAGCTCTGGGGAGTGGTGTGAGGCGAGCAGAGGTGCCCATTCTCCTCTTTCCACCCCGCCCTGCCTAAGACCCCCACCTTGGCCTCAGCCCTCACCAGCCCTGGCTGGGTTTCTGGCCTGGGGTTGGCACAGCTATATG\n",
            "TTGTCTTTAATGAATAGGAAAATATTGAGAAAATCTGGATTGGGAAAGTCGAAGAGCAGGTATAGATGGGGTCCAGTTGCCCAAAGACAGTGAAGCAGTCTGGGAATATCTGCTCTGCCTTGCTGATTGGTGAAAGCAGAACTATGCAGAGACTGCGAGTCCCTTGGCTGGGAAGAAAAGGGCAGGGACTCCCAGGATATGGAGCTCTGGGGAGTGGTGTGAGGCGAGCAGAGGTGCCCATTCTCCTCTTTCCACCCCGCCCTGCCTAAGACCCCCACCTTGGCCTCAGCCCTCACCAGCCCTGGCTGGGTTTCTGGCCTGGGGTTGGCACAGCTATATG\n",
            "AAGGAGTGGGCTCAATGTCTAAAAATAGTCTTGTGACCGGTCTGTCTCCTGGATTGCCTCTGAGACCAGTGCCCTCCTAGTCACCCTATCTGTGGGTCTGCGCAGGCCCCAGTGCTGCTCCCTGACCTTCTGAAGGATGGGTTAGGGGATGAGGAGAGGGGAGCCCCCATAGGAATGCAGAACAGGAGGTGAGGAGGTGGGAACTGCAGGGCAGAGGGG\n",
            "AAGGAGTGGGCTCAATGTCTAAAAATAGTCTTGTGACCGGTCTGTCTCCTGGATTGCCTCTGAGACCAGTGCCCTCCTAGTCACCCTATCTGTGGGTCTGCGCAGGCCCCAGTGCTGCTCCCTGACCTTCTGAAGGATGGGTTAGGGGATGAGGAGAGGGGAGCCCCCATAGGAATGCAGAACAGGAGGTGAGGAGGTGGGAACTGCAGGGCAGAGGGG\n",
            "CTACTAGAGCCCCTGGGGTTTTTACTTTTAATGGCTGCCTAATATTCTGCTGAGATCACCATATCATATTTTATTATTCCTTCTGTTAGGTATTTGGGTTGCTTCTGATTTTTCACTTTAGAAATAACACTATGATGAGTATCTTCAGGCATATAGCCTTTCCTTTCCTTAGGATTATTTTCTTGGGAGTAAAGAAGACAC\n",
            "CTACTAGAGCCCCTGGGGTTTTTACTTTTAATGGCTGCCTAATATTCTGCTGAGATCACCATATCATATTTTATTATTCCTTCTGTTAGGTATTTGGGTTGCTTCTGATTTTTCACTTTAGAAATAACACTATGATGAGTATCTTCAGGCATATAGCCTTTCCTTTCCTTAGGATTATTTTCTTGGGAGTAAAGAAGACAC\n",
            "TTTATCACAGGGATGGGTAAGGGTCTCCACAGGCACCTGACACAGGAGCTCAGTGGTGAGACCCAGATGAAGGGTGGGGAGGCCCT\n",
            "GAGGGAACGCAGTCAAGAATTCGAGTGTGACTCATGTGGCCAGAGCCAGGAATGCAGAGATGAGGCTGGTGCAAGGGCCAGGCCGGGGTCAGGCAGGCCCCATCTATTGCAGGGGGCTGACCATTGGGCAGGGCTGCTGCTCAGACACTTAGGC\n",
            "GAGGGAACGCAGTCAAGAATTCGAGTGTGACTCATGTGGCCAGAGCCAGGAATGCAGAGATGAGGCTGGTGCAAGGGCCAGGCCGGGGTCAGGCAGGCCCCATCTATTGCAGGGGGCTGACCATTGGGCAGGGCTGCTGCTCAGACACTTAGGC\n",
            "GCAGGGCCTGCTCTGCAGAGACAGGGGAACTGCCACAGCCACCTCGTCTGGCACACACACACCACACGTGTCCCACATGGAGACAGCAGCCCGGGCACCCGGCCCTGTGACTGCCACACAGTGCCACACACTGCCCTCTCAGCGTTTGGTGTGGGCTGGGAGGAGAACCAGGACATGCCCCAGCTGCACGAGGAAGTGGG\n",
            "GCAGGGCCTGCTCTGCAGAGACAGGGGAACTGCCACAGCCACCTCGTCTGGCACACACACACCACACGTGTCCCACATGGAGACAGCAGCCCGGGCACCCGGCCCTGTGACTGCCACACAGTGCCACACACTGCCCTCTCAGCGTTTGGTGTGGGCTGGGAGGAGAACCAGGACATGCCCCAGCTGCACGAGGAAGTGGG\n",
            "GCAGGGCCTGCTCTGCAGAGACAGGGGAACTGCCACAGCCACCTCGTCTGGCACACACACACCACACGTGTCCCACATGGAGACAGCAGCCCGGGCACCCGGCCCTGTGACTGCCACACAGTGCCACACACTGCCCTCTCAGCGTTTGGTGTGGGCTGGGAGGAGAACCAGGACATGCCCCAGCTGCACGAGGAAGTGGG\n",
            "TAGCCCTGACACAGCCCTCCCCTCCCAGGCCAGCAGACCGTGACCCACATCCCCCACCTCTCTGTCCCTTACCCTCGGGGGCCTCTGGATCTGGGGACAA\n",
            "TAGCCCTGACACAGCCCTCCCCTCCCAGGCCAGCAGACCGTGACCCACATCCCCCACCTCTCTGTCCCTTACCCTCGGGGGCCTCTGGATCTGGGGACAA\n",
            "TAGCCCTGACACAGCCCTCCCCTCCCAGGCCAGCAGACCGTGACCCACATCCCCCACCTCTCTGTCCCTTACCCTCGGGGGCCTCTGGATCTGGGGACAA\n",
            "AGGCTCTTCCCCAAACCCTCACCACCCCATGCATACACACACACGTGCATGCACTCGCCACTACCCAGCACACGCCATCAGCCCTCTCCCTCTCCCACTTGCACAATCTTACACTCACCTAACACGGTTCCAGCCACCTCAACTTTTGTCCCAAACCCAGGCACTTATTCTCCAAGGATCTGAATTCAGGGTATTACCACCTCCAACAAAAGAAATGCTA\n",
            "CCGGCCGGCGGCGCGGGCTCCGCTCCGCTCTCCTTGGCGGCCGGGGGCCGAGGCCCGGCGGCCGGCGAGCTCCCGGCTGCGCGGAGCCGGCTTGGGCTGCGCGTGGGGGCCGGGTTTGTCAGGGTGTGCGGGGCGTAAGTGTGGTCGTGCGCGCGCGGACCCGGTGCCCGCCTCCTGCCGGCGCGCCTCCAGCCCTCGCTCCCTACACCCGGGGGCCGGGCAGGACCGGGAATCGGAGGAGGGAGGGAAGGACCGCCTCGGTCTCCTTCGCCTCCTCTGGGGA\n",
            "CGCTCTCCTTGGCGGCCGGGGGCCGAGGCCCGGCGGCCGGCGAGCTCCCGGCTGCGCGGAGCCGGCTTGGGCTGCGCGTGGGGGCCGGGTTTGTCAGGGTGTGCGGGGCGTAAGTGTGGTCGTGCGCGCGCGGACCCGGTGCCCGCCTCCTGCCGGCGCGCCTCCAGCCCTCGCTCCCTACACCCGGGGGCCGGGCAGGACCGGGAATCGGAGGAGGGAGGGAAGGACCGCCTCGGTCTCCTTCGCCTCCTCTGGGGA\n",
            "TATGGAGCTTCTTTTGTCATCATCCATCCATTCAGGTAAGCATTGATTGAGCCCCTCTTACATCCATCCGTCTAACCAAATGAGCATTGATCTCTCATTGATCATGTATGATC\n",
            "CCACGGGGTTGGTGGTTCCAGCCAAGATGGGGACAGAGCCAGCCCAGGCCAGCCACCCTGGAGCCAAGTAAGACTGGGCTGGGTCCACCAGAAGCCCAGGGTCACAGGCAGGCATGGGCTTGCTGGAGGCCGACAGCACCGAGGGATTGTCCA\n",
            "CCACGGGGTTGGTGGTTCCAGCCAAGATGGGGACAGAGCCAGCCCAGGCCAGCCACCCTGGAGCCAAGTAAGACTGGGCTGGGTCCACCAGAAGCCCAGGGTCACAGGCAGGCATGGGCTTGCTGGAGGCCGACAGCACCGAGGGATTGTCCA\n",
            "TTGTCTTTAATGAATAGGAAAATATTGAGAAAATCTGGATTGGGAAAGTCGAAGAGCAGGTATAGATGGGGTCCAGTTGCCCAAAGACAGTGAAGCAGTCTGGGAATATCTGCTCTGCCTTGCTGATTGGTGAAAGCAGAACTATGCAGAGACTGCGAGTCCCTTGGCTGGGAAGAAAAGGGCAGGGACTCCCAGGATATGGAGCTCTGGGGAGTGGTGTGAGGCGAGCAGAGGTGCCCATTCTCCTCTTTCCACCCCGCCCTGCCTAAGACCCCCACCTTGGCCTCAGCCCTCACCAGCCCTGGCTGGGTTTCTGGCCTGGGGTTGGCACAGCTATATG\n",
            "TTGTCTTTAATGAATAGGAAAATATTGAGAAAATCTGGATTGGGAAAGTCGAAGAGCAGGTATAGATGGGGTCCAGTTGCCCAAAGACAGTGAAGCAGTCTGGGAATATCTGCTCTGCCTTGCTGATTGGTGAAAGCAGAACTATGCAGAGACTGCGAGTCCCTTGGCTGGGAAGAAAAGGGCAGGGACTCCCAGGATATGGAGCTCTGGGGAGTGGTGTGAGGCGAGCAGAGGTGCCCATTCTCCTCTTTCCACCCCGCCCTGCCTAAGACCCCCACCTTGGCCTCAGCCCTCACCAGCCCTGGCTGGGTTTCTGGCCTGGGGTTGGCACAGCTATATG\n",
            "AAGGAGTGGGCTCAATGTCTAAAAATAGTCTTGTGACCGGTCTGTCTCCTGGATTGCCTCTGAGACCAGTGCCCTCCTAGTCACCCTATCTGTGGGTCTGCGCAGGCCCCAGTGCTGCTCCCTGACCTTCTGAAGGATGGGTTAGGGGATGAGGAGAGGGGAGCCCCCATAGGAATGCAGAACAGGAGGTGAGGAGGTGGGAACTGCAGGGCAGAGGGG\n",
            "AAGGAGTGGGCTCAATGTCTAAAAATAGTCTTGTGACCGGTCTGTCTCCTGGATTGCCTCTGAGACCAGTGCCCTCCTAGTCACCCTATCTGTGGGTCTGCGCAGGCCCCAGTGCTGCTCCCTGACCTTCTGAAGGATGGGTTAGGGGATGAGGAGAGGGGAGCCCCCATAGGAATGCAGAACAGGAGGTGAGGAGGTGGGAACTGCAGGGCAGAGGGG\n",
            "TTCCGATGGAGCTGGTCTGTAAACTTCAGGTTCCCTGACCTCCCAAGGGCTCTTGCTCATCCTCAGATTGCTTCATGGGGCCCACTTGCTGGGTGCTTGGAGGGTGAGGTGCATCGTGGGTGCATTCCTCAGATCCCACTGGGATTGCTGAAGCGGCCAGCACA\n",
            "GGTGTGGCAGCTTA\n",
            "GAGAGCTGTCGCTTGCGCCCGGGCGCGCGGCTGGCTGCCTCCTCGGCGGCGGCGGCGGCGGCCCCATTAGCGGAGCCTCCGCCTATGATTGGCTTCGCCCGGGAAGCTGGAGACGGGCGATGAA\n",
            "GAGAGCTGTCGCTTGCGCCCGGGCGCGCGGCTGGCTGCCTCCTCGGCGGCGGCGGCGGCGGCCCCATTAGCGGAGCCTCCGCCTATGATTGGCTTCGCCCGGGAAGCTGGAGACGGGCGATGAA\n",
            "GAGAGCTGTCGCTTGCGCCCGGGCGCGCGGCTGGCTGCCTCCTCGGCGGCGGCGGCGGCGGCCCCATTAGCGGAGCCTCCGCCTATGATTGGCTTCGCCCGGGAAGCTGGAGACGGGCGATGAA\n",
            "GTAAACAGGAGAACTTCCTCTGTGTTACTGGCTTCAGGGCTCCAAGGTCTGAGGTCTCTGACCCTGGATGTATGGCCTGAGTTGGATTTCATCAAAATATTGT\n",
            "CCCTGGGCTCCCTCCGGGCGCGGGGGTGAGTGTGCGTGCCTGTGTGTGACAGAGGAGGGCCGGTGCATTGTGGGTAGCGCCGTGTGCTGCATGGTGTCAGCCCCAGGCTTGGCCGCGAGAGTGGCACCGGCGTGCACCGGGCCGGTTTTGTGTGCCCACAGACGGGGCCTTCATTTCTCTTTGGGGCCACGGGGCAGAGTGCGCCCCCACCCAGGTCCCTGAAGGTGAAGTGTTGGCTCTCCAGCAGACATTTTACGAGGCTCTGTGCTCAGTCGGCTCACGCTGCAAAAATAAGAATCGGCCAGGATGCCTTCGGAGGCAGCGCCTGAGGGTCCTCGACTTCAAGCTGCTCCTTT\n",
            "AAGGAGTGGGCTCAATGTCTAAAAATAGTCTTGTGACCGGTCTGTCTCCTGGATTGCCTCTGAGACCAGTGCCCTCCTAGTCACCCTATCTGT\n",
            "AAGGAGTGGGCTCAATGTCTAAAAATAGTCTTGTGACCGGTCTGTCTCCTGGATTGCCTCTGAGACCAGTGCCCTCCTAGTCACCCTATCTGT\n",
            "AAGGAGTGGGCTCAATGTCTAAAAATAGTCTTGTGACCGGTCTGTCTCCTGGATTGCCTCTGAGACCAGTGCCCTCCTAGTCACCCTATCTGT\n",
            "AGGCTCTTCCCCAAACCCTCACCACCCCATGCATACACACACACGTGCATGCACTCGCCACTACCCAGCACACGCCATCAGCCCTCTCCCTCTCCCACTTGCACAATCTTACACTCACCTAACACGGTTCCAGCCACCTCAACTTTTGTCCCAAACCCAGGCACTTATTCTCCAAGGATCTGAATTCAGGGTATTACCACCTCCAACAAAAGAAATGCTAGGAAGTCTGATTATCTGAAGAGCATTCACTTCTCATTCTGTTTCACACTTATCGACATATGCCCTGTGTGGCTGAGGACGTCCCTGGTGGGGTAAGGTGTCCTGTGTTGTCCACAGGCCCTCCCTTCCTGTGGCCC\n",
            "AGGCTCTTCCCCAAACCCTCACCACCCCATGCATACACACACACGTGCATGCACTCGCCACTACCCAGCACACGCCATCAGCCCTCTCCCTCTCCCACTTGCACAATCTTACACTCACCTAACACGGTTCCAGCCACCTCAACTTTTGTCCCAAACCCAGGCACTTATTCTCCAAGGATCTGAATTCAGGGTATTACCACCTCCAACAAAAGAAATGCTAGGAAGTCTGATTATCTGAAGAGCATTCACTTCTCATTCTGTTTCACACTTATCGACATATGCCCTGTGTGGCTGAGGACGTCCCTGGTGGGGTAAGGTGTCCTGTGTTGTCCACAGGCCCTCCCTTCCTGTGGCCC\n",
            "Exon file type summaries for the second gene from: ENSG00000108175-ENSMUSG00000007817\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t5 exons with: Only nonintersecting\n",
            "\t16 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t23 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108175-ENSMUSG00000007817/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108175-ENSMUSG00000007817/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 4, Nonintersecting exons= 27, OTHER= 2\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 17, Nonintersecting exons= 28, OTHER= 1\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108175-ENSMUSG00000007817/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108175-ENSMUSG00000007817/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108175-ENSMUSG00000007817/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108175-ENSMUSG00000007817/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108175-ENSMUSG00000007817/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3220\tENSG00000108176-ENSMUSG00000036764\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108176-ENSMUSG00000036764\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108176\tchr10\t67796669\t67838188\t-\tENSG00000108176\tDNAJC12\tprotein_coding\t5\t18\tgene\n",
            "\n",
            "ENSMUSG00000036764\tchr10\t63218222\t63246355\t+\tENSMUSG00000036764\tDnajc12\tprotein_coding\t3\t13\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t18\t16\n",
            "\n",
            "Org2\t13\t10\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108176-ENSMUSG00000036764\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t7 exons with: Only nonintersecting\n",
            "\t6 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108176-ENSMUSG00000036764\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t7 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108176-ENSMUSG00000036764/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108176-ENSMUSG00000036764/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 11, Nonintersecting exons= 0, OTHER= 7\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 13, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108176-ENSMUSG00000036764/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108176-ENSMUSG00000036764/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108176-ENSMUSG00000036764/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108176-ENSMUSG00000036764/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108176-ENSMUSG00000036764/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3221\tENSG00000108179-ENSMUSG00000021868\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108179-ENSMUSG00000021868\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108179\tchr10\t79347469\t79355334\t+\tENSG00000108179\tPPIF\tprotein_coding\t5\t17\tgene\n",
            "\n",
            "ENSMUSG00000021868\tchr14\t25694578\t25700892\t+\tENSMUSG00000021868\tPpif\tprotein_coding\t3\t11\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t17\t13\n",
            "\n",
            "Org2\t11\t10\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108179-ENSMUSG00000021868\n",
            "\t0 exons with: No file exists\n",
            "\t8 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t5 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "AGAAAGCTCAAGGTCAGATCGCAGAGAGGGAGGGCTCAGCACCTCTGGGCCGGCCCGGGCACGAGGGAGGGGCTCCAGGAGCCTTCTGGGGCTGAGCCTAGATCCGGAGCTCCGAGGTGGGTGTCGGGGGTCTTGGGGTGAGCGTCGTGGCCCAGCGGGTGCTCACGTGGCGGCCCTTGCACAACACGGAGCGCTTCCTGGCTCCGGCCCCGCCCCTGCGGTCGGGCTCACACTGGGGGTGCTGGGAAATGGAGCGAGAGGTGGTTTCCAGCAGTAGTGCGGGCCCAGTAGGCCTCAGGCCCCGGCCACCTGGTGGACCCCAGAATGCCCCTCCTGCGAGTCGGGACACACTCAGAGACAGTGTGCCCGGCGCCTCAACCCCTGCCACTGTCCTTGGGGGCCACACTGAGCACCTCCCCTAACTCTGTTTTTGGGTCTTTTCTAAAG\n",
            "AGAAAGCTCAAGGTCAGATCGCAGAGAGGGAGGGCTCAGCACCTCTGGGCCGGCCCGGGCACGAGGGAGGGGCTCCAGGAGCCTTCTGGGGCTGAGCCTAGATCCGGAGCTCCGAGGTGGGTGTCGGGGGTCTTGGGGTGAGCGTCGTGGCCCAGCGGGTGCTCACGTGGCGGCCCTTGCACAACACGGAGCGCTTCCTGGCTCCGGCCCCGCCCCTGCGGTCGGGCTCACACTGGGGGTGCTGGGAAATGGAGCGAGAGGTGGTTTCCAGCAGTAGTGCGGGCCCAGTAGGCCTCAGGCCCCGGCCACCTGGTGGACCCCAGAATGCCCCTCCTGCGAGTCGGGACACACTCAGAGACAGTGTGCCCGGCGCCTCAACCCCTGCCACTGTCCTTGGGGGCCACACTGAGCACCTCCCCTAACTCTGTTTTTGGGTCTTTTCTAAAG\n",
            "GAGTCTTTATCCCCCTGTTCACTGCTGGGGATGTAG\n",
            "GAGTCTTTATCCCCCTGTTCACTGCTGGGGATGTAG\n",
            "GATCGCAGAGAGGGAGGGCTCAGCACCTCTGGGCCGGCCCGGGCACGAGGGAGGGGCTCCAGGAGCCTTCTGGGGCTGAGCCTAGATCCGGAGCTCCGAGGTGGGTGTCGGGGGTCTTGGGGTGAGCGTCGTGGCCCAGCGGGTGCTCACGTGGCGGCCCTTGCACAACACGGAGCGCTTCCTGGCTCCGGCCCCGCCCCTGCGGTCGGGCTCACACTGGGGGTGCTGGGAAATGGAGCGAGAGGTGGTTTCCAGCAGTAGTGCGGGCCCAGTAGGCCTCAGGCCCCGGCCACCTGGTGGACCCCAGAATGCCCCTCCTGCGAGTCGGGACACACTCAGAGACAGTGTGCCCGGCGCCTCAACCCCTGCCACTGTCCTTGGGGGCCACACTGAGCACCTCCCCTAACTCTGTTTTTGGGTCTTTTCTAAAG\n",
            "GATCGCAGAGAGGGAGGGCTCAGCACCTCTGGGCCGGCCCGGGCACGAGGGAGGGGCTCCAGGAGCCTTCTGGGGCTGAGCCTAGATCCGGAGCTCCGAGGTGGGTGTCGGGGGTCTTGGGGTGAGCGTCGTGGCCCAGCGGGTGCTCACGTGGCGGCCCTTGCACAACACGGAGCGCTTCCTGGCTCCGGCCCCGCCCCTGCGGTCGGGCTCACACTGGGGGTGCTGGGAAATGGAGCGAGAGGTGGTTTCCAGCAGTAGTGCGGGCCCAGTAGGCCTCAGGCCCCGGCCACCTGGTGGACCCCAGAATGCCCCTCCTGCGAGTCGGGACACACTCAGAGACAGTGTGCCCGGCGCCTCAACCCCTGCCACTGTCCTTGGGGGCCACACTGAGCACCTCCCCTAACTCTGTTTTTGGGTCTTTTCTAAAG\n",
            "GAGTCTTTATCCCCCTGTTCACTGCTGGGGATGTAG\n",
            "GAGTCTTTATCCCCCTGTTCACTGCTGGGGATGTAG\n",
            "TGTGGGTCAGGACCCAGGTGGTGTTGGGGCTGGTCTTCTATTAGCACAGACCTTCAAGATCAGGGATTGGGTGTCCTGGGCAGGCAGAGCCTGGAGGCAGGGGCATGGCTGTCATGACCTCTTTGAAGACCTTCCTGCCTGGAATTCTCTCCCCATTCCCTCCACCC\n",
            "TGTGGGTCAGGACCCAGGTGGTGTTGGGGCTGGTCTTCTATTAGCACAGACCTTCAAGATCAGGGATTGGGTGTCCTGGGCAGGCAGAGCCTGGAGGCAGGGGCATGGCTGTCATGACCTCTTTGAAGACCTTCCTGCCTGGAATTCTCTCCCCATTCCCTCCACCC\n",
            "TGTGGGTCAGGACCCAGGTGGTGTTGGGGCTGGTCTTCTATTAGCACAGACCTTCAAGATCAGGGATTGGGTGTCCTGGGCAGGCAGAGCCTGGAGGCAGGGGCATGGCTGTCATGACCTCTTTGAAGACCTTCCTGCCTGGAATTCTCTCCCCATTCCCTCCACCC\n",
            "AGAATGTGGAGATGTCTGGAGGTCAGGTCCTGGGGTCCTGCCCCAGGTCCCGCCCCAGCTCTACCACTGAGAGCTTCCACAAGGCCCTGTCCTCTCAGGCCACCCCACTGAGAACTA\n",
            "AGAATGTGGAGATGTCTGGAGGTCAGGTCCTGGGGTCCTGCCCCAGGTCCCGCCCCAGCTCTACCACTGAGAGCTTCCACAAGGCCCTGTCCTCTCAGGCCACCCCACTGAGAACTA\n",
            "GGCAATGCTGCAGCAGACATTTGGATATCAGCAAATTTCCAAATGAAGCCTCTGTATTATCATAAACCTGCAGATAGCCACAGGCCTGTAAGGGCCC\n",
            "GGCAATGCTGCAGCAGACATTTGGATATCAGCAAATTTCCAAATGAAGCCTCTGTATTATCATAAACCTGCAGATAGCCACAGGCCTGTAAGGGCCC\n",
            "Exon file type summaries for the second gene from: ENSG00000108179-ENSMUSG00000021868\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t3 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108179-ENSMUSG00000021868/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108179-ENSMUSG00000021868/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 9, Nonintersecting exons= 0, OTHER= 8\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 4, Nonintersecting exons= 7, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108179-ENSMUSG00000021868/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108179-ENSMUSG00000021868/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108179-ENSMUSG00000021868/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108179-ENSMUSG00000021868/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108179-ENSMUSG00000021868/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3222\tENSG00000108219-ENSMUSG00000037824\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108219-ENSMUSG00000037824\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108219\tchr10\t80454265\t80533124\t+\tENSG00000108219\tTSPAN14\tprotein_coding\t10\t33\tgene\n",
            "\n",
            "ENSMUSG00000037824\tchr14\t40628402\t40688764\t-\tENSMUSG00000037824\tTspan14\tprotein_coding\t3\t15\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t33\t29\n",
            "\n",
            "Org2\t15\t13\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108219-ENSMUSG00000037824\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t15 exons with: Only nonintersecting\n",
            "\t10 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t8 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108219-ENSMUSG00000037824\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t15 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108219-ENSMUSG00000037824/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108219-ENSMUSG00000037824/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 18, Nonintersecting exons= 0, OTHER= 15\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 15, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108219-ENSMUSG00000037824/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108219-ENSMUSG00000037824/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108219-ENSMUSG00000037824/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108219-ENSMUSG00000037824/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108219-ENSMUSG00000037824/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3223\tENSG00000108231-ENSMUSG00000067242\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108231-ENSMUSG00000067242\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108231\tchr10\t93757840\t93806272\t+\tENSG00000108231\tLGI1\tprotein_coding\t31\t75\tgene\n",
            "\n",
            "ENSMUSG00000067242\tchr19\t38252984\t38300662\t+\tENSMUSG00000067242\tLgi1\tprotein_coding\t9\t31\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t75\t66\n",
            "\n",
            "Org2\t31\t25\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "CATCACTTCGCTTGCCTTCGAAGACTGGTCTGTGCTCAGTGTTTTCTGTGAAGTGAGTCGGCTCTCTTCCTCCAGCAGTTGGCTCGCTCCCACCTCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGCCCGT\n",
            "CATCACTTCGCTTGCCTTCGAAGACTGGTCTGTGCTCAGTGTTTTCTGTGAAGTGAGTCGGCTCTCTTCCTCCAGCAGTTGGCTCGCTCCCACCTCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGCCCGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "CTTGCCTTCGAAGACTGGTCTGTGCTCAGTGTTTTCTGTGAAGTGAGTCGGCTCTCTTCCTCCAGCAGTTGGCTCGCTCCCACCTCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "CTTGCCTTCGAAGACTGGTCTGTGCTCAGTGTTTTCTGTGAAGTGAGTCGGCTCTCTTCCTCCAGCAGTTGGCTCGCTCCCACCTCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "AGGTTATTTACGTCGAACTCCTTTGATGTGATCAGTGATGATGCTTTTATTGGTCTTCCACATCTAGAATATTTGT\n",
            "AGGTTATTTACGTCGAACTCCTTTGATGTGATCAGTGATGATGCTTTTATTGGTCTTCCACATCTAGAATATTTGT\n",
            "AGGTTATTTACGTCGAACTCCTTTGATGTGATCAGTGATGATGCTTTTATTGGTCTTCCACATCTAGAATATTTGT\n",
            "AGATTCATAGAAAACAACAATATCAAGTCCATTTCAAGACATACTTTCCGGGGACTCAAGTCTTTGATTCACTTGT\n",
            "AGATTCATAGAAAACAACAATATCAAGTCCATTTCAAGACATACTTTCCGGGGACTCAAGTCTTTGATTCACTTGT\n",
            "AGATTCATAGAAAACAACAATATCAAGTCCATTTCAAGACATACTTTCCGGGGACTCAAGTCTTTGATTCACTTGT\n",
            "CTCAGTGTTTTCTGTGAAGTGAGTCGGCTCTCTTCCTCCAGCAGTTGGCTCGCTCCCACCTCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGCCCGT\n",
            "CTCAGTGTTTTCTGTGAAGTGAGTCGGCTCTCTTCCTCCAGCAGTTGGCTCGCTCCCACCTCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGCCCGT\n",
            "AGGTTATTTACGTCGAACTCCTTTGATGTGATCAGTGATGATGCTTTTATTGGTCTTCCACATCTAGAATATTTGT\n",
            "AGGTTATTTACGTCGAACTCCTTTGATGTGATCAGTGATGATGCTTTTATTGGTCTTCCACATCTAGAATATTTGT\n",
            "AGGTTATTTACGTCGAACTCCTTTGATGTGATCAGTGATGATGCTTTTATTGGTCTTCCACATCTAGAATATTTGT\n",
            "AGATTCATAGAAAACAACAATATCAAGTCCATTTCAAGACATACTTTCCGGGGACTCAAGTCTTTGATTCACTTGT\n",
            "AGATTCATAGAAAACAACAATATCAAGTCCATTTCAAGACATACTTTCCGGGGACTCAAGTCTTTGATTCACTTGT\n",
            "AGATTCATAGAAAACAACAATATCAAGTCCATTTCAAGACATACTTTCCGGGGACTCAAGTCTTTGATTCACTTGT\n",
            "CCAGCAGTTGGCTCGCTCCCACCTCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "CCAGCAGTTGGCTCGCTCCCACCTCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGTGAGAAACATTTATATAATGACTATTTTTAATATGGCATGTATTCGGATAAGCCTTCTAGTAAAACGAGCTTAATATTGTTT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGTGAGAAACATTTATATAATGACTATTTTTAATATGGCATGTATTCGGATAAGCCTTCTAGTAAAACGAGCTTAATATTGTTT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGTGAGAAACATTTATATAATGACTATTTTTAATATGGCATGTATTCGGATAAGCCTTCTAGTAAAACGAGCTTAATATTGTTT\n",
            "TCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGCCCGT\n",
            "TCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGCCCGT\n",
            "TCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "TCGCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "GCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "GCAGCACCTCACAGGTCTCTTCCCCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "CCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "CCCAGCAGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "AGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGC\n",
            "AGTGCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGC\n",
            "GCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "GCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "GCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGT\n",
            "GCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGT\n",
            "GCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGT\n",
            "GCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGT\n",
            "GCGCTGCCTGAGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "AGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "AGCGAGGAACAGCTCACGAATCAGCTGGAGGTCCCTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGTGAGAAACATTTATATAATGACTATTTTTAATATGGCATGTATTCGGATAAGCCTTCTAGTAAAACGAGCTTAATATTGTTTCTGTTGAATGTGATGATGCTTCTGAAAGGGGCGAGAACATGGGTAAAGCGGGAGATAGATGGGTTATTTGTAACTTCACATCCACCATAAAAGTTGATTCTTTAAATTCAATTACCTGTGATGCTGTCTAGCTTAATGGCTATTTGTGAAGTTGTTGGGACGATGCAACAAGAATACTAATTTTGATGATCTTACTTTCATTTTTAACTTGGGGAACTTCGGTTATTTTGCGCTGGAGTTCTTGGGTTTGTCAATTCAGAGCCCAGCATCACATATGCACTCAAGATGTTGTCTCTTGGGGATCTGGGGCAGACTATGTCTCTGTATGATTTACCATGTGGGGAGAAAAAGAGAGAGGAAAAAAAACACCCTTGGTTCTTTCTAGGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGTGAGAAACATTTATATAATGACTATTTTTAATATGGCATGTATTCGGATAAGCCTTCTAGTAAAACGAGCTTAATATTGTTTCTGTTGAATGTGATGATGCTTCTGAAAGGGGCGAGAACATGGGTAAAGCGGGAGATAGATGGGTTATTTGTAACTTCACATCCACCATAAAAGTTGATTCTTTAAATTCAATTACCTGTGATGCTGTCTAGCTTAATGGCTATTTGTGAAGTTGTTGGGACGATGCAACAAGAATACTAATTTTGATGATCTTACTTTCATTTTTAACTTGGGGAACTTCGGTTATTTTGCGCTGGAGTTCTTGGGTTTGTCAATTCAGAGCCCAGCATCACATATGCACTCAAGATGTTGTCTCTTGGGGATCTGGGGCAGACTATGTCTCTGTATGATTTACCATGTGGGGAGAAAAAGAGAGAGGAAAAAAAACACCCTTGGTTCTTTCTAGGT\n",
            "CTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "CTGTTTTGAAAAAAAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "AAGGAGATGGAGCTGGCATCCTTTGAAAATCAAATGTCCCTCTGCCTGCTCCAC\n",
            "AAGGAGATGGAGCTGGCATCCTTTGAAAATCAAATGTCCCTCTGCCTGCTCCAC\n",
            "AAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "AAAAAAAAAAAAAAAAAAAGCAGATAGAGAAGCAGAGAAGGGTGGACTCCTATGTGACCTGTTCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "TCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGCCCGT\n",
            "TCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGCCCGT\n",
            "TCTTAGAGCAAGACAATCAGCAGCTGAATTCCAGAAGCCCTGTTCATGTGTGGGGATATTCTCTCGACTGCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGTAAGGCCCGT\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGTGAGAAACATTTATATAATGACTATTTTTAATATGGCATGTATTCGGATAAGCCTTCTAGTAAAACGAGCTTAATATTGTTTCTGTTGAATGTGATGATGCTTCTGAAAGGGGCGAGAACATGGGTAAAGCGGGAGATAGATGGGTTATTTGTAACTTCACATCCACCATAAAAGTTGATTCTTTAAATTCAATTACCTGTGATGCTGTCTAGCTTAATGGCTATTTGTGAAGTTGTTGGGACGATGCAACAAGAATACTAATTTTGATGATCTTACTTTCATTTTTAAC\n",
            "AGATCCTTTGTGAGATCTGGTTTTACTGAAATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGTGAGAAACATTTATATAATGACTATTTTTAATATGGCATGTATTCGGATAAGCCTTCTAGTAAAACGAGCTTAATATTGTTTCTGTTGAATGTGATGATGCTTCTGAAAGGGGCGAGAACATGGGTAAAGCGGGAGATAGATGGGTTATTTGTAACTTCACATCCACCATAAAAGTTGATTCTTTAAATTCAATTACCTGTGATGCTGTCTAGCTTAATGGCTATTTGTGAAGTTGTTGGGACGATGCAACAAGAATACTAATTTTGATGATCTTACTTTCATTTTTAAC\n",
            "GCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "GCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "GCATGGAATCAGAAAGCAGCAGAAGGATGGGAAATGCCTGCATTCCCCTGAAAAGAATTGCCTATTTCCTATGCCTCTTTTCTGTGGTTTTGCTGACTGAGGGGAAGAAACCAGCGAAGCCAAAATGCCCTGCCGTGTGTACTTGTAGCAAAGATAACGCTTTATGTGAGAATGCGAGATCCATTCCACGCACCGTTCCTCCTGATGTTATCTCACTGT\n",
            "GATCATGCTTAGTTAGCCCCCACCCTGGACATTATCCGGAGGCACCTGGAGCAGGCTTCTCTTTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCGCTTCCTCTAGGAAGGGACAGAATCGTACTTTATAAAATAGTGGGGGAATATAGCAACTGTTATGGTAAATTGGATTAACATAAGAT\n",
            "GATCATGCTTAGTTAGCCCCCACCCTGGACATTATCCGGAGGCACCTGGAGCAGGCTTCTCTTTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCGCTTCCTCTAGGAAGGGACAGAATCGTACTTTATAAAATAGTGGGGGAATATAGCAACTGTTATGGTAAATTGGATTAACATAAGAT\n",
            "GATCATGCTTAGTTAGCCCCCACCCTGGACATTATCCGGAGGCACCTGGAGCAGGCTTCTCTTTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCTCGCTTCCTCTAGGAAGGGACAGAATCGTACTTTATAAAATAGTGGGGGAATATAGCAACTGTTATGGTAAATTGGATTAACATAAGAT\n",
            "AGCCTGTCCTGTTCTCAGATTATTGATTTT\n",
            "AGCCTGTCCTGTTCTCAGATTATTGATTTT\n",
            "AGCCTGTCCTGTTCTCAGATTATTGATTTT\n",
            "ATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "ATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "ATCTCAGAAGGGAGTTTCTTATTCACACCATCGCTGCAGCTCTTGT\n",
            "ATGCAAACTGCACTCAACATTTTTAAATGTTGTACTTTAAAAATTACAGATTGATACAACATCTTGATGTAACTTTGAAACTACTTACATCAATCACTTAATCATGTCTTGCACATGCTGCGTGGTGGCTATTTGTCAGAACACTGAGGGGTGAGGTTATTACATCATACTGAGAAGCCACCCAGTCTGAGGAAAGGGGGTTGGACAAAAGGGTCTTGGGACTCGTGATTTTTAGCAAAGCCAAAAGTATTTCTTTGGACCTTGCTTTCCTGCTGTCAAACAGATAGCAATACTGGACCTACCACCTTCACAACGCTGTTCTAAAGCCCACGTGAGATGCTCTGACAGGGACTAGGAAGTATAGAGTATTTTGAAGGCAACAATTGCCATTTTAAAACGAGCCAGTGCCTGCCTGGAATTGATTGATAGTTGATCTATGCAGTTTTTCTAAATCTTCCTCTTGGTAAGGAAATATACACTCCCTTGGTTCCTTAAAAAAATAAAAACAAAAACCTCCCCACTCCTCCCTCCCACAGCTTGTTCATGACATATTAAAAAATGCTTATCCCTTGAGGAGACTTAGAGAGGTCAAGGCATATCCCTGATGAACGTCCACCGAAACTAGTGGGAAAATGCCTAGTTTCTTCCTTCCTCAGGAAGAGAGGAAGTAGCAGAGACTTCAGAGGTCTTATGGGACTTAGGCAGAGGCAAACCATACATGACCTCTACAAACTGGGATATCTCCTGGCAAGTTGGTTTGCTCAGACCTCTGATCTCTGCTCAGGGAAACCAGGCCATCCGGACGACACACTGGTTCCCACAAATGCTACCATGACACGACACTGCTAATGTGACGTTTGTCCCATCTGCCTCATATTACTGCAGACTCTCATTATGCGTACTTTTGGGGACAGGGAAACCCATAGCCCTTCTATGCCTCATAGCAACCACTGCAAGGATCAGAGGACTCTTGGTGACCATTAGTACAGGGGGTATCTCAACTGACCCAGAAGTTTCTCCCTGTGCACCCTGACTGCATGCATCCACCCCGGCAGCTTTAAAATAGCTTGGGCCACAATCCAAGTGTAGAATTTAACCAGAATCCTTGAGAGTGGGAAGGAGTTGTTAGTATTTTTAGGCTCCTTGGGTGATTCCAAGTGCATCCGAGATAGAAAAGCACTGGTAGGAGCCAACACAGATGCAGGCATCTCCCCACGCGCCTTCAACATTCTCCCCTCTGCGAGGGCTCCGTGCATGGTCAGTCCATACGTGGTCCTCGAACACCCGTCTTAATTGTCATTGATCCAAGAGCTCTCCTTCCCCTACTCGGCAGGCTTAGTGTTTAAGTTGTCAGTTCCACAAACCGAGGAAGAGTACTATTGCCTACCCAATACTACCTTACTGAGCCCGGCCATTTGGATCACTGACTGCTATCCAATCTAGGAATTAGCTACACTGAGGGAAAACCAAGTCTTTCTTGCCCTTAAAATCCATGGAAAGGACCACAGAGGTTATACAAAATAAGAGAACCATTCTGCTCTGACACAGGACTGTATGAACATGTGCTAGGGTGGGTACCATTTCCCTTGATAGACACTGTAGAATACTTGCTACATCGATTTAATCACAAATAATAATATTGAAATAAAATGTATTGCTGAGACGATAAAAATTAAAGATTTGAGATTTTTTTTCTTTATAGAATGTGACCGATTCATGAGTGTTTTCAAGTTTATATTGAATACTGAAATTGTTATATTGAATACTAAAATACTTAAAGCCATCCAGATATATTTAAAATATTTATGTATCATTTTACAATCCCATGTTCATAGAAACCACTATCAAATTTAGAATCAAAAAGCTATGATAGTCATTATTAGCTCACCCAGGGAAATATTTTTTAGCTTTTATAACAAAAAATTTTAAAGAATGTATTTATTTTTATTTTATGTCTTTGTTGTTTTGGCTGCGTGTACGTCTGTATGGGGGTGCTGGAATCCCCAGAACTGGAGTTACAGACTGCTGTGAGCTGCCATGTGGATGCTGGGATTTGAACTGGGATCCTCTGGAAGAGCAGCCCTAACTGCTGAACCACCTCTCCAGCCCTATAAGAAAAAAAATTAATTGAGGTGTAATTATTCATAAAGAGAAGGACACCATCACAAGTATAGTCCTTCTCAGTGGTGTTGGTTTACAAATAAGCATCCTTGTGTCCAGCACCCAGGTTAAGGAACCAACCGTTATTCCCCCAGAGACAATCTTTGTGTTCACTTTCAGTTAGAACTACTACCCTCCCAGAAAAGCCACTGTGTCGATATCTGCCTGCATAGATTTGTTCAACTGGGTTTTTTTTTTTTAAGATTTATTTATTGTTATAAATAAGTACACTGTAGCTGTCTTTAGACACACCAGAAGAGGCCGTCAGATCTCATTATGGGTGGTTGTGAGCCACCATGTGGTTGCTGGGATTTGAACTCAGGACCTTTGGAAGAGCAGTCAGTGCTCTTACCCACTAAGCCATCTCGCCAGCCCCCAGCTGGGTTTTTTGTTGTTGTTTGTTTGTTTGTTTGTTTGTTTTTTCTGTTAGTCATGCAGAATGCACTCTGCCGAATTTTGACTTCTTTCCTTAGAAAATGTTGCTTTGTAGTTATAGTTTTAGCCAGAGATTTTAGTCTTTGAAATGTAGTTCAGCAAACATGAAAAAAAAAAAAACAGGAATTGGGATAGAGTTGGGGATGCAAAAGCAAATAAAGGCAGGTCCTTGCTTTAAGATTATATTCAATATCCTAACCTAGCAGTGACTAGAATGCAATGTAAAGCAAATGTCTCTTTGTTCAACAAATATTTGTTTAAAATTTACAATGGAATGGTTACTAGCACAAGTTCTAAAAGTAAGTACTGAAAAGGCCAACAAAAATTTTAGAAATCCTATCTTTCATCTCTGTGACAGGCATAGGCAAGGGATGCTCACTGGTATCCACTTTCCAGCCTGTCTTACTTTTGACTTAACGTAAGGGATGGAAGCTATTTCTTCCGGCAACTTGGGACTATGGGATGCTGAATTCCCAGCTGCGCCGCAGGGCATTTTGCAGACCATTGGAGAAGCTATCTGCAGCCACTGGATTCTACTCTCTCCATCCCTGTTTGCTGCCCTTAAGGAGAGAAACAAAACCCTGAAGCCTGCTCAAGCCTTTGGGTGTCAGCTTACCAAGGAGTAAGGCTGAGCATCAGGTTGCTTTCCAAAACCATAAAATGAGAAGAGACAATTACCAGAGAAGTGATGGAAGTCATGAACCTTGGACTTCATTTTAATAAGCACAGGCTGTAATGTGTGTTTGATTAGTTGCTCTGGGCTCTTCGGGGCCCAGTGAATTACTTGAGTAAAAAGAGCCATGACGTAAGAGGGAAGAGGGATTAGAAGTGAACTCAACACCATTCCAGGATTGATAGCTTGGGAAGGAACAGAAAAGGTCACTTCACTTCATTTGGCTGGCTCCTCCTCTGCCTCCATTCCTTGACAGGAAAGTCTCAAGTCCCTGGAAAAATAGAGCTGATAGGAAAATCCACACACCTGTGTTCCCATCCTTTGGCTTCGCTTACAGGCTCCTACTCCTCAGCTCATGTTGAACCCTCCCTTGATTACTGTATCCAATAAATTACTCATTTCAAACTACTTCAGAAAGTAAAAATCTCCAGGCTTAATCTCCTCTGTTAACAGGCTCAGGGGAGAAGTCACTTGGCTAGTTACTAGCAATTATATTTAACCACTACCCTCCCAAAGTAAAAATATCACAAGATTTACTAACAGCATTTCTATATTATTATTAAACTGTCTGACACACAAACTTATTATTTAGCCAAATGTGTATATATGTGTATATATATGCTTTAAAGAAATCTGGCAAGTTCTCCAAAGGGTCCAATGGGGTTTGGTGACAGATTGCCTCATGCCCTTTTCCTTGGGGATTTTGAAGCCAGGTTCCTGTGGATTTTGAAAATGCCAATAAATTTGCCCAAGTAAAAGTGTTTATCTTAATTATTTAGTAACAAAATGGCCTCAGATGTGCAAATGTCATCAGTTCCTATCTTCAGCCTGTCCTGTTCTCAGATTATTGATTTTTTTTTAAGCAGCCAAAAAGTTGGTGTTCACTGTTTATGACCACTGTCCACTACACCTGCTATGAGCTATTACACAATAAAGGTGGAGGTCCATTTTTTCCCTACATTTCCTAGTATTAAACTCCTGGTATTAATCATTAAGTAATGATTAAAGATCATTATTTTGGCCTGTCAAAATAAATTCCAGAAGTTTCTAGAGCATGTTTTCCCATGAAGATGACTGCCAGCGTCTTCGTTAATGGGACCACGAAAATGTTGATTTGGTCACACCAGCTGCCCACCTCCTCTGCTTGTGTCTCTCTCAAAATGCTTCCCAGGACGGATGGCACCTCACCCAAAGAGAGAAGCCTGCAAGCACGCCCTCAGTGAGCACGTCAGGAGGAGACATTCAGGAGTGAGAAAAGCTGCCAAGGTTTTCTGTAGAGAAACGCAGACTTACCTGTGTCAGCCTCCACCTGGTTCTCTGAGAGACCAAGGCAGGGGAGATGGGCGGCTGACAGTGTAACTCTTCTCACTGTGTGCACCTCTCTCTGGGCAGGTTATTTACGTCGAACTCCTTTGATGTGATCAGTGATGATGCTTTTATTGGTCTTCCACATCTAGAATATTTGT\n",
            "TCTCAGATTATTGATTTTTTTTTAAGCAGCCAAAAAGTTGGTGTTCACTGTTTATGACCACTGTCCACTACACCTGCTATGAGCTATTACACAATAAAGG\n",
            "TCTCAGATTATTGATTTTTTTTTAAGCAGCCAAAAAGTTGGTGTTCACTGTTTATGACCACTGTCCACTACACCTGCTATGAGCTATTACACAATAAAGG\n",
            "AGGTTATTTACGTCGAACTCCTTTGATGTGA\n",
            "AGGTTATTTACGTCGAACTCCTTTGATGTGA\n",
            "AGGTTATTTACGTCGAACTCCTTTGATGTGA\n",
            "AAACAACAATATCAAGTCCATTTCAAGACATACTTTCCGGGGACTCAAGTCTTTGATTCACTTGT\n",
            "AAACAACAATATCAAGTCCATTTCAAGACATACTTTCCGGGGACTCAAGTCTTTGATTCACTTGT\n",
            "AAACAACAATATCAAGTCCATTTCAAGACATACTTTCCGGGGACTCAAGTCTTTGATTCACTTGT\n",
            "Exon file type summaries for the first gene from: ENSG00000108231-ENSMUSG00000067242\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t24 exons with: Only nonintersecting\n",
            "\t10 exons with: Only unmapped\n",
            "\t5 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t34 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108231-ENSMUSG00000067242\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t27 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108231-ENSMUSG00000067242/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108231-ENSMUSG00000067242/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 30, Nonintersecting exons= 35, OTHER= 10\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 31, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108231-ENSMUSG00000067242/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108231-ENSMUSG00000067242/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108231-ENSMUSG00000067242/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108231-ENSMUSG00000067242/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108231-ENSMUSG00000067242/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3224\tENSG00000108239-ENSMUSG00000048720\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108239-ENSMUSG00000048720\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108239\tchr10\t94402541\t94536332\t+\tENSG00000108239\tTBC1D12\tprotein_coding\t3\t19\tgene\n",
            "\n",
            "ENSMUSG00000048720\tchr19\t38825023\t38908367\t+\tENSMUSG00000048720\tTbc1d12\tprotein_coding\t2\t15\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t19\t17\n",
            "\n",
            "Org2\t15\t15\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "AGAAACCTTTTCCCAAAAAGGACAAAGGAACTCAAATCAGTTGTCCATAGTGCTCCTGGCTGGAAGTTATTTGGTAAAGTCCCTCCCAGAGAAAATCTTCAGAAAACCTCCAAAATTATTCAGCAGGT\n",
            "AGAAACCTTTTCCCAAAAAGGACAAAGGAACTCAAATCAGTTGTCCATAGTGCTCCTGGCTGGAAGTTATTTGGTAAAGTCCCTCCCAGAGAAAATCTTCAGAAAACCTCCAAAATTATTCAGCAGGT\n",
            "AGAAACCTTTTCCCAAAAAGGACAAAGGAACTCAAATCAGTTGTCCATAGTGCTCCTGGCTGGAAGTTATTTGGTAAAGTCCCTCCCAGAGAAAATCTTCAGAAAACCTCCAAAATTATTCAGCAGGT\n",
            "AGGAGTACGAAGCTCGCACAGGGAGGACCTGTAAAGCAGCCCCCCAGTCTTCACGACGCAAGAGCTTTGCGCTGGAGCCACTTTCTACCACTGCCCTGATTCTGGAGGACCGACCACCGT\n",
            "AGGAGTACGAAGCTCGCACAGGGAGGACCTGTAAAGCAGCCCCCCAGTCTTCACGACGCAAGAGCTTTGCGCTGGAGCCACTTTCTACCACTGCCCTGATTCTGGAGGACCGACCACCGT\n",
            "AGGAGTACGAAGCTCGCACAGGGAGGACCTGTAAAGCAGCCCCCCAGTCTTCACGACGCAAGAGCTTTGCGCTGGAGCCACTTTCTACCACTGCCCTGATTCTGGAGGACCGACCACCGT\n",
            "AGGAATCTTCCTGCGAAATCTGTGGAAGAAGCGCTGCGTCACCGGCAGGAGTATGATGAGATGGTGGCCGAAGCTAAAAAACGAGGT\n",
            "AGGAATCTTCCTGCGAAATCTGTGGAAGAAGCGCTGCGTCACCGGCAGGAGTATGATGAGATGGTGGCCGAAGCTAAAAAACGAGGT\n",
            "AGGAATCTTCCTGCGAAATCTGTGGAAGAAGCGCTGCGTCACCGGCAGGAGTATGATGAGATGGTGGCCGAAGCTAAAAAACGAGGT\n",
            "AGAAATTAAAGAAGCACATAAAAGAAAAAGAATAATGAAAGAACGATTTAAGCAGGAAGAAAGCATTGCAAGTGCAATGGTAATTTGGATCAATGAAATACTGCCCAATTGGGAAGTAATGT\n",
            "AGAAATTAAAGAAGCACATAAAAGAAAAAGAATAATGAAAGAACGATTTAAGCAGGAAGAAAGCATTGCAAGTGCAATGGTAATTTGGATCAATGAAATACTGCCCAATTGGGAAGTAATGT\n",
            "AGAAATTAAAGAAGCACATAAAAGAAAAAGAATAATGAAAGAACGATTTAAGCAGGAAGAAAGCATTGCAAGTGCAATGGTAATTTGGATCAATGAAATACTGCCCAATTGGGAAGTAATGT\n",
            "AGGCGTAGTACCAGAAGAGTTCGAGAATTGTGGTGGCAGGGATTGCCCCCTAGTGTCCGTGGGAAAGTTTGGAGTCTAGCTGTAGGAAATGAACTAAATATCACTCCTGGT\n",
            "AGGCGTAGTACCAGAAGAGTTCGAGAATTGTGGTGGCAGGGATTGCCCCCTAGTGTCCGTGGGAAAGTTTGGAGTCTAGCTGTAGGAAATGAACTAAATATCACTCCTGGT\n",
            "AGGCGTAGTACCAGAAGAGTTCGAGAATTGTGGTGGCAGGGATTGCCCCCTAGTGTCCGTGGGAAAGTTTGGAGTCTAGCTGTAGGAAATGAACTAAATATCACTCCTGGT\n",
            "Exon file type summaries for the first gene from: ENSG00000108239-ENSMUSG00000048720\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t12 exons with: Only nonintersecting\n",
            "\t4 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "GAGTAAGGGCTCTAAAGTCAGTCAAATATGGTTCAAATCTTGGTTCCACCATAATATATTAGGTTTGTGACCTTGGGCATATTACTTAACATCTAGAAATCTTAGTTTTTCTTCACCTGTAAAATAGGGGTAAGAATATGAAAGGAGTGTTCTAACAGTTATTTGTGATAATATAAAACAATAGTGAAAGTGCATTTTGCTGAGAGTATGGATAATATCATAATTGTCACAAAATCAGAAAGCAAAATATACACTGAAATCA\n",
            "Exon file type summaries for the second gene from: ENSG00000108239-ENSMUSG00000048720\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t14 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108239-ENSMUSG00000048720/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108239-ENSMUSG00000048720/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 6, Nonintersecting exons= 5, OTHER= 8\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 14, Nonintersecting exons= 1, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108239-ENSMUSG00000048720/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108239-ENSMUSG00000048720/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108239-ENSMUSG00000048720/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108239-ENSMUSG00000048720/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108239-ENSMUSG00000048720/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3225\tENSG00000108242-ENSMUSG00000025002\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108242-ENSMUSG00000025002\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108242\tchr10\t94683729\t94736190\t+\tENSG00000108242\tCYP2C18\tprotein_coding\t2\t11\tgene\n",
            "\n",
            "ENSMUSG00000025002\tchr19\t38995463\t39031137\t+\tENSMUSG00000025002\tCyp2c55\tprotein_coding\t1\t9\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t11\t11\n",
            "\n",
            "Org2\t9\t9\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "AGCTAAAGTCCAGGAAGAGATAGAACGTGTAGTTGGCAGACACAGGAGCCCCTGCGTGCAGGACAGGAGCCACATGTGCCGTGGTGCATGAGACTCAGAGATATATTGTCCTTATCCCCACAAACCTGCCTCATTCAGTGACCTGTGATGCAAAATTCAGAAACTACTTCATTCCCAAGGT\n",
            "AGGGTACCACTGTGATTACATCCCTGACATCCATGTTGCATGATGACAAAGAATTTCCTAATCCAGAGAAGTTTGACCCTGGTTACTTTCTAGATGAAAGAGGAAACGTTAAGAAGAGTGACTACTTTGTGCCATTCTCAGCAGGT\n",
            "AGGGTACCACTGTGATTACATCCCTGACATCCATGTTGCATGATGACAAAGAATTTCCTAATCCAGAGAAGTTTGACCCTGGTTACTTTCTAGATGAAAGAGGAAACGTTAAGAAGAGTGACTACTTTGTGCCATTCTCAGCAGGT\n",
            "AGGGTACCACTGTGATTACATCCCTGACATCCATGTTGCATGATGACAAAGAATTTCCTAATCCAGAGAAGTTTGACCCTGGTTACTTTCTAGATGAAAGAGGAAACGTTAAGAAGAGTGACTACTTTGTGCCATTCTCAGCAGGT\n",
            "AGGAAAGCGGATGTGTGCAGGAGAAGGCCTGACTGGCATGGAGCTGTTTCTGTTCTTCACCATCATTTTACAGAATTTTAACTTGAAGCCTCTGGTTGATGTAAAGGACATTGATACAACACCAGTTGTCAGTGGATTTGGCCATGTGCCCCCTTTGTACCAGGCTCGTTTTATTCCTGTGTAAAGGGCAGACTCTCTATGGTCTCATCTGTAGTGTCTCCTCTGAGGGTCCTGCCTACTTCATTATTGGTACAATAGTGGCTTTCAGTCACATTTCTCCATTTCCTTC\n",
            "Exon file type summaries for the first gene from: ENSG00000108242-ENSMUSG00000025002\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t5 exons with: Only nonintersecting\n",
            "\t0 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "GCTACCAGGAAGTTCTGAATAGGATAGGGGTGTTTGCTGCTTCAGCAAGGTAATTAAGGTTAAGATGACAAAATGATTGGATTTTGGGGTGTATGAACCATTAAAAAATTACAGTTTTGGCTGTGCTGAGCTAGCATGTCCTGAACTCTGTCTAAAGGATAATTGTGTGATTATCTCAACTACTCACGGAAGTATCTTAACT\n",
            "GCTACCAGGAAGTTCTGAATAGGATAGGGGTGTTTGCTGCTTCAGCAAGGTAATTAAGGTTAAGATGACAAAATGATTGGATTTTGGGGTGTATGAACCATTAAAAAATTACAGTTTTGGCTGTGCTGAGCTAGCATGTCCTGAACTCTGTCTAAAGGATAATTGTGTGATTATCTCAACTACTCACGGAAGTATCTTAACT\n",
            "Exon file type summaries for the second gene from: ENSG00000108242-ENSMUSG00000025002\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t6 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108242-ENSMUSG00000025002/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108242-ENSMUSG00000025002/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 3, Nonintersecting exons= 3, OTHER= 5\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 8, Nonintersecting exons= 1, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108242-ENSMUSG00000025002/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108242-ENSMUSG00000025002/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108242-ENSMUSG00000025002/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108242-ENSMUSG00000025002/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108242-ENSMUSG00000025002/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3226\tENSG00000108244-ENSMUSG00000006777\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108244-ENSMUSG00000006777\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108244\tchr17\t40922700\t40937646\t-\tENSG00000108244\tKRT23\tprotein_coding\t9\t33\tgene\n",
            "\n",
            "ENSMUSG00000006777\tchr11\t99368800\t99383963\t-\tENSMUSG00000006777\tKrt23\tprotein_coding\t1\t8\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t33\t24\n",
            "\n",
            "Org2\t8\t8\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108244-ENSMUSG00000006777\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t16 exons with: Only nonintersecting\n",
            "\t0 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t17 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108244-ENSMUSG00000006777\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t4 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108244-ENSMUSG00000006777/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108244-ENSMUSG00000006777/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 17, Nonintersecting exons= 0, OTHER= 16\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 5, Nonintersecting exons= 0, OTHER= 3\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108244-ENSMUSG00000006777/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108244-ENSMUSG00000006777/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108244-ENSMUSG00000006777/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108244-ENSMUSG00000006777/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108244-ENSMUSG00000006777/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3227\tENSG00000108255-ENSMUSG00000000724\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108255-ENSMUSG00000000724\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108255\tchr17\t29246859\t29254494\t+\tENSG00000108255\tCRYBA1\tprotein_coding\t2\t9\tgene\n",
            "\n",
            "ENSMUSG00000000724\tchr11\t77609441\t77616109\t-\tENSMUSG00000000724\tCryba1\tprotein_coding\t2\t9\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t9\t7\n",
            "\n",
            "Org2\t9\t8\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108255-ENSMUSG00000000724\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t7 exons with: Only nonintersecting\n",
            "\t0 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108255-ENSMUSG00000000724\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t9 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108255-ENSMUSG00000000724/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108255-ENSMUSG00000000724/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 2, Nonintersecting exons= 0, OTHER= 7\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 9, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108255-ENSMUSG00000000724/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108255-ENSMUSG00000000724/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108255-ENSMUSG00000000724/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108255-ENSMUSG00000000724/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108255-ENSMUSG00000000724/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3228\tENSG00000108256-ENSMUSG00000037857\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108256-ENSMUSG00000037857\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108256\tchr17\t29255839\t29294148\t-\tENSG00000108256\tNUFIP2\tprotein_coding\t2\t6\tgene\n",
            "\n",
            "ENSMUSG00000037857\tchr11\t77576981\t77632747\t+\tENSMUSG00000037857\tNufip2\tprotein_coding\t4\t11\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t6\t6\n",
            "\n",
            "Org2\t11\t11\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108256-ENSMUSG00000037857\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t2 exons with: Only nonintersecting\n",
            "\t1 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108256-ENSMUSG00000037857\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t9 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108256-ENSMUSG00000037857/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108256-ENSMUSG00000037857/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 2, Nonintersecting exons= 0, OTHER= 4\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 10, Nonintersecting exons= 0, OTHER= 1\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108256-ENSMUSG00000037857/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108256-ENSMUSG00000037857/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108256-ENSMUSG00000037857/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108256-ENSMUSG00000037857/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108256-ENSMUSG00000037857/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3229\tENSG00000108262-ENSMUSG00000011877\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108262-ENSMUSG00000011877\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108262\tchr17\t29573475\t29594054\t-\tENSG00000108262\tGIT1\tprotein_coding\t15\t67\tgene\n",
            "\n",
            "ENSMUSG00000011877\tchr11\t77384388\t77398612\t+\tENSMUSG00000011877\tGit1\tprotein_coding\t6\t43\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t67\t50\n",
            "\n",
            "Org2\t43\t31\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108262-ENSMUSG00000011877\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t48 exons with: Only nonintersecting\n",
            "\t4 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t15 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108262-ENSMUSG00000011877\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t4 exons with: Only nonintersecting\n",
            "\t31 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t8 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108262-ENSMUSG00000011877/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108262-ENSMUSG00000011877/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 19, Nonintersecting exons= 0, OTHER= 48\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 39, Nonintersecting exons= 0, OTHER= 4\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108262-ENSMUSG00000011877/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108262-ENSMUSG00000011877/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108262-ENSMUSG00000011877/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108262-ENSMUSG00000011877/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108262-ENSMUSG00000011877/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3230\tENSG00000108298-ENSMUSG00000017404\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108298-ENSMUSG00000017404\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108298\tchr17\t39200283\t39204840\t+\tENSG00000108298\tRPL19\tprotein_coding\t13\t39\tgene\n",
            "\n",
            "ENSMUSG00000017404\tchr11\t97917536\t97921318\t+\tENSMUSG00000017404\tRpl19\tprotein_coding\t3\t10\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t39\t32\n",
            "\n",
            "Org2\t10\t9\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108298-ENSMUSG00000017404\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t23 exons with: Only nonintersecting\n",
            "\t7 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t9 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108298-ENSMUSG00000017404\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t8 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108298-ENSMUSG00000017404/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108298-ENSMUSG00000017404/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 16, Nonintersecting exons= 0, OTHER= 23\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 10, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108298-ENSMUSG00000017404/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108298-ENSMUSG00000017404/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108298-ENSMUSG00000017404/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108298-ENSMUSG00000017404/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108298-ENSMUSG00000017404/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3231\tENSG00000108306-ENSMUSG00000020883\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108306-ENSMUSG00000020883\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108306\tchr17\t39252663\t39402523\t-\tENSG00000108306\tFBXL20\tprotein_coding\t6\t25\tgene\n",
            "\n",
            "ENSMUSG00000020883\tchr11\t97973382\t98041229\t-\tENSMUSG00000020883\tFbxl20\tprotein_coding\t7\t36\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t25\t25\n",
            "\n",
            "Org2\t36\t27\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108306-ENSMUSG00000020883\n",
            "\t0 exons with: No file exists\n",
            "\t4 exons with: Only Mapped\n",
            "\t12 exons with: Only nonintersecting\n",
            "\t3 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108306-ENSMUSG00000020883\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t5 exons with: Only nonintersecting\n",
            "\t22 exons with: Only unmapped\n",
            "\t7 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108306-ENSMUSG00000020883/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108306-ENSMUSG00000020883/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 9, Nonintersecting exons= 0, OTHER= 16\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 29, Nonintersecting exons= 0, OTHER= 7\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108306-ENSMUSG00000020883/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108306-ENSMUSG00000020883/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108306-ENSMUSG00000020883/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108306-ENSMUSG00000020883/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108306-ENSMUSG00000020883/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3232\tENSG00000108309-ENSMUSG00000006575\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108309-ENSMUSG00000006575\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108309\tchr17\t44308413\t44318670\t+\tENSG00000108309\tRUNDC3A\tprotein_coding\t7\t36\tgene\n",
            "\n",
            "ENSMUSG00000006575\tchr11\t102284229\t102293381\t+\tENSMUSG00000006575\tRundc3a\tprotein_coding\t10\t35\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t36\t26\n",
            "\n",
            "Org2\t35\t27\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108309-ENSMUSG00000006575\n",
            "\t0 exons with: No file exists\n",
            "\t3 exons with: Only Mapped\n",
            "\t18 exons with: Only nonintersecting\n",
            "\t5 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t10 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108309-ENSMUSG00000006575\n",
            "\t0 exons with: No file exists\n",
            "\t7 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t1 exons with: Only unmapped\n",
            "\t24 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108309-ENSMUSG00000006575/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108309-ENSMUSG00000006575/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 15, Nonintersecting exons= 0, OTHER= 21\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 28, Nonintersecting exons= 0, OTHER= 7\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108309-ENSMUSG00000006575/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108309-ENSMUSG00000006575/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108309-ENSMUSG00000006575/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108309-ENSMUSG00000006575/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108309-ENSMUSG00000006575/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3233\tENSG00000108312-ENSMUSG00000020923\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108312-ENSMUSG00000020923\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108312\tchr17\t44205033\t44221626\t-\tENSG00000108312\tUBTF\tprotein_coding\t18\t64\tgene\n",
            "\n",
            "ENSMUSG00000020923\tchr11\t102195386\t102210568\t-\tENSMUSG00000020923\tUbtf\tprotein_coding\t13\t45\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t64\t56\n",
            "\n",
            "Org2\t45\t41\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108312-ENSMUSG00000020923\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t43 exons with: Only nonintersecting\n",
            "\t6 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t15 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108312-ENSMUSG00000020923\n",
            "\t0 exons with: No file exists\n",
            "\t7 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t15 exons with: Only unmapped\n",
            "\t14 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t9 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108312-ENSMUSG00000020923/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108312-ENSMUSG00000020923/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 21, Nonintersecting exons= 0, OTHER= 43\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 38, Nonintersecting exons= 0, OTHER= 7\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108312-ENSMUSG00000020923/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108312-ENSMUSG00000020923/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108312-ENSMUSG00000020923/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108312-ENSMUSG00000020923/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108312-ENSMUSG00000020923/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3234\tENSG00000108342-ENSMUSG00000038067\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108342-ENSMUSG00000038067\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108342\tchr17\t40015361\t40017813\t+\tENSG00000108342\tCSF3\tprotein_coding\t9\t23\tgene\n",
            "\n",
            "ENSMUSG00000038067\tchr11\t98592089\t98594455\t+\tENSMUSG00000038067\tCsf3\tprotein_coding\t1\t5\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t23\t20\n",
            "\n",
            "Org2\t5\t5\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108342-ENSMUSG00000038067\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t9 exons with: Only nonintersecting\n",
            "\t7 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108342-ENSMUSG00000038067\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t4 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108342-ENSMUSG00000038067/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108342-ENSMUSG00000038067/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 14, Nonintersecting exons= 0, OTHER= 9\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 5, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108342-ENSMUSG00000038067/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108342-ENSMUSG00000038067/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108342-ENSMUSG00000038067/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108342-ENSMUSG00000038067/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108342-ENSMUSG00000038067/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3235\tENSG00000108344-ENSMUSG00000017221\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108344-ENSMUSG00000017221\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108344\tchr17\t39980807\t39997959\t+\tENSG00000108344\tPSMD3\tprotein_coding\t5\t32\tgene\n",
            "\n",
            "ENSMUSG00000017221\tchr11\t98573380\t98586805\t+\tENSMUSG00000017221\tPsmd3\tprotein_coding\t5\t27\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t32\t23\n",
            "\n",
            "Org2\t27\t22\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108344-ENSMUSG00000017221\n",
            "\t0 exons with: No file exists\n",
            "\t4 exons with: Only Mapped\n",
            "\t19 exons with: Only nonintersecting\n",
            "\t2 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108344-ENSMUSG00000017221\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t9 exons with: Only unmapped\n",
            "\t9 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108344-ENSMUSG00000017221/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108344-ENSMUSG00000017221/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 9, Nonintersecting exons= 0, OTHER= 23\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 24, Nonintersecting exons= 0, OTHER= 3\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108344-ENSMUSG00000017221/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108344-ENSMUSG00000017221/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108344-ENSMUSG00000017221/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108344-ENSMUSG00000017221/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108344-ENSMUSG00000017221/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3236\tENSG00000108349-ENSMUSG00000078676\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108349-ENSMUSG00000078676\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108349\tchr17\t40140318\t40172171\t+\tENSG00000108349\tCASC3\tprotein_coding\t10\t43\tgene\n",
            "\n",
            "ENSMUSG00000078676\tchr11\t98695731\t98724640\t+\tENSMUSG00000078676\tCasc3\tprotein_coding\t4\t25\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t43\t34\n",
            "\n",
            "Org2\t25\t21\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108349-ENSMUSG00000078676\n",
            "\t0 exons with: No file exists\n",
            "\t3 exons with: Only Mapped\n",
            "\t15 exons with: Only nonintersecting\n",
            "\t6 exons with: Only unmapped\n",
            "\t6 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t13 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108349-ENSMUSG00000078676\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t15 exons with: Only unmapped\n",
            "\t4 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108349-ENSMUSG00000078676/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108349-ENSMUSG00000078676/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 25, Nonintersecting exons= 0, OTHER= 18\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 23, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108349-ENSMUSG00000078676/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108349-ENSMUSG00000078676/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108349-ENSMUSG00000078676/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108349-ENSMUSG00000078676/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108349-ENSMUSG00000078676/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3237\tENSG00000108352-ENSMUSG00000038020\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108352-ENSMUSG00000038020\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108352\tchr17\t40177010\t40195656\t+\tENSG00000108352\tRAPGEFL1\tprotein_coding\t11\t50\tgene\n",
            "\n",
            "ENSMUSG00000038020\tchr11\t98727611\t98744523\t+\tENSMUSG00000038020\tRapgefl1\tprotein_coding\t3\t22\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t50\t39\n",
            "\n",
            "Org2\t22\t19\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108352-ENSMUSG00000038020\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t34 exons with: Only nonintersecting\n",
            "\t6 exons with: Only unmapped\n",
            "\t3 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108352-ENSMUSG00000038020\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t12 exons with: Only unmapped\n",
            "\t5 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108352-ENSMUSG00000038020/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108352-ENSMUSG00000038020/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 15, Nonintersecting exons= 0, OTHER= 35\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 21, Nonintersecting exons= 0, OTHER= 1\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108352-ENSMUSG00000038020/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108352-ENSMUSG00000038020/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108352-ENSMUSG00000038020/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108352-ENSMUSG00000038020/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108352-ENSMUSG00000038020/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3238\tENSG00000108370-ENSMUSG00000020599\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108370-ENSMUSG00000020599\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108370\tchr17\t65100812\t65227703\t+\tENSG00000108370\tRGS9\tprotein_coding\t12\t65\tgene\n",
            "\n",
            "ENSMUSG00000020599\tchr11\t109116181\t109188955\t-\tENSMUSG00000020599\tRgs9\tprotein_coding\t5\t31\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t65\t45\n",
            "\n",
            "Org2\t31\t28\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108370-ENSMUSG00000020599\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t43 exons with: Only nonintersecting\n",
            "\t8 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t14 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108370-ENSMUSG00000020599\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t21 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t9 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108370-ENSMUSG00000020599/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108370-ENSMUSG00000020599/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 22, Nonintersecting exons= 0, OTHER= 43\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 30, Nonintersecting exons= 0, OTHER= 1\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108370-ENSMUSG00000020599/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108370-ENSMUSG00000020599/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108370-ENSMUSG00000020599/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108370-ENSMUSG00000020599/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108370-ENSMUSG00000020599/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3239\tENSG00000108375-ENSMUSG00000034177\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108375-ENSMUSG00000034177\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108375\tchr17\t58353676\t58417595\t-\tENSG00000108375\tRNF43\tprotein_coding\t8\t24\tgene\n",
            "\n",
            "ENSMUSG00000034177\tchr11\t87553548\t87626365\t+\tENSMUSG00000034177\tRnf43\tprotein_coding\t9\t32\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t24\t22\n",
            "\n",
            "Org2\t32\t24\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108375-ENSMUSG00000034177\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t12 exons with: Only nonintersecting\n",
            "\t2 exons with: Only unmapped\n",
            "\t4 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108375-ENSMUSG00000034177\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t3 exons with: Only nonintersecting\n",
            "\t15 exons with: Only unmapped\n",
            "\t4 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t10 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108375-ENSMUSG00000034177/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108375-ENSMUSG00000034177/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 12, Nonintersecting exons= 0, OTHER= 12\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 29, Nonintersecting exons= 0, OTHER= 3\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108375-ENSMUSG00000034177/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108375-ENSMUSG00000034177/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108375-ENSMUSG00000034177/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108375-ENSMUSG00000034177/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108375-ENSMUSG00000034177/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3240\tENSG00000108379-ENSMUSG00000000125\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108379-ENSMUSG00000000125\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108379\tchr17\t46762506\t46833154\t-\tENSG00000108379\tWNT3\tprotein_coding\t8\t25\tgene\n",
            "\n",
            "ENSMUSG00000000125\tchr11\t103664976\t103708783\t+\tENSMUSG00000000125\tWnt3\tprotein_coding\t1\t5\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t25\t22\n",
            "\n",
            "Org2\t5\t5\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108379-ENSMUSG00000000125\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t8 exons with: Only nonintersecting\n",
            "\t14 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108379-ENSMUSG00000000125\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t4 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108379-ENSMUSG00000000125/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108379-ENSMUSG00000000125/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 17, Nonintersecting exons= 0, OTHER= 8\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 5, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108379-ENSMUSG00000000125/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108379-ENSMUSG00000000125/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108379-ENSMUSG00000000125/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108379-ENSMUSG00000000125/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108379-ENSMUSG00000000125/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3241\tENSG00000108381-ENSMUSG00000020774\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108381-ENSMUSG00000020774\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108381\tchr17\t3472374\t3503405\t+\tENSG00000108381\tASPA\tprotein_coding\t4\t15\tgene\n",
            "\n",
            "ENSMUSG00000020774\tchr11\t73195818\t73220422\t-\tENSMUSG00000020774\tAspa\tprotein_coding\t6\t18\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t15\t13\n",
            "\n",
            "Org2\t18\t17\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108381-ENSMUSG00000020774\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t8 exons with: Only nonintersecting\n",
            "\t3 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108381-ENSMUSG00000020774\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t12 exons with: Only unmapped\n",
            "\t4 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108381-ENSMUSG00000020774/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108381-ENSMUSG00000020774/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 7, Nonintersecting exons= 0, OTHER= 8\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 16, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108381-ENSMUSG00000020774/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108381-ENSMUSG00000020774/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108381-ENSMUSG00000020774/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108381-ENSMUSG00000020774/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108381-ENSMUSG00000020774/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3242\tENSG00000108384-ENSMUSG00000007646\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108384-ENSMUSG00000007646\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108384\tchr17\t58692573\t58735611\t+\tENSG00000108384\tRAD51C\tprotein_coding\t38\t93\tgene\n",
            "\n",
            "ENSMUSG00000007646\tchr11\t87267471\t87295780\t-\tENSMUSG00000007646\tRad51c\tprotein_coding\t4\t21\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t93\t78\n",
            "\n",
            "Org2\t21\t16\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108384-ENSMUSG00000007646\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t27 exons with: Only nonintersecting\n",
            "\t43 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t21 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108384-ENSMUSG00000007646\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t21 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108384-ENSMUSG00000007646/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108384-ENSMUSG00000007646/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 64, Nonintersecting exons= 0, OTHER= 29\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 21, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108384-ENSMUSG00000007646/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108384-ENSMUSG00000007646/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108384-ENSMUSG00000007646/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108384-ENSMUSG00000007646/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108384-ENSMUSG00000007646/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3243\tENSG00000108389-ENSMUSG00000018401\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108389-ENSMUSG00000018401\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108389\tchr17\t58489529\t58517905\t-\tENSG00000108389\tMTMR4\tprotein_coding\t10\t44\tgene\n",
            "\n",
            "ENSMUSG00000018401\tchr11\t87482988\t87507128\t+\tENSMUSG00000018401\tMtmr4\tprotein_coding\t6\t28\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t44\t37\n",
            "\n",
            "Org2\t28\t28\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108389-ENSMUSG00000018401\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t31 exons with: Only nonintersecting\n",
            "\t4 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t9 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108389-ENSMUSG00000018401\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t26 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108389-ENSMUSG00000018401/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108389-ENSMUSG00000018401/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 13, Nonintersecting exons= 0, OTHER= 31\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 28, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108389-ENSMUSG00000018401/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108389-ENSMUSG00000018401/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108389-ENSMUSG00000018401/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108389-ENSMUSG00000018401/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108389-ENSMUSG00000018401/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3244\tENSG00000108395-ENSMUSG00000018548\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108395-ENSMUSG00000018548\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108395\tchr17\t58982638\t59106921\t-\tENSG00000108395\tTRIM37\tprotein_coding\t14\t77\tgene\n",
            "\n",
            "ENSMUSG00000018548\tchr11\t87017903\t87111509\t+\tENSMUSG00000018548\tTrim37\tprotein_coding\t4\t34\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t77\t53\n",
            "\n",
            "Org2\t34\t29\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108395-ENSMUSG00000018548\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t53 exons with: Only nonintersecting\n",
            "\t7 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t17 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108395-ENSMUSG00000018548\n",
            "\t0 exons with: No file exists\n",
            "\t3 exons with: Only Mapped\n",
            "\t2 exons with: Only nonintersecting\n",
            "\t26 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108395-ENSMUSG00000018548/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108395-ENSMUSG00000018548/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 24, Nonintersecting exons= 0, OTHER= 53\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 29, Nonintersecting exons= 0, OTHER= 5\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108395-ENSMUSG00000018548/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108395-ENSMUSG00000018548/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108395-ENSMUSG00000018548/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108395-ENSMUSG00000018548/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108395-ENSMUSG00000018548/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3245\tENSG00000108405-ENSMUSG00000020787\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108405-ENSMUSG00000020787\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108405\tchr17\t3896592\t3916476\t-\tENSG00000108405\tP2RX1\tprotein_coding\t4\t28\tgene\n",
            "\n",
            "ENSMUSG00000020787\tchr11\t72889929\t72906026\t+\tENSMUSG00000020787\tP2rx1\tprotein_coding\t3\t20\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t28\t19\n",
            "\n",
            "Org2\t20\t16\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108405-ENSMUSG00000020787\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t18 exons with: Only nonintersecting\n",
            "\t3 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108405-ENSMUSG00000020787\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t16 exons with: Only unmapped\n",
            "\t4 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108405-ENSMUSG00000020787/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108405-ENSMUSG00000020787/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 8, Nonintersecting exons= 0, OTHER= 20\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 20, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108405-ENSMUSG00000020787/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108405-ENSMUSG00000020787/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108405-ENSMUSG00000020787/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108405-ENSMUSG00000020787/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108405-ENSMUSG00000020787/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3246\tENSG00000108406-ENSMUSG00000018425\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108406-ENSMUSG00000018425\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108406\tchr17\t59565558\t59608345\t+\tENSG00000108406\tDHX40\tprotein_coding\t10\t43\tgene\n",
            "\n",
            "ENSMUSG00000018425\tchr11\t86659672\t86698572\t-\tENSMUSG00000018425\tDhx40\tprotein_coding\t4\t27\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t43\t36\n",
            "\n",
            "Org2\t27\t23\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108406-ENSMUSG00000018425\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t37 exons with: Only nonintersecting\n",
            "\t1 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108406-ENSMUSG00000018425\n",
            "\t0 exons with: No file exists\n",
            "\t7 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t17 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108406-ENSMUSG00000018425/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108406-ENSMUSG00000018425/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 6, Nonintersecting exons= 0, OTHER= 37\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 20, Nonintersecting exons= 0, OTHER= 7\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108406-ENSMUSG00000018425/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108406-ENSMUSG00000018425/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108406-ENSMUSG00000018425/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108406-ENSMUSG00000018425/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108406-ENSMUSG00000018425/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3247\tENSG00000108423-ENSMUSG00000020513\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108423-ENSMUSG00000020513\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108423\tchr17\t59859479\t59892945\t-\tENSG00000108423\tTUBD1\tprotein_coding\t16\t45\tgene\n",
            "\n",
            "ENSMUSG00000020513\tchr11\t86435817\t86458186\t+\tENSMUSG00000020513\tTubd1\tprotein_coding\t5\t13\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t45\t35\n",
            "\n",
            "Org2\t13\t13\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108423-ENSMUSG00000020513\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t31 exons with: Only nonintersecting\n",
            "\t10 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108423-ENSMUSG00000020513\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t11 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108423-ENSMUSG00000020513/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108423-ENSMUSG00000020513/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 14, Nonintersecting exons= 0, OTHER= 31\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 12, Nonintersecting exons= 0, OTHER= 1\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108423-ENSMUSG00000020513/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108423-ENSMUSG00000020513/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108423-ENSMUSG00000020513/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108423-ENSMUSG00000020513/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108423-ENSMUSG00000020513/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3248\tENSG00000108424-ENSMUSG00000001440\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108424-ENSMUSG00000001440\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108424\tchr17\t47649476\t47685505\t+\tENSG00000108424\tKPNB1\tprotein_coding\t16\t72\tgene\n",
            "\n",
            "ENSMUSG00000001440\tchr11\t97050540\t97078707\t-\tENSMUSG00000001440\tKpnb1\tprotein_coding\t3\t28\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t72\t53\n",
            "\n",
            "Org2\t28\t26\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108424-ENSMUSG00000001440\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t50 exons with: Only nonintersecting\n",
            "\t13 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t9 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108424-ENSMUSG00000001440\n",
            "\t0 exons with: No file exists\n",
            "\t3 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t15 exons with: Only unmapped\n",
            "\t8 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108424-ENSMUSG00000001440/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108424-ENSMUSG00000001440/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 22, Nonintersecting exons= 0, OTHER= 50\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 24, Nonintersecting exons= 0, OTHER= 4\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108424-ENSMUSG00000001440/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108424-ENSMUSG00000001440/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108424-ENSMUSG00000001440/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108424-ENSMUSG00000001440/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108424-ENSMUSG00000001440/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3249\tENSG00000108433-ENSMUSG00000020946\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108433-ENSMUSG00000020946\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108433\tchr17\t46923075\t46975524\t+\tENSG00000108433\tGOSR2\tprotein_coding\t47\t113\tgene\n",
            "\n",
            "ENSMUSG00000020946\tchr11\t103567675\t103588724\t-\tENSMUSG00000020946\tGosr2\tprotein_coding\t3\t13\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t113\t99\n",
            "\n",
            "Org2\t13\t10\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108433-ENSMUSG00000020946\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t26 exons with: Only nonintersecting\n",
            "\t70 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t15 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108433-ENSMUSG00000020946\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t13 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108433-ENSMUSG00000020946/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108433-ENSMUSG00000020946/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 87, Nonintersecting exons= 0, OTHER= 26\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 13, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108433-ENSMUSG00000020946/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108433-ENSMUSG00000020946/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108433-ENSMUSG00000020946/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108433-ENSMUSG00000020946/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108433-ENSMUSG00000020946/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3250\tENSG00000108439-ENSMUSG00000018659\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108439-ENSMUSG00000018659\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108439\tchr17\t47941506\t47949308\t+\tENSG00000108439\tPNPO\tprotein_coding\t17\t49\tgene\n",
            "\n",
            "ENSMUSG00000018659\tchr11\t96828651\t96834812\t-\tENSMUSG00000018659\tPnpo\tprotein_coding\t3\t16\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t49\t42\n",
            "\n",
            "Org2\t16\t11\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108439-ENSMUSG00000018659\n",
            "\t0 exons with: No file exists\n",
            "\t17 exons with: Only Mapped\n",
            "\t7 exons with: Only nonintersecting\n",
            "\t10 exons with: Only unmapped\n",
            "\t13 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108439-ENSMUSG00000018659\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t6 exons with: Only nonintersecting\n",
            "\t9 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108439-ENSMUSG00000018659/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108439-ENSMUSG00000018659/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 25, Nonintersecting exons= 0, OTHER= 24\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 10, Nonintersecting exons= 0, OTHER= 6\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108439-ENSMUSG00000018659/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108439-ENSMUSG00000018659/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108439-ENSMUSG00000018659/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108439-ENSMUSG00000018659/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108439-ENSMUSG00000018659/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3251\tENSG00000108443-ENSMUSG00000020516\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108443-ENSMUSG00000020516\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108443\tchr17\t59893046\t59950574\t+\tENSG00000108443\tRPS6KB1\tprotein_coding\t12\t55\tgene\n",
            "\n",
            "ENSMUSG00000020516\tchr11\t86389697\t86435631\t-\tENSMUSG00000020516\tRps6kb1\tprotein_coding\t7\t45\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t55\t42\n",
            "\n",
            "Org2\t45\t29\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108443-ENSMUSG00000020516\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t35 exons with: Only nonintersecting\n",
            "\t6 exons with: Only unmapped\n",
            "\t3 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t9 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108443-ENSMUSG00000020516\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t9 exons with: Only nonintersecting\n",
            "\t32 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t1 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108443-ENSMUSG00000020516/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108443-ENSMUSG00000020516/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 18, Nonintersecting exons= 0, OTHER= 37\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 35, Nonintersecting exons= 0, OTHER= 10\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108443-ENSMUSG00000020516/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108443-ENSMUSG00000020516/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108443-ENSMUSG00000020516/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108443-ENSMUSG00000020516/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108443-ENSMUSG00000020516/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3252\tENSG00000108465-ENSMUSG00000018669\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108465-ENSMUSG00000018669\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108465\tchr17\t47967810\t47981781\t+\tENSG00000108465\tCDK5RAP3\tprotein_coding\t37\t114\tgene\n",
            "\n",
            "ENSMUSG00000018669\tchr11\t96798252\t96807322\t-\tENSMUSG00000018669\tCdk5rap3\tprotein_coding\t11\t48\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t114\t89\n",
            "\n",
            "Org2\t48\t33\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108465-ENSMUSG00000018669\n",
            "\t0 exons with: No file exists\n",
            "\t21 exons with: Only Mapped\n",
            "\t28 exons with: Only nonintersecting\n",
            "\t33 exons with: Only unmapped\n",
            "\t8 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t24 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108465-ENSMUSG00000018669\n",
            "\t0 exons with: No file exists\n",
            "\t19 exons with: Only Mapped\n",
            "\t3 exons with: Only nonintersecting\n",
            "\t22 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108465-ENSMUSG00000018669/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108465-ENSMUSG00000018669/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 65, Nonintersecting exons= 0, OTHER= 49\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 26, Nonintersecting exons= 0, OTHER= 22\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108465-ENSMUSG00000018669/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108465-ENSMUSG00000018669/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108465-ENSMUSG00000018669/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108465-ENSMUSG00000018669/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108465-ENSMUSG00000018669/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3253\tENSG00000108468-ENSMUSG00000018666\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108468-ENSMUSG00000018666\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108468\tchr17\t48070052\t48101478\t-\tENSG00000108468\tCBX1\tprotein_coding\t6\t18\tgene\n",
            "\n",
            "ENSMUSG00000018666\tchr11\t96679953\t96699466\t+\tENSMUSG00000018666\tCbx1\tprotein_coding\t5\t15\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t18\t18\n",
            "\n",
            "Org2\t15\t15\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108468-ENSMUSG00000018666\n",
            "\t0 exons with: No file exists\n",
            "\t5 exons with: Only Mapped\n",
            "\t5 exons with: Only nonintersecting\n",
            "\t2 exons with: Only unmapped\n",
            "\t3 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108468-ENSMUSG00000018666\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t15 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108468-ENSMUSG00000018666/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108468-ENSMUSG00000018666/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 8, Nonintersecting exons= 0, OTHER= 10\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 15, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108468-ENSMUSG00000018666/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108468-ENSMUSG00000018666/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108468-ENSMUSG00000018666/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108468-ENSMUSG00000018666/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108468-ENSMUSG00000018666/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3254\tENSG00000108469-ENSMUSG00000020752\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108469-ENSMUSG00000020752\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108469\tchr17\t75626845\t75667189\t-\tENSG00000108469\tRECQL5\tprotein_coding\t19\t72\tgene\n",
            "\n",
            "ENSMUSG00000020752\tchr11\t115783421\t115824303\t-\tENSMUSG00000020752\tRecql5\tprotein_coding\t8\t53\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t72\t59\n",
            "\n",
            "Org2\t53\t33\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108469-ENSMUSG00000020752\n",
            "\t0 exons with: No file exists\n",
            "\t5 exons with: Only Mapped\n",
            "\t36 exons with: Only nonintersecting\n",
            "\t17 exons with: Only unmapped\n",
            "\t4 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t10 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108469-ENSMUSG00000020752\n",
            "\t0 exons with: No file exists\n",
            "\t9 exons with: Only Mapped\n",
            "\t4 exons with: Only nonintersecting\n",
            "\t40 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108469-ENSMUSG00000020752/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108469-ENSMUSG00000020752/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 31, Nonintersecting exons= 0, OTHER= 41\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 40, Nonintersecting exons= 0, OTHER= 13\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108469-ENSMUSG00000020752/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108469-ENSMUSG00000020752/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108469-ENSMUSG00000020752/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108469-ENSMUSG00000020752/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108469-ENSMUSG00000020752/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3255\tENSG00000108474-ENSMUSG00000014245\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108474-ENSMUSG00000014245\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108474\tchr17\t16217191\t16351797\t+\tENSG00000108474\tPIGL\tprotein_coding\t17\t46\tgene\n",
            "\n",
            "ENSMUSG00000014245\tchr11\t62349286\t62405243\t+\tENSMUSG00000014245\tPigl\tprotein_coding\t2\t14\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t46\t37\n",
            "\n",
            "Org2\t14\t10\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108474-ENSMUSG00000014245\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t23 exons with: Only nonintersecting\n",
            "\t19 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108474-ENSMUSG00000014245\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t2 exons with: Only nonintersecting\n",
            "\t6 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108474-ENSMUSG00000014245/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108474-ENSMUSG00000014245/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 23, Nonintersecting exons= 0, OTHER= 23\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 10, Nonintersecting exons= 0, OTHER= 4\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108474-ENSMUSG00000014245/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108474-ENSMUSG00000014245/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108474-ENSMUSG00000014245/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108474-ENSMUSG00000014245/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108474-ENSMUSG00000014245/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3256\tENSG00000108479-ENSMUSG00000020766\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108479-ENSMUSG00000020766\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108479\tchr17\t75751594\t75765236\t-\tENSG00000108479\tGALK1\tprotein_coding\t9\t29\tgene\n",
            "\n",
            "ENSMUSG00000020766\tchr11\t115899283\t115903545\t-\tENSMUSG00000020766\tGalk1\tprotein_coding\t1\t8\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t29\t26\n",
            "\n",
            "Org2\t8\t8\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108479-ENSMUSG00000020766\n",
            "\t0 exons with: No file exists\n",
            "\t3 exons with: Only Mapped\n",
            "\t16 exons with: Only nonintersecting\n",
            "\t3 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108479-ENSMUSG00000020766\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t5 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108479-ENSMUSG00000020766/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108479-ENSMUSG00000020766/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 10, Nonintersecting exons= 0, OTHER= 19\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 8, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108479-ENSMUSG00000020766/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108479-ENSMUSG00000020766/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108479-ENSMUSG00000020766/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108479-ENSMUSG00000020766/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108479-ENSMUSG00000020766/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3257\tENSG00000108506-ENSMUSG00000018068\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108506-ENSMUSG00000018068\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108506\tchr17\t61865367\t61928016\t-\tENSG00000108506\tINTS2\tprotein_coding\t10\t62\tgene\n",
            "\n",
            "ENSMUSG00000018068\tchr11\t86101507\t86148401\t-\tENSMUSG00000018068\tInts2\tprotein_coding\t11\t61\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t62\t40\n",
            "\n",
            "Org2\t61\t47\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108506-ENSMUSG00000018068\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t46 exons with: Only nonintersecting\n",
            "\t9 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t7 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108506-ENSMUSG00000018068\n",
            "\t0 exons with: No file exists\n",
            "\t3 exons with: Only Mapped\n",
            "\t2 exons with: Only nonintersecting\n",
            "\t51 exons with: Only unmapped\n",
            "\t5 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108506-ENSMUSG00000018068/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108506-ENSMUSG00000018068/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 16, Nonintersecting exons= 0, OTHER= 46\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 56, Nonintersecting exons= 0, OTHER= 5\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108506-ENSMUSG00000018068/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108506-ENSMUSG00000018068/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108506-ENSMUSG00000018068/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108506-ENSMUSG00000018068/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108506-ENSMUSG00000018068/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3258\tENSG00000108509-ENSMUSG00000040712\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108509-ENSMUSG00000040712\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108509\tchr17\t4967992\t4987675\t-\tENSG00000108509\tCAMTA2\tprotein_coding\t16\t74\tgene\n",
            "\n",
            "ENSMUSG00000040712\tchr11\t70560289\t70578931\t-\tENSMUSG00000040712\tCamta2\tprotein_coding\t9\t53\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t74\t53\n",
            "\n",
            "Org2\t53\t35\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108509-ENSMUSG00000040712\n",
            "\t0 exons with: No file exists\n",
            "\t7 exons with: Only Mapped\n",
            "\t52 exons with: Only nonintersecting\n",
            "\t5 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t10 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108509-ENSMUSG00000040712\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t34 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t16 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108509-ENSMUSG00000040712/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108509-ENSMUSG00000040712/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 15, Nonintersecting exons= 0, OTHER= 59\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 52, Nonintersecting exons= 0, OTHER= 1\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108509-ENSMUSG00000040712/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108509-ENSMUSG00000040712/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108509-ENSMUSG00000040712/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108509-ENSMUSG00000040712/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108509-ENSMUSG00000040712/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3259\tENSG00000108510-ENSMUSG00000034297\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108510-ENSMUSG00000034297\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108510\tchr17\t61942605\t62065278\t-\tENSG00000108510\tMED13\tprotein_coding\t6\t41\tgene\n",
            "\n",
            "ENSMUSG00000034297\tchr11\t86157859\t86248428\t-\tENSMUSG00000034297\tMed13\tprotein_coding\t4\t39\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t41\t39\n",
            "\n",
            "Org2\t39\t36\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108510-ENSMUSG00000034297\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t33 exons with: Only nonintersecting\n",
            "\t4 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108510-ENSMUSG00000034297\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t2 exons with: Only nonintersecting\n",
            "\t36 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108510-ENSMUSG00000034297/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108510-ENSMUSG00000034297/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 8, Nonintersecting exons= 0, OTHER= 33\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 37, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108510-ENSMUSG00000034297/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108510-ENSMUSG00000034297/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108510-ENSMUSG00000034297/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108510-ENSMUSG00000034297/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108510-ENSMUSG00000034297/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3260\tENSG00000108511-ENSMUSG00000000690\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108511-ENSMUSG00000000690\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108511\tchr17\t48595751\t48604992\t-\tENSG00000108511\tHOXB6\tprotein_coding\t4\t10\tgene\n",
            "\n",
            "ENSMUSG00000000690\tchr11\t96183302\t96192395\t+\tENSMUSG00000000690\tHoxb6\tprotein_coding\t2\t5\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t10\t10\n",
            "\n",
            "Org2\t5\t5\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108511-ENSMUSG00000000690\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t6 exons with: Only nonintersecting\n",
            "\t0 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t4 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108511-ENSMUSG00000000690\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t3 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108511-ENSMUSG00000000690/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108511-ENSMUSG00000000690/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 4, Nonintersecting exons= 0, OTHER= 6\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 5, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108511-ENSMUSG00000000690/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108511-ENSMUSG00000000690/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108511-ENSMUSG00000000690/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108511-ENSMUSG00000000690/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108511-ENSMUSG00000000690/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3261\tENSG00000108515-ENSMUSG00000060600\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108515-ENSMUSG00000060600\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108515\tchr17\t4948092\t4957131\t+\tENSG00000108515\tENO3\tprotein_coding\t17\t46\tgene\n",
            "\n",
            "ENSMUSG00000060600\tchr11\t70548028\t70553339\t+\tENSMUSG00000060600\tEno3\tprotein_coding\t8\t28\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t46\t36\n",
            "\n",
            "Org2\t28\t26\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108515-ENSMUSG00000060600\n",
            "\t0 exons with: No file exists\n",
            "\t4 exons with: Only Mapped\n",
            "\t22 exons with: Only nonintersecting\n",
            "\t8 exons with: Only unmapped\n",
            "\t7 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108515-ENSMUSG00000060600\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t22 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108515-ENSMUSG00000060600/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108515-ENSMUSG00000060600/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 20, Nonintersecting exons= 0, OTHER= 26\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 26, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108515-ENSMUSG00000060600/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108515-ENSMUSG00000060600/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108515-ENSMUSG00000060600/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108515-ENSMUSG00000060600/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108515-ENSMUSG00000060600/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3262\tENSG00000108518-ENSMUSG00000018293\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108518-ENSMUSG00000018293\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108518\tchr17\t4945652\t4949061\t-\tENSG00000108518\tPFN1\tprotein_coding\t3\t8\tgene\n",
            "\n",
            "ENSMUSG00000018293\tchr11\t70542676\t70545470\t-\tENSMUSG00000018293\tPfn1\tprotein_coding\t4\t9\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t8\t8\n",
            "\n",
            "Org2\t9\t9\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108518-ENSMUSG00000018293\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t2 exons with: Only nonintersecting\n",
            "\t0 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108518-ENSMUSG00000018293\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t7 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108518-ENSMUSG00000018293/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108518-ENSMUSG00000018293/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 5, Nonintersecting exons= 0, OTHER= 3\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 8, Nonintersecting exons= 0, OTHER= 1\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108518-ENSMUSG00000018293/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108518-ENSMUSG00000018293/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108518-ENSMUSG00000018293/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108518-ENSMUSG00000018293/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108518-ENSMUSG00000018293/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3263\tENSG00000108523-ENSMUSG00000040746\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108523-ENSMUSG00000040746\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108523\tchr17\t4940008\t4945222\t+\tENSG00000108523\tRNF167\tprotein_coding\t16\t47\tgene\n",
            "\n",
            "ENSMUSG00000040746\tchr11\t70538061\t70542247\t+\tENSMUSG00000040746\tRnf167\tprotein_coding\t10\t37\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t47\t39\n",
            "\n",
            "Org2\t37\t30\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108523-ENSMUSG00000040746\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t24 exons with: Only nonintersecting\n",
            "\t6 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t17 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108523-ENSMUSG00000040746\n",
            "\t0 exons with: No file exists\n",
            "\t3 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t19 exons with: Only unmapped\n",
            "\t8 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t7 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108523-ENSMUSG00000040746/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108523-ENSMUSG00000040746/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 23, Nonintersecting exons= 0, OTHER= 24\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 34, Nonintersecting exons= 0, OTHER= 3\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108523-ENSMUSG00000040746/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108523-ENSMUSG00000040746/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108523-ENSMUSG00000040746/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108523-ENSMUSG00000040746/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108523-ENSMUSG00000040746/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3264\tENSG00000108528-ENSMUSG00000014606\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108528-ENSMUSG00000014606\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108528\tchr17\t4937130\t4940053\t-\tENSG00000108528\tSLC25A11\tprotein_coding\t5\t20\tgene\n",
            "\n",
            "ENSMUSG00000014606\tchr11\t70535022\t70538305\t-\tENSMUSG00000014606\tSlc25a11\tprotein_coding\t5\t21\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t20\t16\n",
            "\n",
            "Org2\t21\t18\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108528-ENSMUSG00000014606\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t14 exons with: Only nonintersecting\n",
            "\t0 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108528-ENSMUSG00000014606\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t19 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108528-ENSMUSG00000014606/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108528-ENSMUSG00000014606/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 6, Nonintersecting exons= 0, OTHER= 14\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 21, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108528-ENSMUSG00000014606/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108528-ENSMUSG00000014606/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108528-ENSMUSG00000014606/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108528-ENSMUSG00000014606/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108528-ENSMUSG00000014606/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3265\tENSG00000108551-ENSMUSG00000049892\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108551-ENSMUSG00000049892\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108551\tchr17\t17494437\t17496395\t-\tENSG00000108551\tRASD1\tprotein_coding\t2\t3\tgene\n",
            "\n",
            "ENSMUSG00000049892\tchr11\t59854007\t59855770\t-\tENSMUSG00000049892\tRasd1\tprotein_coding\t1\t2\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t3\t3\n",
            "\n",
            "Org2\t2\t2\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108551-ENSMUSG00000049892\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t1 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108551-ENSMUSG00000049892\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t2 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108551-ENSMUSG00000049892/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108551-ENSMUSG00000049892/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 3, Nonintersecting exons= 0, OTHER= 0\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 2, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108551-ENSMUSG00000049892/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108551-ENSMUSG00000049892/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108551-ENSMUSG00000049892/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108551-ENSMUSG00000049892/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108551-ENSMUSG00000049892/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3266\tENSG00000108556-ENSMUSG00000014609\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108556-ENSMUSG00000014609\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108556\tchr17\t4897771\t4934438\t-\tENSG00000108556\tCHRNE\tprotein_coding\t5\t32\tgene\n",
            "\n",
            "ENSMUSG00000014609\tchr11\t70505709\t70510042\t-\tENSMUSG00000014609\tChrne\tprotein_coding\t4\t19\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t32\t21\n",
            "\n",
            "Org2\t19\t18\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108556-ENSMUSG00000014609\n",
            "\t0 exons with: No file exists\n",
            "\t8 exons with: Only Mapped\n",
            "\t17 exons with: Only nonintersecting\n",
            "\t5 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108556-ENSMUSG00000014609\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t16 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108556-ENSMUSG00000014609/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108556-ENSMUSG00000014609/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 7, Nonintersecting exons= 0, OTHER= 25\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 19, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108556-ENSMUSG00000014609/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108556-ENSMUSG00000014609/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108556-ENSMUSG00000014609/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108556-ENSMUSG00000014609/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108556-ENSMUSG00000014609/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3267\tENSG00000108557-ENSMUSG00000062115\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108557-ENSMUSG00000062115\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108557\tchr17\t17681458\t17811453\t+\tENSG00000108557\tRAI1\tprotein_coding\t6\t17\tgene\n",
            "\n",
            "ENSMUSG00000062115\tchr11\t59995839\t60090023\t+\tENSMUSG00000062115\tRai1\tprotein_coding\t7\t18\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t17\t17\n",
            "\n",
            "Org2\t18\t15\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108557-ENSMUSG00000062115\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t5 exons with: Only nonintersecting\n",
            "\t6 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108557-ENSMUSG00000062115\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t4 exons with: Only nonintersecting\n",
            "\t14 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108557-ENSMUSG00000062115/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108557-ENSMUSG00000062115/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 12, Nonintersecting exons= 0, OTHER= 5\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 14, Nonintersecting exons= 0, OTHER= 4\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108557-ENSMUSG00000062115/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108557-ENSMUSG00000062115/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108557-ENSMUSG00000062115/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108557-ENSMUSG00000062115/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108557-ENSMUSG00000062115/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3268\tENSG00000108559-ENSMUSG00000040667\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108559-ENSMUSG00000040667\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108559\tchr17\t5359668\t5420164\t-\tENSG00000108559\tNUP88\tprotein_coding\t33\t95\tgene\n",
            "\n",
            "ENSMUSG00000040667\tchr11\t70833884\t70860799\t-\tENSMUSG00000040667\tNup88\tprotein_coding\t15\t63\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t95\t71\n",
            "\n",
            "Org2\t63\t48\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108559-ENSMUSG00000040667\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t51 exons with: Only nonintersecting\n",
            "\t24 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t19 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108559-ENSMUSG00000040667\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t63 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108559-ENSMUSG00000040667/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108559-ENSMUSG00000040667/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 44, Nonintersecting exons= 0, OTHER= 51\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 63, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108559-ENSMUSG00000040667/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108559-ENSMUSG00000040667/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108559-ENSMUSG00000040667/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108559-ENSMUSG00000040667/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108559-ENSMUSG00000040667/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3269\tENSG00000108561-ENSMUSG00000018446\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108561-ENSMUSG00000018446\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108561\tchr17\t5432777\t5448830\t-\tENSG00000108561\tC1QBP\tprotein_coding\t7\t21\tgene\n",
            "\n",
            "ENSMUSG00000018446\tchr11\t70868662\t70873852\t-\tENSMUSG00000018446\tC1qbp\tprotein_coding\t3\t10\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t21\t17\n",
            "\n",
            "Org2\t10\t9\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108561-ENSMUSG00000018446\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t10 exons with: Only nonintersecting\n",
            "\t5 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108561-ENSMUSG00000018446\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t2 exons with: Only nonintersecting\n",
            "\t3 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108561-ENSMUSG00000018446/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108561-ENSMUSG00000018446/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 11, Nonintersecting exons= 0, OTHER= 10\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 8, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108561-ENSMUSG00000018446/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108561-ENSMUSG00000018446/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108561-ENSMUSG00000018446/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108561-ENSMUSG00000018446/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108561-ENSMUSG00000018446/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3270\tENSG00000108576-ENSMUSG00000020838\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108576-ENSMUSG00000020838\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108576\tchr17\t30194319\t30236002\t-\tENSG00000108576\tSLC6A4\tprotein_coding\t6\t26\tgene\n",
            "\n",
            "ENSMUSG00000020838\tchr11\t76889429\t76923166\t+\tENSMUSG00000020838\tSlc6a4\tprotein_coding\t4\t22\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t26\t24\n",
            "\n",
            "Org2\t22\t21\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108576-ENSMUSG00000020838\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t16 exons with: Only nonintersecting\n",
            "\t4 exons with: Only unmapped\n",
            "\t3 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108576-ENSMUSG00000020838\n",
            "\t0 exons with: No file exists\n",
            "\t1 exons with: Only Mapped\n",
            "\t1 exons with: Only nonintersecting\n",
            "\t17 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108576-ENSMUSG00000020838/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108576-ENSMUSG00000020838/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 10, Nonintersecting exons= 0, OTHER= 16\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 20, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108576-ENSMUSG00000020838/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108576-ENSMUSG00000020838/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108576-ENSMUSG00000020838/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108576-ENSMUSG00000020838/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108576-ENSMUSG00000020838/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3271\tENSG00000108578-ENSMUSG00000020840\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108578-ENSMUSG00000020840\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108578\tchr17\t30248203\t30292056\t-\tENSG00000108578\tBLMH\tprotein_coding\t13\t48\tgene\n",
            "\n",
            "ENSMUSG00000020840\tchr11\t76815635\t76878205\t+\tENSMUSG00000020840\tBlmh\tprotein_coding\t9\t41\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t48\t38\n",
            "\n",
            "Org2\t41\t30\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108578-ENSMUSG00000020840\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t31 exons with: Only nonintersecting\n",
            "\t7 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t8 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108578-ENSMUSG00000020840\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t9 exons with: Only nonintersecting\n",
            "\t25 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t7 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108578-ENSMUSG00000020840/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108578-ENSMUSG00000020840/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 15, Nonintersecting exons= 0, OTHER= 33\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 32, Nonintersecting exons= 0, OTHER= 9\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108578-ENSMUSG00000020840/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108578-ENSMUSG00000020840/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108578-ENSMUSG00000020840/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108578-ENSMUSG00000020840/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108578-ENSMUSG00000020840/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3272\tENSG00000108582-ENSMUSG00000020841\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108582-ENSMUSG00000020841\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108582\tchr17\t30378927\t30469989\t+\tENSG00000108582\tCPD\tprotein_coding\t10\t46\tgene\n",
            "\n",
            "ENSMUSG00000020841\tchr11\t76669250\t76737844\t-\tENSMUSG00000020841\tCpd\tprotein_coding\t3\t26\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t46\t40\n",
            "\n",
            "Org2\t26\t25\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108582-ENSMUSG00000020841\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t31 exons with: Only nonintersecting\n",
            "\t5 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108582-ENSMUSG00000020841\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t2 exons with: Only nonintersecting\n",
            "\t20 exons with: Only unmapped\n",
            "\t3 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t1 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108582-ENSMUSG00000020841/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108582-ENSMUSG00000020841/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 13, Nonintersecting exons= 0, OTHER= 33\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 24, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108582-ENSMUSG00000020841/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108582-ENSMUSG00000020841/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108582-ENSMUSG00000020841/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108582-ENSMUSG00000020841/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108582-ENSMUSG00000020841/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3273\tENSG00000108587-ENSMUSG00000010392\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108587-ENSMUSG00000010392\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108587\tchr17\t30477362\t30527592\t+\tENSG00000108587\tGOSR1\tprotein_coding\t12\t41\tgene\n",
            "\n",
            "ENSMUSG00000010392\tchr11\t76617428\t76654405\t-\tENSMUSG00000010392\tGosr1\tprotein_coding\t2\t14\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t41\t32\n",
            "\n",
            "Org2\t14\t10\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108587-ENSMUSG00000010392\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t22 exons with: Only nonintersecting\n",
            "\t9 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t8 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108587-ENSMUSG00000010392\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t14 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108587-ENSMUSG00000010392/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108587-ENSMUSG00000010392/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 17, Nonintersecting exons= 0, OTHER= 24\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 14, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108587-ENSMUSG00000010392/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108587-ENSMUSG00000010392/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108587-ENSMUSG00000010392/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108587-ENSMUSG00000010392/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108587-ENSMUSG00000010392/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3274\tENSG00000108588-ENSMUSG00000078622\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108588-ENSMUSG00000078622\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108588\tchr17\t63745255\t63776351\t-\tENSG00000108588\tCCDC47\tprotein_coding\t6\t28\tgene\n",
            "\n",
            "ENSMUSG00000078622\tchr11\t106088234\t106107170\t-\tENSMUSG00000078622\tCcdc47\tprotein_coding\t5\t21\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t28\t24\n",
            "\n",
            "Org2\t21\t21\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108588-ENSMUSG00000078622\n",
            "\t0 exons with: No file exists\n",
            "\t9 exons with: Only Mapped\n",
            "\t7 exons with: Only nonintersecting\n",
            "\t3 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t7 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108588-ENSMUSG00000078622\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t3 exons with: Only nonintersecting\n",
            "\t14 exons with: Only unmapped\n",
            "\t1 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t3 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108588-ENSMUSG00000078622/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108588-ENSMUSG00000078622/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 12, Nonintersecting exons= 0, OTHER= 16\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 18, Nonintersecting exons= 0, OTHER= 3\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108588-ENSMUSG00000078622/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108588-ENSMUSG00000078622/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108588-ENSMUSG00000078622/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108588-ENSMUSG00000078622/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108588-ENSMUSG00000078622/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3275\tENSG00000108590-ENSMUSG00000020801\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108590-ENSMUSG00000020801\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108590\tchr17\t6643311\t6651634\t-\tENSG00000108590\tMED31\tprotein_coding\t4\t12\tgene\n",
            "\n",
            "ENSMUSG00000020801\tchr11\t72102550\t72106418\t-\tENSMUSG00000020801\tMed31\tprotein_coding\t2\t7\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t12\t10\n",
            "\n",
            "Org2\t7\t6\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108590-ENSMUSG00000020801\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t5 exons with: Only nonintersecting\n",
            "\t2 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108590-ENSMUSG00000020801\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t7 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108590-ENSMUSG00000020801/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108590-ENSMUSG00000020801/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 7, Nonintersecting exons= 0, OTHER= 5\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 7, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108590-ENSMUSG00000020801/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108590-ENSMUSG00000020801/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108590-ENSMUSG00000020801/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108590-ENSMUSG00000020801/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108590-ENSMUSG00000020801/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3276\tENSG00000108591-ENSMUSG00000020537\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108591-ENSMUSG00000020537\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108591\tchr17\t18087892\t18107970\t+\tENSG00000108591\tDRG2\tprotein_coding\t21\t79\tgene\n",
            "\n",
            "ENSMUSG00000020537\tchr11\t60345417\t60359580\t+\tENSMUSG00000020537\tDrg2\tprotein_coding\t5\t28\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t79\t59\n",
            "\n",
            "Org2\t28\t20\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108591-ENSMUSG00000020537\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t57 exons with: Only nonintersecting\n",
            "\t17 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t5 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108591-ENSMUSG00000020537\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t9 exons with: Only nonintersecting\n",
            "\t8 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t7 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108591-ENSMUSG00000020537/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108591-ENSMUSG00000020537/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 22, Nonintersecting exons= 0, OTHER= 57\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 17, Nonintersecting exons= 0, OTHER= 11\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108591-ENSMUSG00000020537/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108591-ENSMUSG00000020537/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108591-ENSMUSG00000020537/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108591-ENSMUSG00000020537/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108591-ENSMUSG00000020537/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3277\tENSG00000108592-ENSMUSG00000020706\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108592-ENSMUSG00000020706\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108592\tchr17\t63819433\t63830012\t-\tENSG00000108592\tFTSJ3\tprotein_coding\t16\t67\tgene\n",
            "\n",
            "ENSMUSG00000020706\tchr11\t106139968\t106146905\t-\tENSMUSG00000020706\tFtsj3\tprotein_coding\t6\t36\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t67\t56\n",
            "\n",
            "Org2\t36\t32\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108592-ENSMUSG00000020706\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t38 exons with: Only nonintersecting\n",
            "\t9 exons with: Only unmapped\n",
            "\t2 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t18 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108592-ENSMUSG00000020706\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t2 exons with: Only nonintersecting\n",
            "\t23 exons with: Only unmapped\n",
            "\t5 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t6 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108592-ENSMUSG00000020706/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108592-ENSMUSG00000020706/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 29, Nonintersecting exons= 0, OTHER= 38\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 34, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108592-ENSMUSG00000020706/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108592-ENSMUSG00000020706/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108592-ENSMUSG00000020706/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108592-ENSMUSG00000020706/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108592-ENSMUSG00000020706/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3278\tENSG00000108599-ENSMUSG00000047804\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108599-ENSMUSG00000047804\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108599\tchr17\t19904302\t19978343\t-\tENSG00000108599\tAKAP10\tprotein_coding\t11\t44\tgene\n",
            "\n",
            "ENSMUSG00000047804\tchr11\t61762133\t61821078\t-\tENSMUSG00000047804\tAkap10\tprotein_coding\t5\t25\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t44\t36\n",
            "\n",
            "Org2\t25\t23\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108599-ENSMUSG00000047804\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t38 exons with: Only nonintersecting\n",
            "\t4 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108599-ENSMUSG00000047804\n",
            "\t0 exons with: No file exists\n",
            "\t2 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t21 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t2 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108599-ENSMUSG00000047804/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108599-ENSMUSG00000047804/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 6, Nonintersecting exons= 0, OTHER= 38\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 23, Nonintersecting exons= 0, OTHER= 2\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108599-ENSMUSG00000047804/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108599-ENSMUSG00000047804/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108599-ENSMUSG00000047804/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108599-ENSMUSG00000047804/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108599-ENSMUSG00000047804/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3279\tENSG00000108602-ENSMUSG00000019102\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108602-ENSMUSG00000019102\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108602\tchr17\t19737984\t19748943\t-\tENSG00000108602\tALDH3A1\tprotein_coding\t17\t49\tgene\n",
            "\n",
            "ENSMUSG00000019102\tchr11\t61098363\t61109247\t+\tENSMUSG00000019102\tAldh3a1\tprotein_coding\t3\t15\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t49\t38\n",
            "\n",
            "Org2\t15\t15\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108602-ENSMUSG00000019102\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t29 exons with: Only nonintersecting\n",
            "\t11 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t9 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108602-ENSMUSG00000019102\n",
            "\t0 exons with: No file exists\n",
            "\t0 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t15 exons with: Only unmapped\n",
            "\t0 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108602-ENSMUSG00000019102/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108602-ENSMUSG00000019102/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 20, Nonintersecting exons= 0, OTHER= 29\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 15, Nonintersecting exons= 0, OTHER= 0\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108602-ENSMUSG00000019102/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108602-ENSMUSG00000019102/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108602-ENSMUSG00000019102/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108602-ENSMUSG00000019102/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108602-ENSMUSG00000019102/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3280\tENSG00000108604-ENSMUSG00000078619\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108604-ENSMUSG00000078619\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108604\tchr17\t63832081\t63843065\t-\tENSG00000108604\tSMARCD2\tprotein_coding\t28\t89\tgene\n",
            "\n",
            "ENSMUSG00000078619\tchr11\t106154005\t106163798\t-\tENSMUSG00000078619\tSmarcd2\tprotein_coding\t9\t40\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t89\t70\n",
            "\n",
            "Org2\t40\t31\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Exon file type summaries for the first gene from: ENSG00000108604-ENSMUSG00000078619\n",
            "\t0 exons with: No file exists\n",
            "\t11 exons with: Only Mapped\n",
            "\t29 exons with: Only nonintersecting\n",
            "\t20 exons with: Only unmapped\n",
            "\t6 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t23 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Exon file type summaries for the second gene from: ENSG00000108604-ENSMUSG00000078619\n",
            "\t0 exons with: No file exists\n",
            "\t7 exons with: Only Mapped\n",
            "\t0 exons with: Only nonintersecting\n",
            "\t3 exons with: Only unmapped\n",
            "\t30 exons with: Mapped and unmapped\n",
            "\t0 exons with: Mapped and nonintersecting\n",
            "\t0 exons with: Nonintersecting and unmapped\n",
            "\t0 exons with: All three files\n",
            "Writing exon-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108604-ENSMUSG00000078619/exonLevelSimilarities-1.0.txt\n",
            "\n",
            "Writing exon classes into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108604-ENSMUSG00000078619/exonClasses-1.0.txt\n",
            "\tFor org1: Mapped exons= 0, Unmapped exons= 49, Nonintersecting exons= 0, OTHER= 40\n",
            "\tFor org2: Mapped exons= 0, Unmapped exons= 33, Nonintersecting exons= 0, OTHER= 7\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n",
            "Writing exon-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108604-ENSMUSG00000078619/exonLevelMappings-1.0.txt\n",
            "Writing trascript-level similarity scores into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108604-ENSMUSG00000078619/transcriptLevelSimilarities-1.0.txt\n",
            "Writing transcript-level mappings into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108604-ENSMUSG00000078619/transcriptLevelMappings-1.0.txt\n",
            "\n",
            "Condition counter from the greedy transcript mapping stage:\n",
            "\t0 pairs with Condition1: Unique winner pair\n",
            "\t0 pairs with Condition2: Tie in one score, not in the other\n",
            "\t0 pairs with Condition3: Tie in both scores but coding exon length diff breaks the tie\n",
            "\t0 pairs with Condition4: Tie in both scores and coding exon length diff but overall exon length breaks the tie\n",
            "\t0 pairs with Condition5: Tie in all the above but coding length (bp) diff breaks the tie\n",
            "\t0 pairs with Condition6: Tie in all the above, just give up and report all\n",
            "\n",
            "Writing UCSC browser bed output for org1 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108604-ENSMUSG00000078619/org1-ucsc-1.0.bed\n",
            "Writing UCSC browser bed output for org2 into file:\n",
            " /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse/ENSG00000108604-ENSMUSG00000078619/org2-ucsc-1.0.bed\n",
            "Finding exon mappings for gene pair number 3281\tENSG00000108622-ENSMUSG00000001029\n",
            "*****************************************************************\n",
            "Gene pair ID: ENSG00000108622-ENSMUSG00000001029\n",
            "\n",
            "Information about each gene. Last two numbers are no of transcripts and exons\n",
            "ENSG00000108622\tchr17\t64002594\t64020634\t-\tENSG00000108622\tICAM2\tprotein_coding\t16\t40\tgene\n",
            "\n",
            "ENSMUSG00000001029\tchr11\t106268482\t106278901\t-\tENSMUSG00000001029\tIcam2\tprotein_coding\t5\t15\tgene\n",
            "\n",
            "Number of exons before and after duplicate removal according to coordinates\n",
            "Org1\t40\t33\n",
            "\n",
            "Org2\t15\t14\n",
            "\n",
            "*****************************************************************\n",
            "\n",
            "*****************************************************************\n"
          ]
        }
      ],
      "source": [
        "!python ExTraMapper.py -m 1 -o1 human -o2 mouse -p all"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "z9MNUxYNGzbl"
      },
      "outputs": [],
      "source": [
        "!ls -hltr"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "cvRdTlpzHAq0"
      },
      "outputs": [],
      "source": [
        "#summarize result, outputs a dataframe\n",
        "!chmod +x extMsummarise"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "6QZwsN2aLjTx"
      },
      "outputs": [],
      "source": [
        "!ls -hltr /content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse | wc -l"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "xH1ejfFWNVWf"
      },
      "outputs": [],
      "source": [
        "#create a list of orthologous genes for input into summary code\n",
        "import os\n",
        "\n",
        "# Specify the directory you want to search\n",
        "directory = \"/content/ExTraMapper/Human-Mouse-Preprocess-Data/output/human-mouse\"\n",
        "\n",
        "# Specify the output file name\n",
        "output_file = \"gene-pair.list\"\n",
        "\n",
        "# Get all subdirectories within the specified directory\n",
        "subdirectories = [subdir for subdir in os.listdir(directory) if os.path.isdir(os.path.join(directory, subdir))]\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "5i6jBFrlNkVa"
      },
      "outputs": [],
      "source": [
        "# Write the subdirectories to the output file\n",
        "with open(output_file, 'w') as file:\n",
        "    file.write('\\n'.join(subdirectories))\n",
        "\n",
        "print(f\"Subdirectories list has been saved to {output_file}.\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "jPFYCwSIHDaF"
      },
      "outputs": [],
      "source": [
        "!./extMsummarise /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess /content/ExTraMapper/Human-Mouse-Preprocess-Data/output /content/ExTraMapper/gene-pair.list human mouse extramapper-result-ensembl109"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "WLwtfBTyQKHV"
      },
      "outputs": [],
      "source": [
        "## optional, copy output to google drive ##\n",
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "c_5VlwiJumaG"
      },
      "outputs": [],
      "source": [
        "!cp -r /content/ExTraMapper/Human-Mouse-Preprocess-Data/output /content/drive/MyDrive/ExTraMapper"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "Of9kyL0C2ZaS"
      },
      "outputs": [],
      "source": [
        "!cp -r /content/ExTraMapper/Human-Mouse-Preprocess-Data/preprocess/data/human-mouse/GTFsummaries /content/drive/MyDrive/ExtramapperGTFs"
      ]
    }
  ],
  "metadata": {
    "colab": {
      "provenance": []
    },
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}